
[centos@localhost test]$ amm
OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N
Loading...
Welcome to the Ammonite Repl 1.0.3
(Scala 2.12.4 Java 1.8.0_141)
If you like Ammonite, please support our development at www.patreon.com/lihaoyi
centos-test@ import $file.analysis 
import $file.$       

centos-test@ 
 var unique_genome_list = analysis.find_unique_genomes_fastq(analysis.all_fastq_files).toList 
unique_genome_list: List[String] = List("G04878")

centos-test@  analysis.analyse_all_genomes(unique_genome_list) 


>>>>> GENOME => G04878




>>>>> copy_reference_genome <<<<<




>>>>> merge_and_rename <<<<<




 ~~~~~~~~~~~~~~~~~~~~~ 

working on the G04878 files



>>>>       R1      <<<<


cat /media/sf_BioDragon/Genesis/test/BSSE_QGF_40823_C7JREANXX_6_D10_G04878_LRu_261015_CGAGGCTG_AGAGTAGA_S76_L006_R1_001.fastq /media/sf_BioDragon/Genesis/test/BSSE_QGF_40823_C7JREANXX_7_D10_G04878_LRu_261015_CGAGGCTG_AGAGTAGA_S76_L007_R1_001.fastq /media/sf_BioDragon/Genesis/test/BSSE_QGF_40823_C898AANXX_3_D10_G04878_LRu_261015_CGAGGCTG_AGAGTAGA_S76_L003_R1_001.fastq  > /media/sf_BioDragon/Genesis/test/G04878_analysis/G04878_1.fastq






>>>>       R2      <<<<


cat /media/sf_BioDragon/Genesis/test/BSSE_QGF_40823_C7JREANXX_6_D10_G04878_LRu_261015_CGAGGCTG_AGAGTAGA_S76_L006_R2_001.fastq /media/sf_BioDragon/Genesis/test/BSSE_QGF_40823_C7JREANXX_7_D10_G04878_LRu_261015_CGAGGCTG_AGAGTAGA_S76_L007_R2_001.fastq /media/sf_BioDragon/Genesis/test/BSSE_QGF_40823_C898AANXX_3_D10_G04878_LRu_261015_CGAGGCTG_AGAGTAGA_S76_L003_R2_001.fastq  > /media/sf_BioDragon/Genesis/test/G04878_analysis/G04878_2.fastq





@@@@@@@@@@@@@@@@@@@@@@@@


All Done!


>>>>> Current Directory => /media/sf_BioDragon/Genesis/test




>>>>> move_into_genome_folder <<<<<




>>>>> Current Directory => /media/sf_BioDragon/Genesis/test/G04878_analysis




>>>>> trimmomatic <<<<<


java -jar /opt/Trimmomatic-0.36/trimmomatic-0.36.jar PE -phred33 G04878_1.fastq G04878_2.fastq G04878_1_trimmed_paired.fastq G04878_1_trimmed_unpaired.fastq G04878_2_trimmed_paired.fastq G04878_2_trimmed_unpaired.fastq LEADING:3 TRAILING:3 SLIDINGWINDOW:4:20 MINLEN:36


TrimmomaticPE: Started with arguments:
 -phred33 G04878_1.fastq G04878_2.fastq G04878_1_trimmed_paired.fastq G04878_1_trimmed_unpaired.fastq G04878_2_trimmed_paired.fastq G04878_2_trimmed_unpaired.fastq LEADING:3 TRAILING:3 SLIDINGWINDOW:4:20 MINLEN:36
Input Read Pairs: 4964126 Both Surviving: 4166621 (83.93%) Forward Only Surviving: 433770 (8.74%) Reverse Only Surviving: 68389 (1.38%) Dropped: 295346 (5.95%)
TrimmomaticPE: Completed successfully





>>>>> bwa_index_reference_genome <<<<<


bwa index NC000962_3.fasta


[bwa_index] Pack FASTA... 0.02 sec
[bwa_index] Construct BWT for the packed sequence...
[bwa_index] 0.98 seconds elapse.
[bwa_index] Update BWT... 0.03 sec
[bwa_index] Pack forward-only FASTA... 0.03 sec
[bwa_index] Construct SA from BWT and Occ... 0.28 sec
[main] Version: 0.7.12-r1039
[main] CMD: bwa index NC000962_3.fasta
[main] Real time: 1.682 sec; CPU: 1.355 sec


>>>>> map_and_generate_sam_file <<<<<


bwa mem -R "@RG\tID:G04878\tSM:G04878\tPL:Illumina" -M NC000962_3.fasta G04878_1_trimmed_paired.fastq G04878_2_trimmed_paired.fastq > G04878.sam


[M::bwa_idx_load_from_disk] read 0 ALT contigs
[M::process] read 100602 sequences (10000195 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (108, 48283, 23, 98)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (84, 164, 272)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 648)
[M::mem_pestat] mean and std.dev: (196.15, 144.49)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 836)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (133, 214, 334)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 736)
[M::mem_pestat] mean and std.dev: (245.01, 144.58)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 937)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (410, 719, 3840)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 10700)
[M::mem_pestat] mean and std.dev: (1892.13, 2214.41)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 14130)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (95, 161, 282)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 656)
[M::mem_pestat] mean and std.dev: (199.56, 137.53)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 843)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100602 reads in 2.548 CPU sec, 2.664 real sec
[M::process] read 100670 sequences (10000165 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (108, 48349, 16, 89)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (86, 154, 259)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 605)
[M::mem_pestat] mean and std.dev: (175.47, 123.04)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 778)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (130, 207, 321)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 703)
[M::mem_pestat] mean and std.dev: (236.30, 138.47)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 894)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (288, 587, 2430)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6714)
[M::mem_pestat] mean and std.dev: (1284.19, 1600.26)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8856)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (96, 154, 297)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 699)
[M::mem_pestat] mean and std.dev: (200.03, 138.79)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 900)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100670 reads in 2.347 CPU sec, 2.673 real sec
[M::process] read 101082 sequences (10000138 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (106, 48512, 21, 98)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (91, 173, 283)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 667)
[M::mem_pestat] mean and std.dev: (192.79, 125.09)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 859)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (130, 208, 321)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 703)
[M::mem_pestat] mean and std.dev: (236.35, 137.32)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 894)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (134, 468, 2429)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 7019)
[M::mem_pestat] mean and std.dev: (1477.10, 1641.62)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 9314)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (93, 176, 249)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 561)
[M::mem_pestat] mean and std.dev: (172.51, 102.75)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 717)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101082 reads in 2.209 CPU sec, 2.644 real sec
[M::process] read 100486 sequences (10000086 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (121, 48181, 16, 97)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (97, 187, 286)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 664)
[M::mem_pestat] mean and std.dev: (206.75, 135.59)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 853)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (133, 218, 337)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 745)
[M::mem_pestat] mean and std.dev: (247.65, 145.87)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 949)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (232, 735, 4370)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12646)
[M::mem_pestat] mean and std.dev: (1707.44, 1851.24)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 16784)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (82, 182, 310)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 766)
[M::mem_pestat] mean and std.dev: (216.46, 168.36)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 994)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100486 reads in 2.202 CPU sec, 2.583 real sec
[M::process] read 100412 sequences (10000014 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (101, 48224, 14, 101)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (91, 162, 281)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 661)
[M::mem_pestat] mean and std.dev: (183.74, 112.89)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 851)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (134, 219, 342)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 758)
[M::mem_pestat] mean and std.dev: (251.20, 149.89)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 966)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (182, 878, 1410)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 3866)
[M::mem_pestat] mean and std.dev: (599.00, 522.43)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 5094)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (102, 200, 291)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 669)
[M::mem_pestat] mean and std.dev: (201.56, 119.58)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 858)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100412 reads in 2.256 CPU sec, 2.626 real sec
[M::process] read 100360 sequences (10000144 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (98, 48154, 9, 100)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (108, 185, 280)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 624)
[M::mem_pestat] mean and std.dev: (204.04, 126.26)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 796)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 219, 343)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 759)
[M::mem_pestat] mean and std.dev: (251.34, 149.43)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 967)
[M::mem_pestat] skip orientation RF as there are not enough pairs
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (81, 187, 331)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 831)
[M::mem_pestat] mean and std.dev: (219.40, 167.16)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1081)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100360 reads in 2.133 CPU sec, 2.608 real sec
[M::process] read 100466 sequences (10000070 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (117, 48181, 13, 100)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (91, 173, 288)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 682)
[M::mem_pestat] mean and std.dev: (209.41, 146.85)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 879)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 218, 341)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 753)
[M::mem_pestat] mean and std.dev: (250.14, 149.10)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (688, 1240, 2447)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 5965)
[M::mem_pestat] mean and std.dev: (1499.33, 1217.58)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 7724)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (119, 200, 321)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 725)
[M::mem_pestat] mean and std.dev: (225.77, 143.38)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 927)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100466 reads in 2.254 CPU sec, 2.650 real sec
[M::process] read 101104 sequences (10000054 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (108, 48552, 15, 84)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (110, 231, 380)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 920)
[M::mem_pestat] mean and std.dev: (257.62, 186.62)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1190)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 213, 334)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 738)
[M::mem_pestat] mean and std.dev: (244.61, 145.89)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 940)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (153, 530, 1102)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 3000)
[M::mem_pestat] mean and std.dev: (408.25, 339.38)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 3949)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (79, 142, 244)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 574)
[M::mem_pestat] mean and std.dev: (169.22, 123.99)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 739)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101104 reads in 2.168 CPU sec, 2.590 real sec
[M::process] read 100638 sequences (10000124 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (104, 48355, 16, 99)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (115, 202, 300)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 670)
[M::mem_pestat] mean and std.dev: (213.24, 129.00)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 855)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 215, 338)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 752)
[M::mem_pestat] mean and std.dev: (247.59, 149.57)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (303, 465, 2435)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6699)
[M::mem_pestat] mean and std.dev: (1336.69, 1619.10)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8831)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (89, 180, 323)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 791)
[M::mem_pestat] mean and std.dev: (197.24, 131.71)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1025)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100638 reads in 2.196 CPU sec, 2.597 real sec
[M::process] read 101120 sequences (10000081 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (102, 48556, 11, 85)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (99, 181, 316)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 750)
[M::mem_pestat] mean and std.dev: (217.94, 147.80)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 967)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 214, 335)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 741)
[M::mem_pestat] mean and std.dev: (246.03, 146.92)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 944)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (514, 1483, 2370)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6082)
[M::mem_pestat] mean and std.dev: (1531.73, 1265.26)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 7938)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (93, 163, 288)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 678)
[M::mem_pestat] mean and std.dev: (197.27, 122.89)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 873)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101120 reads in 2.219 CPU sec, 2.613 real sec
[M::process] read 100750 sequences (10000162 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (117, 48321, 12, 90)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (94, 165, 262)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 598)
[M::mem_pestat] mean and std.dev: (188.42, 120.78)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 766)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 220, 342)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 756)
[M::mem_pestat] mean and std.dev: (251.02, 149.31)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (391, 742, 1518)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 3772)
[M::mem_pestat] mean and std.dev: (848.64, 671.20)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 4899)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (111, 193, 299)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 675)
[M::mem_pestat] mean and std.dev: (203.06, 117.77)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 863)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100750 reads in 2.224 CPU sec, 2.588 real sec
[M::process] read 100484 sequences (10000058 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (104, 48211, 17, 117)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (86, 178, 285)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 683)
[M::mem_pestat] mean and std.dev: (194.21, 129.26)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 882)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (136, 222, 348)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 772)
[M::mem_pestat] mean and std.dev: (255.10, 152.64)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 984)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (215, 369, 2282)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6416)
[M::mem_pestat] mean and std.dev: (1289.88, 1847.37)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8483)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (101, 165, 311)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 731)
[M::mem_pestat] mean and std.dev: (222.34, 159.77)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 941)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100484 reads in 2.241 CPU sec, 2.615 real sec
[M::process] read 100384 sequences (10000183 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (93, 48229, 14, 89)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (98, 195, 355)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 869)
[M::mem_pestat] mean and std.dev: (237.91, 167.16)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1126)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 219, 342)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 756)
[M::mem_pestat] mean and std.dev: (251.26, 150.00)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (200, 1501, 4714)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 13742)
[M::mem_pestat] mean and std.dev: (2119.71, 2204.20)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 18256)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (80, 178, 262)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 626)
[M::mem_pestat] mean and std.dev: (178.85, 121.02)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 808)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100384 reads in 2.042 CPU sec, 2.593 real sec
[M::process] read 100582 sequences (10000086 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (88, 48294, 20, 95)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (105, 221, 280)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 630)
[M::mem_pestat] mean and std.dev: (218.06, 141.81)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 805)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 220, 344)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 762)
[M::mem_pestat] mean and std.dev: (252.75, 151.61)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 971)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (393, 1326, 3299)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 9111)
[M::mem_pestat] mean and std.dev: (1708.70, 1591.02)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 12017)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (107, 168, 269)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 593)
[M::mem_pestat] mean and std.dev: (177.00, 103.88)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 755)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100582 reads in 2.211 CPU sec, 2.586 real sec
[M::process] read 100680 sequences (10000173 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (109, 48284, 20, 106)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (115, 202, 332)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 766)
[M::mem_pestat] mean and std.dev: (227.66, 144.88)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 983)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (134, 217, 340)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 752)
[M::mem_pestat] mean and std.dev: (249.43, 149.57)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (376, 1349, 4224)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 11920)
[M::mem_pestat] mean and std.dev: (1852.00, 1689.48)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 15768)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (96, 193, 293)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 687)
[M::mem_pestat] mean and std.dev: (206.76, 135.28)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 884)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100680 reads in 2.244 CPU sec, 2.622 real sec
[M::process] read 101022 sequences (10000073 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (109, 48500, 18, 94)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (113, 213, 344)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 806)
[M::mem_pestat] mean and std.dev: (241.20, 164.56)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1037)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 213, 338)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 752)
[M::mem_pestat] mean and std.dev: (246.72, 149.81)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (213, 1677, 2972)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 8490)
[M::mem_pestat] mean and std.dev: (1907.22, 1926.62)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 11249)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (79, 131, 269)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 649)
[M::mem_pestat] mean and std.dev: (185.42, 143.83)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 839)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101022 reads in 2.226 CPU sec, 2.611 real sec
[M::process] read 101140 sequences (10000098 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (98, 48539, 11, 100)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (119, 182, 281)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 605)
[M::mem_pestat] mean and std.dev: (202.78, 121.15)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 767)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 212, 333)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 737)
[M::mem_pestat] mean and std.dev: (244.46, 146.96)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 939)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (456, 1926, 2622)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6954)
[M::mem_pestat] mean and std.dev: (1922.00, 1623.86)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 9120)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (101, 161, 309)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 725)
[M::mem_pestat] mean and std.dev: (197.15, 141.07)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 933)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101140 reads in 2.237 CPU sec, 2.634 real sec
[M::process] read 100274 sequences (10000114 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (99, 48104, 14, 99)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (105, 189, 303)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 699)
[M::mem_pestat] mean and std.dev: (211.41, 132.16)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 897)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (136, 220, 342)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 754)
[M::mem_pestat] mean and std.dev: (251.22, 148.47)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 960)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (140, 1439, 2545)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 7355)
[M::mem_pestat] mean and std.dev: (1559.29, 1498.29)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 9760)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (77, 150, 254)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 608)
[M::mem_pestat] mean and std.dev: (180.04, 130.88)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 785)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100274 reads in 2.248 CPU sec, 2.607 real sec
[M::process] read 100310 sequences (10000001 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (101, 48113, 13, 110)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (93, 177, 308)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 738)
[M::mem_pestat] mean and std.dev: (215.10, 151.15)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 953)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 220, 347)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 771)
[M::mem_pestat] mean and std.dev: (253.07, 152.20)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 983)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (123, 485, 1528)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 4338)
[M::mem_pestat] mean and std.dev: (960.08, 1065.42)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 5743)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (99, 170, 296)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 690)
[M::mem_pestat] mean and std.dev: (210.12, 136.28)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 887)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100310 reads in 2.233 CPU sec, 2.620 real sec
[M::process] read 100340 sequences (10000074 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (109, 48045, 6, 111)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (105, 174, 276)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 618)
[M::mem_pestat] mean and std.dev: (192.14, 123.24)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 789)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (136, 220, 343)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 757)
[M::mem_pestat] mean and std.dev: (251.66, 149.90)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 964)
[M::mem_pestat] skip orientation RF as there are not enough pairs
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (100, 177, 298)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 694)
[M::mem_pestat] mean and std.dev: (199.01, 126.81)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 892)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100340 reads in 2.237 CPU sec, 2.586 real sec
[M::process] read 100544 sequences (10000061 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (105, 48243, 13, 105)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (131, 184, 293)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 617)
[M::mem_pestat] mean and std.dev: (206.80, 125.34)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 779)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 212, 333)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 737)
[M::mem_pestat] mean and std.dev: (244.04, 146.19)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 939)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (222, 395, 4424)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12828)
[M::mem_pestat] mean and std.dev: (1447.38, 1803.83)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 17030)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (94, 193, 331)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 805)
[M::mem_pestat] mean and std.dev: (222.19, 161.08)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1042)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100544 reads in 2.207 CPU sec, 2.621 real sec
[M::process] read 100826 sequences (10000054 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (108, 48429, 14, 103)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (94, 169, 343)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 841)
[M::mem_pestat] mean and std.dev: (233.53, 187.92)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1090)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 210, 327)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 719)
[M::mem_pestat] mean and std.dev: (241.02, 142.66)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 915)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (220, 475, 694)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 1642)
[M::mem_pestat] mean and std.dev: (403.83, 358.85)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 2116)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (125, 186, 324)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 722)
[M::mem_pestat] mean and std.dev: (217.57, 131.63)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 921)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100826 reads in 2.200 CPU sec, 2.607 real sec
[M::process] read 100930 sequences (10000034 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (107, 48418, 15, 111)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (99, 195, 320)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 762)
[M::mem_pestat] mean and std.dev: (223.18, 154.70)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 983)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (130, 209, 328)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 724)
[M::mem_pestat] mean and std.dev: (240.78, 143.68)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 922)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (352, 1520, 4504)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12808)
[M::mem_pestat] mean and std.dev: (2382.27, 2381.60)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 16960)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (108, 180, 283)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 633)
[M::mem_pestat] mean and std.dev: (197.81, 119.52)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 808)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100930 reads in 2.177 CPU sec, 2.634 real sec
[M::process] read 101104 sequences (10000129 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (117, 48506, 12, 88)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (100, 196, 315)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 745)
[M::mem_pestat] mean and std.dev: (217.64, 150.98)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 960)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (129, 208, 325)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 717)
[M::mem_pestat] mean and std.dev: (238.59, 141.33)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 913)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (244, 494, 2329)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6499)
[M::mem_pestat] mean and std.dev: (1085.50, 1337.45)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8584)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (95, 158, 299)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 707)
[M::mem_pestat] mean and std.dev: (204.08, 138.93)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 911)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101104 reads in 2.261 CPU sec, 2.620 real sec
[M::process] read 100350 sequences (10000159 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (101, 48142, 16, 103)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (105, 192, 301)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 693)
[M::mem_pestat] mean and std.dev: (210.67, 134.89)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 889)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 214, 333)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 735)
[M::mem_pestat] mean and std.dev: (244.55, 144.54)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 936)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (306, 1218, 4516)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12936)
[M::mem_pestat] mean and std.dev: (1957.94, 1892.34)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 17146)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (105, 205, 319)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 747)
[M::mem_pestat] mean and std.dev: (230.17, 150.57)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 961)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100350 reads in 2.253 CPU sec, 2.597 real sec
[M::process] read 100384 sequences (10000057 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (115, 48169, 16, 102)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (115, 222, 322)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 736)
[M::mem_pestat] mean and std.dev: (215.10, 126.99)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 943)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (134, 216, 335)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 737)
[M::mem_pestat] mean and std.dev: (246.33, 145.40)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 938)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (208, 1267, 2816)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 8032)
[M::mem_pestat] mean and std.dev: (1529.12, 1627.74)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 10640)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (107, 204, 304)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 698)
[M::mem_pestat] mean and std.dev: (208.83, 128.82)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 895)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100384 reads in 2.225 CPU sec, 2.619 real sec
[M::process] read 100546 sequences (10000050 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (119, 48153, 19, 107)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (98, 174, 253)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 563)
[M::mem_pestat] mean and std.dev: (179.96, 112.21)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 718)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (133, 215, 336)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 742)
[M::mem_pestat] mean and std.dev: (246.30, 146.17)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 945)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (443, 951, 2376)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6242)
[M::mem_pestat] mean and std.dev: (1408.53, 1370.13)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8175)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (76, 156, 284)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 700)
[M::mem_pestat] mean and std.dev: (187.52, 131.40)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 908)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100546 reads in 2.304 CPU sec, 2.663 real sec
[M::process] read 100712 sequences (10000150 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (94, 48336, 13, 105)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (96, 196, 308)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 732)
[M::mem_pestat] mean and std.dev: (222.70, 162.83)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 944)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (133, 215, 336)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 742)
[M::mem_pestat] mean and std.dev: (246.75, 146.96)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 945)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (399, 771, 3686)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 10260)
[M::mem_pestat] mean and std.dev: (1472.08, 1556.24)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 13547)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (92, 163, 250)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 566)
[M::mem_pestat] mean and std.dev: (184.71, 116.72)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 724)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100712 reads in 2.224 CPU sec, 2.582 real sec
[M::process] read 100664 sequences (10000129 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (108, 48257, 11, 99)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (110, 185, 304)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 692)
[M::mem_pestat] mean and std.dev: (213.76, 141.35)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 886)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 212, 335)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 743)
[M::mem_pestat] mean and std.dev: (245.01, 146.79)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 947)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (124, 792, 4281)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12595)
[M::mem_pestat] mean and std.dev: (1911.00, 2168.63)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 16752)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (96, 145, 236)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 516)
[M::mem_pestat] mean and std.dev: (170.07, 109.77)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 656)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100664 reads in 2.224 CPU sec, 2.604 real sec
[M::process] read 101232 sequences (10000021 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (104, 48614, 16, 81)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (101, 191, 326)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 776)
[M::mem_pestat] mean and std.dev: (211.93, 140.13)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1001)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (130, 210, 328)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 724)
[M::mem_pestat] mean and std.dev: (240.93, 143.33)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 922)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (373, 2714, 4387)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12415)
[M::mem_pestat] mean and std.dev: (2414.06, 1929.89)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 16429)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (93, 147, 294)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 696)
[M::mem_pestat] mean and std.dev: (196.69, 146.32)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 897)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101232 reads in 2.248 CPU sec, 2.636 real sec
[M::process] read 100770 sequences (10000081 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (95, 48296, 11, 91)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (91, 189, 291)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 691)
[M::mem_pestat] mean and std.dev: (213.38, 140.33)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 891)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 219, 340)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 750)
[M::mem_pestat] mean and std.dev: (250.55, 148.03)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 955)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (261, 2348, 4386)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12636)
[M::mem_pestat] mean and std.dev: (2266.27, 1861.47)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 16761)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (122, 209, 323)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 725)
[M::mem_pestat] mean and std.dev: (222.38, 137.88)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 926)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100770 reads in 2.268 CPU sec, 2.645 real sec
[M::process] read 100302 sequences (10000075 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (113, 48144, 19, 97)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (92, 189, 309)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 743)
[M::mem_pestat] mean and std.dev: (220.23, 160.81)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 960)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (136, 221, 345)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 763)
[M::mem_pestat] mean and std.dev: (252.93, 150.85)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 972)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (270, 1363, 4459)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12837)
[M::mem_pestat] mean and std.dev: (1961.00, 1930.34)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 17026)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (115, 160, 307)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 691)
[M::mem_pestat] mean and std.dev: (203.93, 132.84)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 883)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100302 reads in 2.227 CPU sec, 2.599 real sec
[M::process] read 100476 sequences (10000156 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (97, 48220, 14, 99)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (92, 207, 306)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 734)
[M::mem_pestat] mean and std.dev: (214.72, 142.74)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 948)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (137, 223, 351)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 779)
[M::mem_pestat] mean and std.dev: (256.24, 153.21)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 993)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (278, 1644, 4483)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12893)
[M::mem_pestat] mean and std.dev: (2304.79, 2231.19)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 17098)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (106, 186, 295)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 673)
[M::mem_pestat] mean and std.dev: (209.04, 132.73)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 862)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100476 reads in 2.276 CPU sec, 2.647 real sec
[M::process] read 100380 sequences (10000198 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (117, 48161, 17, 106)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (110, 206, 323)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 749)
[M::mem_pestat] mean and std.dev: (228.42, 151.57)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 962)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 218, 339)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 747)
[M::mem_pestat] mean and std.dev: (248.87, 147.56)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (274, 1173, 4665)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 13447)
[M::mem_pestat] mean and std.dev: (2103.94, 1948.15)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 17838)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (80, 170, 282)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 686)
[M::mem_pestat] mean and std.dev: (192.51, 133.51)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 888)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100380 reads in 2.247 CPU sec, 2.641 real sec
[M::process] read 100292 sequences (10000156 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (114, 48083, 17, 107)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (136, 199, 355)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 793)
[M::mem_pestat] mean and std.dev: (232.28, 144.77)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1012)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 218, 335)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 735)
[M::mem_pestat] mean and std.dev: (247.00, 144.24)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 935)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (508, 1485, 2335)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 5989)
[M::mem_pestat] mean and std.dev: (1573.00, 1287.46)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 7816)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (113, 192, 280)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 614)
[M::mem_pestat] mean and std.dev: (208.89, 126.15)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 781)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100292 reads in 2.258 CPU sec, 2.612 real sec
[M::process] read 100412 sequences (10000028 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (105, 48117, 11, 120)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (80, 151, 295)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 725)
[M::mem_pestat] mean and std.dev: (200.12, 154.79)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 940)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (134, 216, 337)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 743)
[M::mem_pestat] mean and std.dev: (247.64, 147.31)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 946)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (660, 961, 4234)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 11382)
[M::mem_pestat] mean and std.dev: (1762.18, 1739.52)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 14956)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (98, 202, 333)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 803)
[M::mem_pestat] mean and std.dev: (232.20, 169.26)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1038)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100412 reads in 2.239 CPU sec, 2.641 real sec
[M::process] read 100518 sequences (10000038 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (114, 48272, 16, 76)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (92, 197, 312)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 752)
[M::mem_pestat] mean and std.dev: (224.67, 164.91)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 972)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 219, 343)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 759)
[M::mem_pestat] mean and std.dev: (251.23, 150.08)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 967)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (398, 1513, 2775)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 7529)
[M::mem_pestat] mean and std.dev: (1887.62, 1615.52)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 9906)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (110, 191, 328)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 764)
[M::mem_pestat] mean and std.dev: (228.66, 154.70)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 982)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100518 reads in 2.255 CPU sec, 2.638 real sec
[M::process] read 100598 sequences (10000183 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (117, 48205, 12, 96)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (95, 156, 233)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 509)
[M::mem_pestat] mean and std.dev: (168.49, 103.97)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 647)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 212, 331)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 729)
[M::mem_pestat] mean and std.dev: (242.71, 143.87)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 928)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (67, 1575, 3872)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 11482)
[M::mem_pestat] mean and std.dev: (1929.50, 1791.90)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 15287)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (105, 180, 286)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 648)
[M::mem_pestat] mean and std.dev: (200.47, 123.09)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 829)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100598 reads in 2.298 CPU sec, 2.647 real sec
[M::process] read 100826 sequences (10000061 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (91, 48357, 13, 95)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (94, 190, 285)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 667)
[M::mem_pestat] mean and std.dev: (195.48, 122.53)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 858)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 211, 330)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 728)
[M::mem_pestat] mean and std.dev: (242.59, 143.65)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 927)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (1210, 1539, 3022)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6646)
[M::mem_pestat] mean and std.dev: (1891.38, 1285.54)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8458)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (112, 180, 316)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 724)
[M::mem_pestat] mean and std.dev: (207.16, 131.59)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 928)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100826 reads in 2.246 CPU sec, 2.620 real sec
[M::process] read 101220 sequences (10000066 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (108, 48541, 19, 84)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (85, 168, 264)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 622)
[M::mem_pestat] mean and std.dev: (179.46, 116.00)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 801)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 212, 329)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 723)
[M::mem_pestat] mean and std.dev: (242.07, 142.69)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 920)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (277, 545, 2297)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6337)
[M::mem_pestat] mean and std.dev: (1235.53, 1262.75)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8357)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (96, 163, 260)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 588)
[M::mem_pestat] mean and std.dev: (182.56, 121.26)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 752)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101220 reads in 2.275 CPU sec, 2.648 real sec
[M::process] read 100916 sequences (10000117 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (112, 48439, 11, 87)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (103, 188, 329)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 781)
[M::mem_pestat] mean and std.dev: (218.43, 145.89)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1007)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (130, 211, 329)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 727)
[M::mem_pestat] mean and std.dev: (241.47, 143.22)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 926)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (184, 273, 2139)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6049)
[M::mem_pestat] mean and std.dev: (1088.82, 1507.14)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8004)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (73, 167, 286)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 712)
[M::mem_pestat] mean and std.dev: (190.37, 138.32)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 925)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100916 reads in 2.258 CPU sec, 2.610 real sec
[M::process] read 101050 sequences (10000005 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (108, 48478, 17, 99)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (87, 180, 264)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 618)
[M::mem_pestat] mean and std.dev: (191.32, 116.87)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 795)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (129, 208, 325)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 717)
[M::mem_pestat] mean and std.dev: (238.30, 141.26)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 913)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (240, 836, 2140)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 5940)
[M::mem_pestat] mean and std.dev: (1243.00, 1190.07)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 7840)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (102, 174, 277)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 627)
[M::mem_pestat] mean and std.dev: (198.63, 130.95)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 802)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101050 reads in 2.213 CPU sec, 2.636 real sec
[M::process] read 101842 sequences (10000047 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (110, 48912, 14, 71)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (95, 166, 284)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 662)
[M::mem_pestat] mean and std.dev: (191.23, 128.74)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 851)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (127, 203, 315)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 691)
[M::mem_pestat] mean and std.dev: (232.45, 136.95)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 879)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (77, 1485, 2409)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 7073)
[M::mem_pestat] mean and std.dev: (1243.69, 1459.29)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 9405)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (93, 159, 276)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 642)
[M::mem_pestat] mean and std.dev: (191.25, 133.03)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 825)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101842 reads in 2.248 CPU sec, 2.626 real sec
[M::process] read 100970 sequences (10000199 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (112, 48461, 16, 99)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (78, 153, 315)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 789)
[M::mem_pestat] mean and std.dev: (206.74, 155.21)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1026)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 211, 325)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 713)
[M::mem_pestat] mean and std.dev: (239.71, 140.42)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 907)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (311, 2339, 4428)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12662)
[M::mem_pestat] mean and std.dev: (2287.62, 1847.23)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 16779)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (79, 159, 254)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 604)
[M::mem_pestat] mean and std.dev: (181.41, 125.95)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 779)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100970 reads in 2.257 CPU sec, 2.601 real sec
[M::process] read 100704 sequences (10000055 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (115, 48279, 17, 106)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (104, 182, 288)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 656)
[M::mem_pestat] mean and std.dev: (208.17, 141.14)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 840)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 214, 331)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 729)
[M::mem_pestat] mean and std.dev: (243.11, 142.74)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 928)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (317, 823, 4561)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 13049)
[M::mem_pestat] mean and std.dev: (1969.76, 2167.39)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 17293)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (86, 175, 264)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 620)
[M::mem_pestat] mean and std.dev: (198.42, 139.69)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 798)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100704 reads in 2.168 CPU sec, 2.618 real sec
[M::process] read 100678 sequences (10000123 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (112, 48319, 7, 80)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (104, 224, 387)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 953)
[M::mem_pestat] mean and std.dev: (254.62, 168.26)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1236)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 212, 331)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 729)
[M::mem_pestat] mean and std.dev: (243.62, 144.65)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 928)
[M::mem_pestat] skip orientation RF as there are not enough pairs
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (67, 176, 265)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 661)
[M::mem_pestat] mean and std.dev: (182.92, 137.75)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 859)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100678 reads in 2.252 CPU sec, 2.635 real sec
[M::process] read 100756 sequences (10000058 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (100, 48412, 15, 93)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (88, 161, 311)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 757)
[M::mem_pestat] mean and std.dev: (196.65, 126.53)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 980)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 213, 333)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 735)
[M::mem_pestat] mean and std.dev: (244.73, 145.93)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 936)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (397, 778, 2409)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6433)
[M::mem_pestat] mean and std.dev: (1371.13, 1236.74)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8445)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (84, 156, 249)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 579)
[M::mem_pestat] mean and std.dev: (176.47, 118.76)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 744)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100756 reads in 2.253 CPU sec, 2.618 real sec
[M::process] read 100868 sequences (10000037 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (104, 48380, 15, 108)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (127, 207, 296)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 634)
[M::mem_pestat] mean and std.dev: (212.71, 121.13)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 803)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (136, 219, 341)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 751)
[M::mem_pestat] mean and std.dev: (250.99, 148.31)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 956)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (584, 1622, 4591)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12605)
[M::mem_pestat] mean and std.dev: (2212.13, 1871.61)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 16612)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (95, 174, 286)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 668)
[M::mem_pestat] mean and std.dev: (191.00, 132.88)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 859)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100868 reads in 2.389 CPU sec, 2.759 real sec
[M::process] read 100632 sequences (10000021 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (97, 48276, 12, 102)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (108, 180, 314)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 726)
[M::mem_pestat] mean and std.dev: (215.35, 149.92)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 932)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (135, 218, 341)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 753)
[M::mem_pestat] mean and std.dev: (250.43, 149.03)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 959)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (292, 1526, 3872)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 11032)
[M::mem_pestat] mean and std.dev: (1772.42, 1631.46)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 14612)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (90, 163, 260)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 600)
[M::mem_pestat] mean and std.dev: (180.51, 114.38)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 770)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100632 reads in 2.250 CPU sec, 2.655 real sec
[M::process] read 100848 sequences (10000128 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (95, 48356, 11, 102)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (110, 156, 275)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 605)
[M::mem_pestat] mean and std.dev: (190.13, 123.02)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 770)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (134, 217, 340)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 752)
[M::mem_pestat] mean and std.dev: (249.35, 148.53)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (207, 412, 1396)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 3774)
[M::mem_pestat] mean and std.dev: (413.00, 409.00)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 4963)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (83, 151, 281)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 677)
[M::mem_pestat] mean and std.dev: (177.94, 129.96)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 875)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100848 reads in 2.184 CPU sec, 2.610 real sec
[M::process] read 100822 sequences (10000138 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (103, 48417, 13, 103)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (94, 160, 319)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 769)
[M::mem_pestat] mean and std.dev: (214.17, 163.21)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 994)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (130, 211, 329)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 727)
[M::mem_pestat] mean and std.dev: (240.89, 142.28)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 926)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (328, 874, 4581)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 13087)
[M::mem_pestat] mean and std.dev: (1947.23, 1811.60)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 17340)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (91, 150, 265)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 613)
[M::mem_pestat] mean and std.dev: (175.87, 118.47)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 787)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100822 reads in 2.272 CPU sec, 2.672 real sec
[M::process] read 100864 sequences (10000003 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (125, 48435, 19, 106)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (101, 184, 316)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 746)
[M::mem_pestat] mean and std.dev: (214.39, 144.34)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 961)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 214, 332)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 732)
[M::mem_pestat] mean and std.dev: (244.89, 145.25)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 932)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (433, 2397, 4553)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12793)
[M::mem_pestat] mean and std.dev: (2707.21, 1935.45)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 16913)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (93, 172, 291)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 687)
[M::mem_pestat] mean and std.dev: (203.49, 143.48)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 885)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100864 reads in 2.399 CPU sec, 2.785 real sec
[M::process] read 101310 sequences (10000033 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (101, 48567, 14, 116)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (100, 154, 267)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 601)
[M::mem_pestat] mean and std.dev: (190.75, 126.13)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 768)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (128, 204, 317)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 695)
[M::mem_pestat] mean and std.dev: (234.06, 136.79)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 884)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (346, 1825, 2670)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 7318)
[M::mem_pestat] mean and std.dev: (1590.77, 1288.16)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 9642)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (80, 164, 254)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 602)
[M::mem_pestat] mean and std.dev: (174.68, 114.30)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 776)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101310 reads in 2.337 CPU sec, 2.709 real sec
[M::process] read 101086 sequences (10000182 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (110, 48496, 10, 91)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (92, 166, 277)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 647)
[M::mem_pestat] mean and std.dev: (191.88, 125.73)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 832)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 212, 328)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 720)
[M::mem_pestat] mean and std.dev: (241.18, 140.67)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 916)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (308, 1138, 4518)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12938)
[M::mem_pestat] mean and std.dev: (1917.00, 1969.78)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 17148)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (92, 177, 300)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 716)
[M::mem_pestat] mean and std.dev: (203.98, 136.60)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 924)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101086 reads in 2.382 CPU sec, 2.757 real sec
[M::process] read 100884 sequences (10000063 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (120, 48386, 11, 83)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (122, 207, 341)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 779)
[M::mem_pestat] mean and std.dev: (229.29, 147.56)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 998)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 214, 331)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 729)
[M::mem_pestat] mean and std.dev: (243.98, 143.60)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 928)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (41, 270, 1418)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 4172)
[M::mem_pestat] mean and std.dev: (319.89, 424.93)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 5549)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (113, 211, 346)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 812)
[M::mem_pestat] mean and std.dev: (244.44, 174.37)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1045)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100884 reads in 2.545 CPU sec, 2.982 real sec
[M::process] read 100742 sequences (10000087 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (98, 48343, 15, 95)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (116, 206, 302)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 674)
[M::mem_pestat] mean and std.dev: (210.18, 126.26)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 860)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 215, 334)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 738)
[M::mem_pestat] mean and std.dev: (245.16, 145.00)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 940)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (186, 407, 3150)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 9078)
[M::mem_pestat] mean and std.dev: (1423.07, 1760.04)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 12042)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (103, 169, 277)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 625)
[M::mem_pestat] mean and std.dev: (185.25, 114.95)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 799)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100742 reads in 2.457 CPU sec, 2.809 real sec
[M::process] read 100722 sequences (10000160 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (100, 48365, 11, 92)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (110, 157, 331)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 773)
[M::mem_pestat] mean and std.dev: (221.82, 169.52)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 994)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 214, 334)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 738)
[M::mem_pestat] mean and std.dev: (245.65, 145.53)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 940)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (415, 776, 2815)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 7615)
[M::mem_pestat] mean and std.dev: (1758.91, 1844.24)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 10015)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (94, 176, 319)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 769)
[M::mem_pestat] mean and std.dev: (205.39, 144.38)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 994)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100722 reads in 2.199 CPU sec, 2.641 real sec
[M::process] read 100802 sequences (10000037 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (112, 48399, 19, 78)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (101, 157, 347)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 839)
[M::mem_pestat] mean and std.dev: (219.84, 154.76)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1085)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (133, 213, 332)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 730)
[M::mem_pestat] mean and std.dev: (244.10, 144.60)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 929)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (110, 337, 1256)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 3548)
[M::mem_pestat] mean and std.dev: (619.83, 683.55)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 4694)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (101, 157, 283)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 647)
[M::mem_pestat] mean and std.dev: (196.68, 137.27)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 829)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100802 reads in 2.265 CPU sec, 2.652 real sec
[M::process] read 100990 sequences (10000192 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (117, 48386, 17, 97)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (92, 177, 325)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 791)
[M::mem_pestat] mean and std.dev: (207.97, 143.82)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1024)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 212, 332)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 734)
[M::mem_pestat] mean and std.dev: (243.39, 144.57)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 935)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (264, 426, 2069)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 5679)
[M::mem_pestat] mean and std.dev: (1235.53, 1671.21)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 7484)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (114, 201, 271)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 585)
[M::mem_pestat] mean and std.dev: (199.55, 119.64)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 742)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100990 reads in 2.250 CPU sec, 2.633 real sec
[M::process] read 100964 sequences (10000103 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (109, 48470, 15, 96)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (94, 224, 369)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 919)
[M::mem_pestat] mean and std.dev: (247.83, 173.68)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1194)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (133, 214, 335)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 739)
[M::mem_pestat] mean and std.dev: (245.14, 145.35)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 941)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (168, 660, 2416)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6912)
[M::mem_pestat] mean and std.dev: (1257.40, 1505.78)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 9160)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (106, 171, 321)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 751)
[M::mem_pestat] mean and std.dev: (211.63, 147.54)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 966)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100964 reads in 2.237 CPU sec, 2.633 real sec
[M::process] read 101572 sequences (10000003 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (101, 48757, 15, 91)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (114, 183, 286)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 630)
[M::mem_pestat] mean and std.dev: (199.98, 120.18)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 802)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (130, 208, 323)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 709)
[M::mem_pestat] mean and std.dev: (237.43, 139.85)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 902)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (409, 1006, 1490)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 3652)
[M::mem_pestat] mean and std.dev: (719.67, 490.49)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 4733)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (99, 186, 270)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 612)
[M::mem_pestat] mean and std.dev: (193.88, 126.88)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 783)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101572 reads in 2.228 CPU sec, 2.625 real sec
[M::process] read 100914 sequences (10000093 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (88, 48463, 14, 104)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (115, 185, 327)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 751)
[M::mem_pestat] mean and std.dev: (218.95, 146.68)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 963)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (133, 214, 333)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 733)
[M::mem_pestat] mean and std.dev: (244.78, 144.84)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 933)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (369, 1667, 4499)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12759)
[M::mem_pestat] mean and std.dev: (2174.86, 1858.10)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 16889)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (111, 176, 291)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 651)
[M::mem_pestat] mean and std.dev: (204.08, 121.35)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 831)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100914 reads in 2.222 CPU sec, 2.619 real sec
[M::process] read 100992 sequences (10000157 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (84, 48526, 8, 94)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (96, 148, 261)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 591)
[M::mem_pestat] mean and std.dev: (194.35, 137.94)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 756)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 212, 331)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 729)
[M::mem_pestat] mean and std.dev: (242.89, 143.98)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 928)
[M::mem_pestat] skip orientation RF as there are not enough pairs
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (92, 136, 267)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 617)
[M::mem_pestat] mean and std.dev: (174.04, 125.75)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 792)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100992 reads in 2.250 CPU sec, 2.640 real sec
[M::process] read 100764 sequences (10000073 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (111, 48350, 17, 118)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (91, 183, 273)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 637)
[M::mem_pestat] mean and std.dev: (198.79, 133.35)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 819)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 215, 334)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 738)
[M::mem_pestat] mean and std.dev: (245.80, 146.75)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 940)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (226, 1685, 4725)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 13723)
[M::mem_pestat] mean and std.dev: (2065.88, 1974.74)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 18222)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (75, 147, 269)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 657)
[M::mem_pestat] mean and std.dev: (177.18, 124.90)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 851)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100764 reads in 2.232 CPU sec, 2.610 real sec
[M::process] read 100824 sequences (10000195 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (119, 48351, 14, 74)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (107, 178, 319)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 743)
[M::mem_pestat] mean and std.dev: (205.71, 135.11)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 955)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (134, 216, 336)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 740)
[M::mem_pestat] mean and std.dev: (247.01, 146.79)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 942)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (146, 566, 909)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 2435)
[M::mem_pestat] mean and std.dev: (356.45, 256.83)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 3198)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (109, 164, 301)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 685)
[M::mem_pestat] mean and std.dev: (202.01, 127.32)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 877)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100824 reads in 2.253 CPU sec, 2.639 real sec
[M::process] read 100906 sequences (10000177 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (101, 48479, 10, 95)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (103, 183, 301)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 697)
[M::mem_pestat] mean and std.dev: (202.34, 126.78)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 895)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 214, 336)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 746)
[M::mem_pestat] mean and std.dev: (245.83, 147.44)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 951)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (58, 436, 1604)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 4696)
[M::mem_pestat] mean and std.dev: (923.40, 947.56)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 6242)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (79, 185, 285)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 697)
[M::mem_pestat] mean and std.dev: (214.02, 151.83)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 903)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100906 reads in 2.223 CPU sec, 2.666 real sec
[M::process] read 100946 sequences (10000029 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (105, 48420, 12, 88)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (95, 156, 291)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 683)
[M::mem_pestat] mean and std.dev: (196.45, 136.41)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 879)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 211, 329)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 725)
[M::mem_pestat] mean and std.dev: (242.40, 144.89)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 923)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (404, 1066, 4336)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 12200)
[M::mem_pestat] mean and std.dev: (1649.75, 1729.58)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 16132)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (92, 199, 333)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 815)
[M::mem_pestat] mean and std.dev: (225.51, 159.22)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1056)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100946 reads in 2.222 CPU sec, 2.655 real sec
[M::process] read 101320 sequences (10000105 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (112, 48611, 16, 107)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (104, 189, 303)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 701)
[M::mem_pestat] mean and std.dev: (202.04, 131.09)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 900)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 212, 332)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 732)
[M::mem_pestat] mean and std.dev: (243.56, 144.31)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 932)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (176, 1994, 3335)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 9653)
[M::mem_pestat] mean and std.dev: (1870.25, 1583.99)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 12812)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (97, 203, 293)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 685)
[M::mem_pestat] mean and std.dev: (209.72, 140.40)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 881)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101320 reads in 2.200 CPU sec, 2.624 real sec
[M::process] read 101122 sequences (10000143 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (127, 48486, 17, 92)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (111, 192, 283)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 627)
[M::mem_pestat] mean and std.dev: (204.18, 122.21)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 799)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 211, 329)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 725)
[M::mem_pestat] mean and std.dev: (242.25, 143.71)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 923)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (389, 1441, 2857)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 7793)
[M::mem_pestat] mean and std.dev: (1488.00, 1803.05)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 10261)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (114, 180, 283)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 621)
[M::mem_pestat] mean and std.dev: (191.80, 112.50)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 790)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101122 reads in 2.265 CPU sec, 2.637 real sec
[M::process] read 100910 sequences (10000151 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (102, 48438, 11, 94)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (90, 163, 275)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 645)
[M::mem_pestat] mean and std.dev: (195.54, 132.49)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 830)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (133, 214, 335)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 739)
[M::mem_pestat] mean and std.dev: (246.25, 147.04)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 941)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (362, 1675, 2456)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6644)
[M::mem_pestat] mean and std.dev: (1804.36, 1593.76)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8738)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (92, 163, 286)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 674)
[M::mem_pestat] mean and std.dev: (191.71, 133.66)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 868)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100910 reads in 2.264 CPU sec, 2.648 real sec
[M::process] read 100826 sequences (10000160 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (117, 48408, 11, 101)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (89, 183, 306)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 740)
[M::mem_pestat] mean and std.dev: (212.83, 159.51)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 957)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 212, 334)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 738)
[M::mem_pestat] mean and std.dev: (245.11, 146.11)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 940)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (883, 2619, 4153)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 10693)
[M::mem_pestat] mean and std.dev: (2339.45, 1645.15)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 13963)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (99, 164, 298)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 696)
[M::mem_pestat] mean and std.dev: (192.25, 131.09)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 895)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100826 reads in 2.220 CPU sec, 2.624 real sec
[M::process] read 100846 sequences (10000132 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (102, 48425, 13, 103)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (120, 214, 302)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 666)
[M::mem_pestat] mean and std.dev: (229.56, 145.28)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 848)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 211, 330)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 728)
[M::mem_pestat] mean and std.dev: (242.44, 144.10)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 927)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (691, 1620, 3017)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 7669)
[M::mem_pestat] mean and std.dev: (1885.31, 1485.03)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 9995)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (120, 204, 345)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 795)
[M::mem_pestat] mean and std.dev: (245.26, 170.36)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1020)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100846 reads in 2.204 CPU sec, 2.606 real sec
[M::process] read 100902 sequences (10000015 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (111, 48482, 15, 90)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (118, 195, 311)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 697)
[M::mem_pestat] mean and std.dev: (226.11, 152.94)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 890)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 211, 328)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 722)
[M::mem_pestat] mean and std.dev: (241.45, 142.86)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 919)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (251, 582, 2798)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 7892)
[M::mem_pestat] mean and std.dev: (1611.47, 1716.82)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 10439)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (89, 164, 279)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 659)
[M::mem_pestat] mean and std.dev: (192.86, 130.81)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 849)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100902 reads in 2.209 CPU sec, 2.638 real sec
[M::process] read 100906 sequences (10000142 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (125, 48384, 17, 101)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (95, 184, 291)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 683)
[M::mem_pestat] mean and std.dev: (216.51, 153.01)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 879)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (133, 213, 332)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 730)
[M::mem_pestat] mean and std.dev: (244.47, 144.19)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 929)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (391, 1273, 3863)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 10807)
[M::mem_pestat] mean and std.dev: (1719.24, 1611.62)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 14279)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (110, 167, 310)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 710)
[M::mem_pestat] mean and std.dev: (211.19, 142.16)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 910)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100906 reads in 2.274 CPU sec, 2.639 real sec
[M::process] read 101178 sequences (10000047 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (97, 48569, 13, 87)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (112, 187, 330)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 766)
[M::mem_pestat] mean and std.dev: (218.03, 149.75)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 984)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (129, 207, 323)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 711)
[M::mem_pestat] mean and std.dev: (237.35, 139.90)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 905)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (220, 923, 3444)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 9892)
[M::mem_pestat] mean and std.dev: (1536.31, 1635.66)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 13116)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (110, 168, 260)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 560)
[M::mem_pestat] mean and std.dev: (187.25, 109.24)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 710)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101178 reads in 2.243 CPU sec, 2.609 real sec
[M::process] read 101390 sequences (10000164 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (112, 48659, 9, 91)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (97, 187, 283)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 655)
[M::mem_pestat] mean and std.dev: (191.44, 118.70)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 841)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (127, 206, 318)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 700)
[M::mem_pestat] mean and std.dev: (234.14, 137.25)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 891)
[M::mem_pestat] skip orientation RF as there are not enough pairs
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (110, 183, 286)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 638)
[M::mem_pestat] mean and std.dev: (211.01, 137.13)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 814)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101390 reads in 2.218 CPU sec, 2.637 real sec
[M::process] read 100834 sequences (10000132 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (134, 48362, 14, 110)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (110, 216, 347)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 821)
[M::mem_pestat] mean and std.dev: (232.10, 157.79)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1058)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 212, 331)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 729)
[M::mem_pestat] mean and std.dev: (242.63, 143.38)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 928)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (270, 444, 1418)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 3714)
[M::mem_pestat] mean and std.dev: (801.00, 862.83)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 4862)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (97, 193, 308)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 730)
[M::mem_pestat] mean and std.dev: (212.83, 138.54)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 941)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100834 reads in 2.199 CPU sec, 2.623 real sec
[M::process] read 101066 sequences (10000080 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (105, 48438, 12, 102)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (94, 168, 297)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 703)
[M::mem_pestat] mean and std.dev: (200.62, 138.80)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 906)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (130, 210, 327)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 721)
[M::mem_pestat] mean and std.dev: (240.42, 142.12)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 918)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (436, 2090, 2902)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 7834)
[M::mem_pestat] mean and std.dev: (1835.83, 1575.76)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 10300)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (93, 180, 294)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 696)
[M::mem_pestat] mean and std.dev: (204.09, 142.75)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 897)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101066 reads in 2.283 CPU sec, 2.690 real sec
[M::process] read 100880 sequences (10000156 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (115, 48354, 17, 117)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (107, 184, 300)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 686)
[M::mem_pestat] mean and std.dev: (206.70, 127.86)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 879)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (132, 213, 334)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 738)
[M::mem_pestat] mean and std.dev: (244.57, 146.08)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 940)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (61, 413, 2357)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6949)
[M::mem_pestat] mean and std.dev: (947.12, 1000.11)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 9245)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (94, 157, 260)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 592)
[M::mem_pestat] mean and std.dev: (192.29, 128.82)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 758)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100880 reads in 2.351 CPU sec, 2.751 real sec
[M::process] read 100930 sequences (10000020 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (103, 48335, 10, 102)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (102, 206, 316)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 744)
[M::mem_pestat] mean and std.dev: (221.54, 147.95)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 958)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 211, 328)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 722)
[M::mem_pestat] mean and std.dev: (241.16, 142.56)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 919)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (134, 1110, 2359)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6809)
[M::mem_pestat] mean and std.dev: (1651.50, 1975.45)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 9034)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (83, 163, 277)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 665)
[M::mem_pestat] mean and std.dev: (193.32, 139.64)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 859)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100930 reads in 2.337 CPU sec, 2.719 real sec
[M::process] read 100918 sequences (10000126 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (108, 48426, 21, 105)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (102, 174, 304)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 708)
[M::mem_pestat] mean and std.dev: (203.66, 149.30)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 910)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (131, 214, 334)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 740)
[M::mem_pestat] mean and std.dev: (245.44, 146.77)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 943)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (196, 610, 2362)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 6694)
[M::mem_pestat] mean and std.dev: (1298.24, 1493.71)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 8860)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (121, 201, 270)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 568)
[M::mem_pestat] mean and std.dev: (204.62, 116.96)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 717)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 100918 reads in 2.289 CPU sec, 2.667 real sec
[M::process] read 101730 sequences (10000048 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (104, 48792, 13, 89)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (93, 173, 273)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 633)
[M::mem_pestat] mean and std.dev: (196.57, 137.92)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 813)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (129, 208, 323)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 711)
[M::mem_pestat] mean and std.dev: (237.55, 139.95)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 905)
[M::mem_pestat] analyzing insert size distribution for orientation RF...
[M::mem_pestat] (25, 50, 75) percentile: (402, 1626, 4656)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 13164)
[M::mem_pestat] mean and std.dev: (2608.15, 2303.35)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 17418)
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (92, 166, 319)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 773)
[M::mem_pestat] mean and std.dev: (203.99, 144.38)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 1000)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 101730 reads in 2.573 CPU sec, 3.022 real sec
[M::process] read 66124 sequences (6228652 bp)...
[M::mem_pestat] # candidate unique pairs for (FF, FR, RF, RR): (84, 31497, 7, 63)
[M::mem_pestat] analyzing insert size distribution for orientation FF...
[M::mem_pestat] (25, 50, 75) percentile: (104, 153, 286)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 650)
[M::mem_pestat] mean and std.dev: (190.35, 137.35)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 832)
[M::mem_pestat] analyzing insert size distribution for orientation FR...
[M::mem_pestat] (25, 50, 75) percentile: (106, 172, 286)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 646)
[M::mem_pestat] mean and std.dev: (206.79, 132.13)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 826)
[M::mem_pestat] skip orientation RF as there are not enough pairs
[M::mem_pestat] analyzing insert size distribution for orientation RR...
[M::mem_pestat] (25, 50, 75) percentile: (71, 119, 228)
[M::mem_pestat] low and high boundaries for computing mean and std.dev: (1, 542)
[M::mem_pestat] mean and std.dev: (156.38, 113.09)
[M::mem_pestat] low and high boundaries for proper pairs: (1, 699)
[M::mem_pestat] skip orientation FF
[M::mem_pestat] skip orientation RR
[M::mem_process_seqs] Processed 66124 reads in 1.908 CPU sec, 2.195 real sec
[main] Version: 0.7.12-r1039
[main] CMD: bwa mem -R @RG\tID:G04878\tSM:G04878\tPL:Illumina -M NC000962_3.fasta G04878_1_trimmed_paired.fastq G04878_2_trimmed_paired.fastq
[main] Real time: 255.760 sec; CPU: 230.183 sec


>>>>> samtools_faidx_reference_genome <<<<<


samtools faidx NC000962_3.fasta




>>>>> convert_sam_file_to_bam_file <<<<<


samtools view -bt NC000962_3.fasta.fai G04878.sam > G04878.bam




>>>>> sort_bam_file <<<<<


samtools sort G04878.bam -o G04878.sorted.bam


[bam_sort_core] merging from 4 files...


>>>>> samtools_index_sorted_bam <<<<<


samtools index G04878.sorted.bam




>>>>> mapping_statistics <<<<<


samtools flagstat G04878.sorted.bam > G04878_stats.txt




>>>>> samtools_mpileup <<<<<


samtools mpileup -Q 23 -d 2000 -C 50 -ugf NC000962_3.fasta G04878.sorted.bam | bcftools call -O v -vm -o G04878.raw.vcf


[mpileup] 1 samples in 1 input files
Note: none of --samples-file, --ploidy or --ploidy-file given, assuming all sites are diploid
<mpileup> Set max per-file depth to 8000


>>>>> vcfutils_filter<<<<<


vcfutils.pl varFilter -d 10 -D 2000 G04878.raw.vcf > G04878.filt.vcf




>>>>> bgzip_filt_file <<<<<


bgzip -c G04878.filt.vcf > G04878.filt.vcf.gz




>>>>> run_tabix <<<<<


tabix -p vcf G04878.filt.vcf.gz




>>>>> snpEff <<<<<


java -Xmx4g -jar /opt/snpEff/snpEff.jar -no-downstream -no-upstream -v -c /opt/snpEff/snpEff.config NC000962_3 G04878.filt.vcf > G04878.ann.vcf.gz


00:00:00	SnpEff version SnpEff 4.3p (build 2017-06-06 09:55), by Pablo Cingolani
00:00:00	Command: 'ann'
00:00:00	Reading configuration file '/opt/snpEff/snpEff.config'. Genome: 'NC000962_3'
00:00:00	Reading config file: /opt/snpEff/snpEff.config
00:00:00	done
00:00:00	Reading database for genome version 'NC000962_3' from file '/opt/snpEff/./data/NC000962_3/snpEffectPredictor.bin' (this might take a while)
00:00:01	done
00:00:01	Loading Motifs and PWMs
00:00:01	Building interval forest
00:00:01	done.
00:00:01	Genome stats :
#-----------------------------------------------
# Genome name                : 'Mycobacterium tuberculosis'
# Genome version             : 'NC000962_3'
# Genome ID                  : 'NC000962_3[0]'
# Has protein coding info    : true
# Has Tr. Support Level info : true
# Genes                      : 4111
# Protein coding genes       : 4018
#-----------------------------------------------
# Transcripts                : 4018
# Avg. transcripts per gene  : 0.98
# TSL transcripts            : 0
#-----------------------------------------------
# Checked transcripts        : 
#               AA sequences :   4018 ( 100.00% )
#              DNA sequences :      0 ( 0.00% )
#-----------------------------------------------
# Protein coding transcripts : 4018
#              Length errors :      0 ( 0.00% )
#  STOP codons in CDS errors :      0 ( 0.00% )
#         START codon errors :     27 ( 0.67% )
#        STOP codon warnings :      0 ( 0.00% )
#              UTR sequences :      0 ( 0.00% )
#               Total Errors :     27 ( 0.67% )
# WARNING                    : No protein coding transcript has UTR
#-----------------------------------------------
# Cds                        : 4018
# Exons                      : 4018
# Exons with sequence        : 4018
# Exons without sequence     : 0
# Avg. exons per transcript  : 1.00
#-----------------------------------------------
# Number of chromosomes      : 1
# Chromosomes                : Format 'chromo_name size codon_table'
#		'NC000962_3'	4411532	Bacterial_and_Plant_Plastid
#-----------------------------------------------

00:00:02	Predicting variants
00:00:02	Creating sequences from exon information 'NC000962_3'
00:00:03	Building sequence tree for chromosome 'NC000962_3'
00:00:03	Building sequence tree for genome sequences
00:00:03	Done.
00:00:03	Done. Loaded 3228 sequences.

WARNINGS: Some warning were detected
Warning type	Number of warnings
WARNING_TRANSCRIPT_NO_START_CODON	180


00:00:07	Creating summary file: snpEff_summary.html
00:00:08	Creating genes file: snpEff_genes.txt
00:00:09	done.
00:00:09	Logging
00:00:12	Checking for updates...


NEW VERSION!
	There is a new SnpEff version available: 
		Version      : 4.3T
		Release date : 2017-11-24
		Download URL : http://sourceforge.net/projects/snpeff/files/snpEff_latest_core.zip

00:00:13	Done.


>>>>> velveth_assembly <<<<<


velveth G04878_41 41 -fastq -shortPaired  G04878_1_trimmed_paired.fastq G04878_1_trimmed_unpaired.fastq -fastq -short G04878_2_trimmed_paired.fastq G04878_2_trimmed_unpaired.fastq


[0.000000] Reading FastQ file G04878_1_trimmed_paired.fastq;
[17.789715] 4166621 sequences found
[17.789724] Done
[17.789945] Reading FastQ file G04878_1_trimmed_unpaired.fastq;
[19.300599] 433770 sequences found
[19.300607] Done
[19.300790] Reading FastQ file G04878_2_trimmed_paired.fastq;
[38.595136] 4166621 sequences found
[38.595145] Done
[38.595313] Reading FastQ file G04878_2_trimmed_unpaired.fastq;
[38.829938] 68389 sequences found
[38.829946] Done
[38.830203] Reading read set file G04878_41/Sequences;
[50.615632] 8835401 sequences found
[77.412202] Done
[77.412816] 8835401 sequences in total.
[77.413619] Writing into roadmap file G04878_41/Roadmaps...
[85.395207] Inputting sequences...
[85.398628] Inputting sequence 0 / 8835401
[97.758949] Inputting sequence 1000000 / 8835401
[110.396919] Inputting sequence 2000000 / 8835401
[123.465467] Inputting sequence 3000000 / 8835401
[136.472052] Inputting sequence 4000000 / 8835401
[148.793008] Inputting sequence 5000000 / 8835401
[161.754577] Inputting sequence 6000000 / 8835401
[174.941463] Inputting sequence 7000000 / 8835401
[196.965053] Inputting sequence 8000000 / 8835401
[210.344585]  === Sequences loaded in 124.979025 s
[210.346371] Done inputting sequences
[210.346376] Destroying splay table
[210.430498] Splay table destroyed


>>>>> velvetg_produce_graph <<<<<


velvetg G04878_41 -exp_cov auto -cov_cutoff auto


[0.000000] Reading roadmap file G04878_41/Roadmaps
[18.473266] 8835401 roadmaps read
[18.473723] Creating insertion markers
[20.576964] Ordering insertion markers
[27.466618] Counting preNodes
[28.373214] 5084115 preNodes counted, creating them now
[33.203200] Sequence 1000000 / 8835401
[36.224835] Sequence 2000000 / 8835401
[39.192080] Sequence 3000000 / 8835401
[42.179765] Sequence 4000000 / 8835401
[45.059274] Sequence 5000000 / 8835401
[47.973431] Sequence 6000000 / 8835401
[50.826395] Sequence 7000000 / 8835401
[53.686105] Sequence 8000000 / 8835401
[56.035573] Adjusting marker info...
[57.342857] Connecting preNodes
[58.949668] Connecting 1000000 / 8835401
[60.685863] Connecting 2000000 / 8835401
[62.571926] Connecting 3000000 / 8835401
[64.512876] Connecting 4000000 / 8835401
[66.343961] Connecting 5000000 / 8835401
[68.302071] Connecting 6000000 / 8835401
[70.326259] Connecting 7000000 / 8835401
[72.411692] Connecting 8000000 / 8835401
[74.155685] Cleaning up memory
[74.160360] Done creating preGraph
[74.160360] Concatenation...
[75.362280] Renumbering preNodes
[75.362294] Initial preNode count 5084115
[75.538729] Destroyed 3232233 preNodes
[75.538800] Concatenation over!
[75.538822] Clipping short tips off preGraph
[75.764929] Concatenation...
[76.278812] Renumbering preNodes
[76.278841] Initial preNode count 1851882
[76.403022] Destroyed 654199 preNodes
[76.403084] Concatenation over!
[76.403103] 354227 tips cut off
[76.403118] 1197683 nodes left
[76.403826] Writing into pregraph file G04878_41/PreGraph...
[80.142736] Reading read set file G04878_41/Sequences;
[92.429500] 8835401 sequences found
[119.419780] Done
[126.049698] Reading pre-graph file G04878_41/PreGraph
[126.097900] Graph has 1197683 nodes and 8835401 sequences
[127.856073] Scanning pre-graph file G04878_41/PreGraph for k-mers
[128.934986] 18409704 kmers found
[130.900396] Sorting kmer occurence table ... 
[140.251848] Sorting done.
[140.251934] Computing acceleration table... 
[140.398454] Computing offsets... 
[140.519374] Ghost Threading through reads 0 / 8835401
[160.352307] Ghost Threading through reads 1000000 / 8835401
[180.573602] Ghost Threading through reads 2000000 / 8835401
[201.518958] Ghost Threading through reads 3000000 / 8835401
[221.301828] Ghost Threading through reads 4000000 / 8835401
[239.694040] Ghost Threading through reads 5000000 / 8835401
[258.778494] Ghost Threading through reads 6000000 / 8835401
[277.709635] Ghost Threading through reads 7000000 / 8835401
[296.612262] Ghost Threading through reads 8000000 / 8835401
[312.078673]  === Ghost-Threaded in 171.559299 s
[312.078709] Threading through reads 0 / 8835401
[336.340552] Threading through reads 1000000 / 8835401
[359.716295] Threading through reads 2000000 / 8835401
[383.265101] Threading through reads 3000000 / 8835401
[408.936728] Threading through reads 4000000 / 8835401
[445.265237] Threading through reads 5000000 / 8835401
[473.977464] Threading through reads 6000000 / 8835401
[500.029643] Threading through reads 7000000 / 8835401
[527.951273] Threading through reads 8000000 / 8835401
[546.681875]  === Threaded in 234.602758 s
[553.614838] Correcting graph with cutoff 0.200000
[554.845985] Determining eligible starting points
[560.306122] Done listing starting nodes
[560.308909] Initializing todo lists
[560.584755] Done with initilization
[560.588708] Activating arc lookup table
[561.245654] Done activating arc lookup table
[561.271100] 10000 / 1197683 nodes visited
[561.311885] 20000 / 1197683 nodes visited
[561.342160] 30000 / 1197683 nodes visited
[561.403063] 40000 / 1197683 nodes visited
[561.432794] 50000 / 1197683 nodes visited
[561.475994] 60000 / 1197683 nodes visited
[561.504333] 70000 / 1197683 nodes visited
[561.545292] 80000 / 1197683 nodes visited
[561.570066] 90000 / 1197683 nodes visited
[561.598206] 100000 / 1197683 nodes visited
[561.640887] 110000 / 1197683 nodes visited
[561.682155] 120000 / 1197683 nodes visited
[561.754328] 130000 / 1197683 nodes visited
[561.795618] 140000 / 1197683 nodes visited
[561.834087] 150000 / 1197683 nodes visited
[561.865778] 160000 / 1197683 nodes visited
[561.922825] 170000 / 1197683 nodes visited
[561.947433] 180000 / 1197683 nodes visited
[561.983637] 190000 / 1197683 nodes visited
[562.024927] 200000 / 1197683 nodes visited
[562.062756] 210000 / 1197683 nodes visited
[562.107280] 220000 / 1197683 nodes visited
[562.158670] 230000 / 1197683 nodes visited
[562.205981] 240000 / 1197683 nodes visited
[562.305361] 250000 / 1197683 nodes visited
[562.385522] 260000 / 1197683 nodes visited
[562.456531] 270000 / 1197683 nodes visited
[562.545902] 280000 / 1197683 nodes visited
[562.622847] 290000 / 1197683 nodes visited
[562.696594] 300000 / 1197683 nodes visited
[562.817360] 310000 / 1197683 nodes visited
[562.992104] 320000 / 1197683 nodes visited
[563.125976] 330000 / 1197683 nodes visited
[563.447330] 340000 / 1197683 nodes visited
[563.608203] 350000 / 1197683 nodes visited
[564.434893] 360000 / 1197683 nodes visited
[564.641740] 370000 / 1197683 nodes visited
[564.760550] 380000 / 1197683 nodes visited
[565.014040] 390000 / 1197683 nodes visited
[565.158000] 400000 / 1197683 nodes visited
[565.299535] 410000 / 1197683 nodes visited
[565.441220] 420000 / 1197683 nodes visited
[565.591292] 430000 / 1197683 nodes visited
[565.717763] 440000 / 1197683 nodes visited
[565.831933] 450000 / 1197683 nodes visited
[565.939001] 460000 / 1197683 nodes visited
[566.035632] 470000 / 1197683 nodes visited
[566.131451] 480000 / 1197683 nodes visited
[566.282332] 490000 / 1197683 nodes visited
[566.362624] 500000 / 1197683 nodes visited
[566.528404] 510000 / 1197683 nodes visited
[566.721371] 520000 / 1197683 nodes visited
[567.002215] 530000 / 1197683 nodes visited
[567.522707] 540000 / 1197683 nodes visited
[567.683842] 550000 / 1197683 nodes visited
[567.913225] 560000 / 1197683 nodes visited
[568.218216] 570000 / 1197683 nodes visited
[568.429032] 580000 / 1197683 nodes visited
[568.608247] 590000 / 1197683 nodes visited
[568.833160] 600000 / 1197683 nodes visited
[568.965874] 610000 / 1197683 nodes visited
[569.291768] 620000 / 1197683 nodes visited
[569.468231] 630000 / 1197683 nodes visited
[569.620568] 640000 / 1197683 nodes visited
[569.764629] 650000 / 1197683 nodes visited
[569.920768] 660000 / 1197683 nodes visited
[570.069052] 670000 / 1197683 nodes visited
[570.214356] 680000 / 1197683 nodes visited
[570.343335] 690000 / 1197683 nodes visited
[570.462679] 700000 / 1197683 nodes visited
[570.828225] 710000 / 1197683 nodes visited
[571.265288] 720000 / 1197683 nodes visited
[571.395770] 730000 / 1197683 nodes visited
[571.541125] 740000 / 1197683 nodes visited
[571.667099] 750000 / 1197683 nodes visited
[571.812186] 760000 / 1197683 nodes visited
[571.998327] 770000 / 1197683 nodes visited
[572.134061] 780000 / 1197683 nodes visited
[572.272709] 790000 / 1197683 nodes visited
[572.416741] 800000 / 1197683 nodes visited
[572.586943] 810000 / 1197683 nodes visited
[572.725157] 820000 / 1197683 nodes visited
[572.865313] 830000 / 1197683 nodes visited
[573.000376] 840000 / 1197683 nodes visited
[573.132466] 850000 / 1197683 nodes visited
[573.307548] 860000 / 1197683 nodes visited
[573.490039] 870000 / 1197683 nodes visited
[573.646087] 880000 / 1197683 nodes visited
[573.831166] 890000 / 1197683 nodes visited
[573.996394] 900000 / 1197683 nodes visited
[574.134826] 910000 / 1197683 nodes visited
[574.295596] 920000 / 1197683 nodes visited
[574.470573] 930000 / 1197683 nodes visited
[574.612656] 940000 / 1197683 nodes visited
[574.758226] 950000 / 1197683 nodes visited
[574.906572] 960000 / 1197683 nodes visited
[575.081658] 970000 / 1197683 nodes visited
[575.294368] 980000 / 1197683 nodes visited
[575.487932] 990000 / 1197683 nodes visited
[575.687056] 1000000 / 1197683 nodes visited
[575.903865] 1010000 / 1197683 nodes visited
[576.064058] 1020000 / 1197683 nodes visited
[576.265198] 1030000 / 1197683 nodes visited
[576.488041] 1040000 / 1197683 nodes visited
[576.720063] 1050000 / 1197683 nodes visited
[576.908598] 1060000 / 1197683 nodes visited
[577.215668] 1070000 / 1197683 nodes visited
[577.529964] 1080000 / 1197683 nodes visited
[578.121215] 1090000 / 1197683 nodes visited
[578.408136] 1100000 / 1197683 nodes visited
[578.579765] 1110000 / 1197683 nodes visited
[578.760398] 1120000 / 1197683 nodes visited
[578.913096] 1130000 / 1197683 nodes visited
[579.116583] 1140000 / 1197683 nodes visited
[579.320821] 1150000 / 1197683 nodes visited
[579.543921] 1160000 / 1197683 nodes visited
[579.711995] 1170000 / 1197683 nodes visited
[579.945484] 1180000 / 1197683 nodes visited
[580.131325] 1190000 / 1197683 nodes visited
[580.343161] 1200000 / 1197683 nodes visited
[580.628804] 1210000 / 1197683 nodes visited
[580.822704] 1220000 / 1197683 nodes visited
[580.996748] 1230000 / 1197683 nodes visited
[581.163210] 1240000 / 1197683 nodes visited
[581.341314] 1250000 / 1197683 nodes visited
[581.510241] 1260000 / 1197683 nodes visited
[581.706221] 1270000 / 1197683 nodes visited
[581.900326] 1280000 / 1197683 nodes visited
[582.091116] 1290000 / 1197683 nodes visited
[582.293185] 1300000 / 1197683 nodes visited
[582.504437] 1310000 / 1197683 nodes visited
[582.694210] 1320000 / 1197683 nodes visited
[582.932530] 1330000 / 1197683 nodes visited
[583.167081] 1340000 / 1197683 nodes visited
[583.365808] 1350000 / 1197683 nodes visited
[583.590963] 1360000 / 1197683 nodes visited
[583.799033] 1370000 / 1197683 nodes visited
[584.006580] 1380000 / 1197683 nodes visited
[584.232222] 1390000 / 1197683 nodes visited
[584.461972] 1400000 / 1197683 nodes visited
[584.677866] 1410000 / 1197683 nodes visited
[585.007024] 1420000 / 1197683 nodes visited
[585.294579] 1430000 / 1197683 nodes visited
[585.533675] 1440000 / 1197683 nodes visited
[585.772352] 1450000 / 1197683 nodes visited
[586.011791] 1460000 / 1197683 nodes visited
[586.275381] 1470000 / 1197683 nodes visited
[586.526751] 1480000 / 1197683 nodes visited
[586.784262] 1490000 / 1197683 nodes visited
[587.066073] 1500000 / 1197683 nodes visited
[587.325022] 1510000 / 1197683 nodes visited
[587.589755] 1520000 / 1197683 nodes visited
[587.902677] 1530000 / 1197683 nodes visited
[588.215373] 1540000 / 1197683 nodes visited
[588.566205] 1550000 / 1197683 nodes visited
[588.958308] 1560000 / 1197683 nodes visited
[589.542089] 1570000 / 1197683 nodes visited
[589.778973] 1580000 / 1197683 nodes visited
[590.137190] 1590000 / 1197683 nodes visited
[590.444292] 1600000 / 1197683 nodes visited
[590.741697] 1610000 / 1197683 nodes visited
[591.052875] 1620000 / 1197683 nodes visited
[591.350396] 1630000 / 1197683 nodes visited
[591.684360] 1640000 / 1197683 nodes visited
[591.982947] 1650000 / 1197683 nodes visited
[592.302234] 1660000 / 1197683 nodes visited
[592.599040] 1670000 / 1197683 nodes visited
[592.895620] 1680000 / 1197683 nodes visited
[593.195113] 1690000 / 1197683 nodes visited
[593.513692] 1700000 / 1197683 nodes visited
[593.836049] 1710000 / 1197683 nodes visited
[594.163206] 1720000 / 1197683 nodes visited
[594.483038] 1730000 / 1197683 nodes visited
[594.797023] 1740000 / 1197683 nodes visited
[595.112750] 1750000 / 1197683 nodes visited
[595.474474] 1760000 / 1197683 nodes visited
[595.802303] 1770000 / 1197683 nodes visited
[596.129665] 1780000 / 1197683 nodes visited
[596.459173] 1790000 / 1197683 nodes visited
[596.801080] 1800000 / 1197683 nodes visited
[597.136251] 1810000 / 1197683 nodes visited
[597.481992] 1820000 / 1197683 nodes visited
[597.844302] 1830000 / 1197683 nodes visited
[598.191108] 1840000 / 1197683 nodes visited
[598.553704] 1850000 / 1197683 nodes visited
[598.938692] 1860000 / 1197683 nodes visited
[599.303709] 1870000 / 1197683 nodes visited
[599.662441] 1880000 / 1197683 nodes visited
[600.035428] 1890000 / 1197683 nodes visited
[600.399385] 1900000 / 1197683 nodes visited
[600.734178] 1910000 / 1197683 nodes visited
[601.494186] 1920000 / 1197683 nodes visited
[601.531506] 1930000 / 1197683 nodes visited
[601.598216] 1940000 / 1197683 nodes visited
[601.621952] 1950000 / 1197683 nodes visited
[601.652985] 1960000 / 1197683 nodes visited
[601.688516] 1970000 / 1197683 nodes visited
[601.709955] 1980000 / 1197683 nodes visited
[601.749124] 1990000 / 1197683 nodes visited
[601.786858] 2000000 / 1197683 nodes visited
[601.818562] 2010000 / 1197683 nodes visited
[601.839667] 2020000 / 1197683 nodes visited
[601.870295] 2030000 / 1197683 nodes visited
[601.878216] Concatenation...
[602.037922] Renumbering nodes
[602.037948] Initial node count 1197683
[602.068101] Removed 126819 null nodes
[602.068111] Concatenation over!
[602.068112] Clipping short tips off graph, drastic
[605.079767] Concatenation...
[607.180832] Renumbering nodes
[607.180844] Initial node count 1070864
[607.187779] Removed 1006790 null nodes
[607.187813] Concatenation over!
[607.187835] 64074 nodes left
[607.188908] Writing into graph file G04878_41/Graph2...
[610.917544] Measuring median coverage depth...
[610.942104] Median coverage depth = 103.662539
[610.942224] Removing contigs with coverage < 51.831269...
[610.995158] Concatenation...
[614.375406] Renumbering nodes
[614.375419] Initial node count 64074
[614.375602] Removed 62803 null nodes
[614.375605] Concatenation over!
[614.375642] Concatenation...
[614.375741] Renumbering nodes
[614.375743] Initial node count 1271
[614.375745] Removed 0 null nodes
[614.375746] Concatenation over!
[614.375754] Clipping short tips off graph, drastic
[614.375980] Concatenation...
[614.376463] Renumbering nodes
[614.376467] Initial node count 1271
[614.376486] Removed 98 null nodes
[614.376487] Concatenation over!
[614.376488] 1173 nodes left
[614.376491] Read coherency...
[614.376506] Identifying unique nodes
[614.376527] Done, 762 unique nodes counted
[614.376528] Trimming read tips
[614.376576] Renumbering nodes
[614.376578] Initial node count 1173
[614.376580] Removed 0 null nodes
[614.376581] Confronted to 0 multiple hits and 0 null over 0
[614.376582] Read coherency over!
[614.753473] Starting pebble resolution...
[614.830093] Computing read to node mapping array sizes
[615.035523] Computing read to node mappings
[616.162824] Estimating library insert lengths...
[616.233487] Paired-end library 1 has length: 71, sample standard deviation: 137
[616.233531] Done
[616.233593] Computing direct node to node mappings
[617.006454] Scaffolding node 0
[617.776101]  === Nodes Scaffolded in 1.542500 s
[617.950094] Preparing to correct graph with cutoff 0.200000
[617.952478] Cleaning memory
[617.952484] Deactivating local correction settings
[617.952506] Pebble done.
[617.952508] Starting pebble resolution...
[617.986583] Computing read to node mapping array sizes
[618.195509] Computing read to node mappings
[619.298129] Estimating library insert lengths...
[619.368800] Paired-end library 1 has length: 71, sample standard deviation: 137
[619.368816] Done
[619.368856] Computing direct node to node mappings
[620.148696] Scaffolding node 0
[620.918744]  === Nodes Scaffolded in 1.549886 s
[621.080012] Preparing to correct graph with cutoff 0.200000
[621.082235] Cleaning memory
[621.082240] Deactivating local correction settings
[621.082261] Pebble done.
[621.082264] Concatenation...
[621.082308] Renumbering nodes
[621.082310] Initial node count 1173
[621.082313] Removed 0 null nodes
[621.082319] Concatenation over!
[621.082320] Removing reference contigs with coverage < 51.831269...
[621.082351] Concatenation...
[621.082391] Renumbering nodes
[621.082393] Initial node count 1173
[621.082395] Removed 0 null nodes
[621.082400] Concatenation over!
[621.102662] Writing contigs into G04878_41/contigs.fa...
[623.073214] Writing into stats file G04878_41/stats.txt...
[623.205637] Writing into graph file G04878_41/LastGraph...
[625.715040] Estimated Coverage = 103.662539
[625.715052] Estimated Coverage cutoff = 51.831269
Final graph has 1173 nodes and n50 of 13377, max 70279, total 4229861, using 8608657/8835401 reads


>>>>> assemblathon_stats <<<<<


assemblathon_stats.pl ./G04878_41/contigs.fa




>>>>> velveth_assembly <<<<<


velveth G04878_49 49 -fastq -shortPaired  G04878_1_trimmed_paired.fastq G04878_1_trimmed_unpaired.fastq -fastq -short G04878_2_trimmed_paired.fastq G04878_2_trimmed_unpaired.fastq


[0.000000] Reading FastQ file G04878_1_trimmed_paired.fastq;
[18.342733] 4166621 sequences found
[18.342742] Done
[18.342935] Reading FastQ file G04878_1_trimmed_unpaired.fastq;
[19.870217] 433770 sequences found
[19.870226] Done
[19.870506] Reading FastQ file G04878_2_trimmed_paired.fastq;
[35.868008] 4166621 sequences found
[35.868028] Done
[35.967008] Reading FastQ file G04878_2_trimmed_unpaired.fastq;
[39.424718] 68389 sequences found
[39.424727] Done
[39.424958] Reading read set file G04878_49/Sequences;
[51.366245] 8835401 sequences found
[77.908060] Done
[77.908071] 8835401 sequences in total.
[77.908671] Writing into roadmap file G04878_49/Roadmaps...
[84.425744] Inputting sequences...
[84.425757] Inputting sequence 0 / 8835401
[94.997022] Inputting sequence 1000000 / 8835401
[107.638303] Inputting sequence 2000000 / 8835401
[120.906117] Inputting sequence 3000000 / 8835401
[132.925290] Inputting sequence 4000000 / 8835401
[144.405670] Inputting sequence 5000000 / 8835401
[156.659770] Inputting sequence 6000000 / 8835401
[169.103641] Inputting sequence 7000000 / 8835401
[199.618089] Inputting sequence 8000000 / 8835401
[219.189429]  === Sequences loaded in 134.791399 s
[219.190132] Done inputting sequences
[219.190510] Destroying splay table
[219.314472] Splay table destroyed


>>>>> velvetg_produce_graph <<<<<


velvetg G04878_49 -exp_cov auto -cov_cutoff auto


[0.000000] Reading roadmap file G04878_49/Roadmaps
[17.928561] 8835401 roadmaps read
[17.928770] Creating insertion markers
[19.873383] Ordering insertion markers
[26.342619] Counting preNodes
[27.170618] 4825778 preNodes counted, creating them now
[32.025895] Sequence 1000000 / 8835401
[35.052564] Sequence 2000000 / 8835401
[38.007505] Sequence 3000000 / 8835401
[40.960583] Sequence 4000000 / 8835401
[43.798338] Sequence 5000000 / 8835401
[46.708814] Sequence 6000000 / 8835401
[49.734380] Sequence 7000000 / 8835401
[52.612207] Sequence 8000000 / 8835401
[54.935402] Adjusting marker info...
[56.118910] Connecting preNodes
[57.531159] Connecting 1000000 / 8835401
[59.031133] Connecting 2000000 / 8835401
[60.584255] Connecting 3000000 / 8835401
[62.192744] Connecting 4000000 / 8835401
[63.687546] Connecting 5000000 / 8835401
[65.285075] Connecting 6000000 / 8835401
[66.913974] Connecting 7000000 / 8835401
[68.543590] Connecting 8000000 / 8835401
[69.875784] Cleaning up memory
[69.876927] Done creating preGraph
[69.876938] Concatenation...
[71.049433] Renumbering preNodes
[71.049585] Initial preNode count 4825778
[71.205715] Destroyed 3395248 preNodes
[71.205730] Concatenation over!
[71.205732] Clipping short tips off preGraph
[71.519345] Concatenation...
[71.874829] Renumbering preNodes
[71.874829] Initial preNode count 1430530
[71.965566] Destroyed 828830 preNodes
[71.965708] Concatenation over!
[71.965754] 449841 tips cut off
[71.965792] 601700 nodes left
[71.967755] Writing into pregraph file G04878_49/PreGraph...
[74.015971] Reading read set file G04878_49/Sequences;
[85.457056] 8835401 sequences found
[111.920786] Done
[118.483682] Reading pre-graph file G04878_49/PreGraph
[118.534412] Graph has 601700 nodes and 8835401 sequences
[119.600867] Scanning pre-graph file G04878_49/PreGraph for k-mers
[120.257671] 13095280 kmers found
[121.502400] Sorting kmer occurence table ... 
[127.632184] Sorting done.
[127.632239] Computing acceleration table... 
[127.749344] Computing offsets... 
[127.860748] Ghost Threading through reads 0 / 8835401
[143.338939] Ghost Threading through reads 1000000 / 8835401
[158.912072] Ghost Threading through reads 2000000 / 8835401
[174.479058] Ghost Threading through reads 3000000 / 8835401
[190.276709] Ghost Threading through reads 4000000 / 8835401
[204.386147] Ghost Threading through reads 5000000 / 8835401
[219.418256] Ghost Threading through reads 6000000 / 8835401
[235.026350] Ghost Threading through reads 7000000 / 8835401
[250.781146] Ghost Threading through reads 8000000 / 8835401
[264.712525]  === Ghost-Threaded in 136.851776 s
[264.712539] Threading through reads 0 / 8835401
[282.452147] Threading through reads 1000000 / 8835401
[299.526183] Threading through reads 2000000 / 8835401
[317.052363] Threading through reads 3000000 / 8835401
[334.243733] Threading through reads 4000000 / 8835401
[350.270098] Threading through reads 5000000 / 8835401
[367.068281] Threading through reads 6000000 / 8835401
[383.530747] Threading through reads 7000000 / 8835401
[399.872213] Threading through reads 8000000 / 8835401
[413.159967]  === Threaded in 148.447428 s
[413.966391] Correcting graph with cutoff 0.200000
[413.982542] Determining eligible starting points
[414.604972] Done listing starting nodes
[414.604994] Initializing todo lists
[414.745238] Done with initilization
[414.745252] Activating arc lookup table
[415.036359] Done activating arc lookup table
[415.058118] 10000 / 601700 nodes visited
[415.093224] 20000 / 601700 nodes visited
[415.129968] 30000 / 601700 nodes visited
[415.155783] 40000 / 601700 nodes visited
[415.193367] 50000 / 601700 nodes visited
[415.225142] 60000 / 601700 nodes visited
[415.265658] 70000 / 601700 nodes visited
[415.303623] 80000 / 601700 nodes visited
[415.324597] 90000 / 601700 nodes visited
[415.357757] 100000 / 601700 nodes visited
[415.391936] 110000 / 601700 nodes visited
[415.422551] 120000 / 601700 nodes visited
[415.445215] 130000 / 601700 nodes visited
[415.489553] 140000 / 601700 nodes visited
[415.533532] 150000 / 601700 nodes visited
[415.587727] 160000 / 601700 nodes visited
[415.673489] 170000 / 601700 nodes visited
[415.779711] 180000 / 601700 nodes visited
[415.866448] 190000 / 601700 nodes visited
[415.964510] 200000 / 601700 nodes visited
[416.053381] 210000 / 601700 nodes visited
[416.139297] 220000 / 601700 nodes visited
[416.219167] 230000 / 601700 nodes visited
[416.337162] 240000 / 601700 nodes visited
[416.441753] 250000 / 601700 nodes visited
[416.540104] 260000 / 601700 nodes visited
[416.665866] 270000 / 601700 nodes visited
[416.782201] 280000 / 601700 nodes visited
[416.898760] 290000 / 601700 nodes visited
[416.980015] 300000 / 601700 nodes visited
[417.068464] 310000 / 601700 nodes visited
[417.158350] 320000 / 601700 nodes visited
[417.239464] 330000 / 601700 nodes visited
[417.327030] 340000 / 601700 nodes visited
[417.416873] 350000 / 601700 nodes visited
[417.514585] 360000 / 601700 nodes visited
[417.609311] 370000 / 601700 nodes visited
[417.696393] 380000 / 601700 nodes visited
[417.783804] 390000 / 601700 nodes visited
[417.874939] 400000 / 601700 nodes visited
[417.978708] 410000 / 601700 nodes visited
[418.089765] 420000 / 601700 nodes visited
[418.237898] 430000 / 601700 nodes visited
[418.363751] 440000 / 601700 nodes visited
[418.544414] 450000 / 601700 nodes visited
[418.741805] 460000 / 601700 nodes visited
[418.976946] 470000 / 601700 nodes visited
[419.235897] 480000 / 601700 nodes visited
[419.536546] 490000 / 601700 nodes visited
[419.714517] 500000 / 601700 nodes visited
[419.878633] 510000 / 601700 nodes visited
[420.073961] 520000 / 601700 nodes visited
[420.225501] 530000 / 601700 nodes visited
[420.376809] 540000 / 601700 nodes visited
[420.505322] 550000 / 601700 nodes visited
[420.633187] 560000 / 601700 nodes visited
[420.787427] 570000 / 601700 nodes visited
[420.912965] 580000 / 601700 nodes visited
[421.045269] 590000 / 601700 nodes visited
[421.182284] 600000 / 601700 nodes visited
[421.342712] 610000 / 601700 nodes visited
[421.485391] 620000 / 601700 nodes visited
[421.652249] 630000 / 601700 nodes visited
[421.805700] 640000 / 601700 nodes visited
[421.957670] 650000 / 601700 nodes visited
[422.113721] 660000 / 601700 nodes visited
[422.279592] 670000 / 601700 nodes visited
[422.457341] 680000 / 601700 nodes visited
[422.638596] 690000 / 601700 nodes visited
[422.820528] 700000 / 601700 nodes visited
[423.014993] 710000 / 601700 nodes visited
[423.242971] 720000 / 601700 nodes visited
[423.452412] 730000 / 601700 nodes visited
[423.665308] 740000 / 601700 nodes visited
[423.882284] 750000 / 601700 nodes visited
[424.110198] 760000 / 601700 nodes visited
[424.344623] 770000 / 601700 nodes visited
[424.586709] 780000 / 601700 nodes visited
[424.828345] 790000 / 601700 nodes visited
[425.102475] 800000 / 601700 nodes visited
[425.436069] 810000 / 601700 nodes visited
[425.704756] 820000 / 601700 nodes visited
[426.104162] 830000 / 601700 nodes visited
[426.419641] 840000 / 601700 nodes visited
[426.706810] 850000 / 601700 nodes visited
[427.007869] 860000 / 601700 nodes visited
[427.308294] 870000 / 601700 nodes visited
[427.629111] 880000 / 601700 nodes visited
[427.947880] 890000 / 601700 nodes visited
[428.273958] 900000 / 601700 nodes visited
[428.599528] 910000 / 601700 nodes visited
[428.929592] 920000 / 601700 nodes visited
[429.274716] 930000 / 601700 nodes visited
[429.618384] 940000 / 601700 nodes visited
[429.960449] 950000 / 601700 nodes visited
[430.288701] 960000 / 601700 nodes visited
[430.593116] 970000 / 601700 nodes visited
[431.003290] 980000 / 601700 nodes visited
[431.021889] 990000 / 601700 nodes visited
[431.043930] 1000000 / 601700 nodes visited
[431.082627] 1010000 / 601700 nodes visited
[431.105028] 1020000 / 601700 nodes visited
[431.132445] Concatenation...
[431.217327] Renumbering nodes
[431.217340] Initial node count 601700
[431.230041] Removed 30474 null nodes
[431.230054] Concatenation over!
[431.230056] Clipping short tips off graph, drastic
[432.422778] Concatenation...
[433.781092] Renumbering nodes
[433.781113] Initial node count 571226
[433.784214] Removed 540619 null nodes
[433.784345] Concatenation over!
[433.784415] 30607 nodes left
[433.785273] Writing into graph file G04878_49/Graph2...
[436.692568] Measuring median coverage depth...
[436.702809] Median coverage depth = 90.955224
[436.703143] Removing contigs with coverage < 45.477612...
[436.725856] Concatenation...
[438.325979] Renumbering nodes
[438.326031] Initial node count 30607
[438.326158] Removed 29691 null nodes
[438.326170] Concatenation over!
[438.326198] Concatenation...
[438.326267] Renumbering nodes
[438.326272] Initial node count 916
[438.326277] Removed 0 null nodes
[438.326281] Concatenation over!
[438.326293] Clipping short tips off graph, drastic
[438.326394] Concatenation...
[438.326586] Renumbering nodes
[438.326588] Initial node count 916
[438.326598] Removed 51 null nodes
[438.326599] Concatenation over!
[438.326599] 865 nodes left
[438.326601] Read coherency...
[438.326609] Identifying unique nodes
[438.326623] Done, 562 unique nodes counted
[438.326625] Trimming read tips
[438.326654] Renumbering nodes
[438.326655] Initial node count 865
[438.326657] Removed 0 null nodes
[438.326658] Confronted to 0 multiple hits and 0 null over 0
[438.326659] Read coherency over!
[438.360345] Starting pebble resolution...
[438.396162] Computing read to node mapping array sizes
[438.592487] Computing read to node mappings
[439.621865] Estimating library insert lengths...
[439.692144] Paired-end library 1 has length: 134, sample standard deviation: 277
[439.692213] Done
[439.692266] Computing direct node to node mappings
[440.418737] Scaffolding node 0
[441.133141]  === Nodes Scaffolded in 1.440854 s
[441.278759] Preparing to correct graph with cutoff 0.200000
[441.281161] Cleaning memory
[441.281171] Deactivating local correction settings
[441.281233] Pebble done.
[441.281249] Starting pebble resolution...
[441.303231] Computing read to node mapping array sizes
[441.498383] Computing read to node mappings
[442.545031] Estimating library insert lengths...
[442.614461] Paired-end library 1 has length: 134, sample standard deviation: 277
[442.614485] Done
[442.614511] Computing direct node to node mappings
[443.343249] Scaffolding node 0
[444.077665]  === Nodes Scaffolded in 1.462828 s
[444.236401] Preparing to correct graph with cutoff 0.200000
[444.238439] Cleaning memory
[444.238445] Deactivating local correction settings
[444.238464] Pebble done.
[444.238467] Concatenation...
[444.238494] Renumbering nodes
[444.238496] Initial node count 865
[444.238498] Removed 0 null nodes
[444.238501] Concatenation over!
[444.238502] Removing reference contigs with coverage < 45.477612...
[444.238560] Concatenation...
[444.238586] Renumbering nodes
[444.238587] Initial node count 865
[444.238589] Removed 0 null nodes
[444.238592] Concatenation over!
[444.257836] Writing contigs into G04878_49/contigs.fa...
[446.286243] Writing into stats file G04878_49/stats.txt...
[446.396580] Writing into graph file G04878_49/LastGraph...
[448.853008] Estimated Coverage = 90.955224
[448.853020] Estimated Coverage cutoff = 45.477612
Final graph has 865 nodes and n50 of 17625, max 82574, total 4248976, using 8445973/8835401 reads


>>>>> assemblathon_stats <<<<<


assemblathon_stats.pl ./G04878_49/contigs.fa




>>>>> velveth_assembly <<<<<


velveth G04878_55 55 -fastq -shortPaired  G04878_1_trimmed_paired.fastq G04878_1_trimmed_unpaired.fastq -fastq -short G04878_2_trimmed_paired.fastq G04878_2_trimmed_unpaired.fastq


[0.000000] Reading FastQ file G04878_1_trimmed_paired.fastq;
[17.922964] 4166621 sequences found
[17.922972] Done
[17.923176] Reading FastQ file G04878_1_trimmed_unpaired.fastq;
[19.455380] 433770 sequences found
[19.455389] Done
[19.455537] Reading FastQ file G04878_2_trimmed_paired.fastq;
[35.330678] 4166621 sequences found
[35.330687] Done
[35.330836] Reading FastQ file G04878_2_trimmed_unpaired.fastq;
[35.559724] 68389 sequences found
[35.559733] Done
[35.559971] Reading read set file G04878_55/Sequences;
[47.427893] 8835401 sequences found
[74.248476] Done
[74.248490] 8835401 sequences in total.
[74.248827] Writing into roadmap file G04878_55/Roadmaps...
[80.780770] Inputting sequences...
[80.780786] Inputting sequence 0 / 8835401
[90.189119] Inputting sequence 1000000 / 8835401
[100.495756] Inputting sequence 2000000 / 8835401
[110.988534] Inputting sequence 3000000 / 8835401
[121.870234] Inputting sequence 4000000 / 8835401
[132.106227] Inputting sequence 5000000 / 8835401
[143.190122] Inputting sequence 6000000 / 8835401
[155.489940] Inputting sequence 7000000 / 8835401
[187.811995] Inputting sequence 8000000 / 8835401
[221.452474]  === Sequences loaded in 140.699368 s
[221.454996] Done inputting sequences
[221.459686] Destroying splay table
[221.651354] Splay table destroyed


>>>>> velvetg_produce_graph <<<<<


velvetg G04878_55 -exp_cov auto -cov_cutoff auto


[0.000000] Reading roadmap file G04878_55/Roadmaps
[17.195022] 8835401 roadmaps read
[17.195349] Creating insertion markers
[19.174345] Ordering insertion markers
[25.276334] Counting preNodes
[26.094930] 4634137 preNodes counted, creating them now
[30.942942] Sequence 1000000 / 8835401
[33.898092] Sequence 2000000 / 8835401
[36.855028] Sequence 3000000 / 8835401
[39.775029] Sequence 4000000 / 8835401
[42.572437] Sequence 5000000 / 8835401
[45.425796] Sequence 6000000 / 8835401
[48.289989] Sequence 7000000 / 8835401
[51.108057] Sequence 8000000 / 8835401
[53.425677] Adjusting marker info...
[54.564758] Connecting preNodes
[55.808551] Connecting 1000000 / 8835401
[57.135637] Connecting 2000000 / 8835401
[58.488792] Connecting 3000000 / 8835401
[59.896351] Connecting 4000000 / 8835401
[61.175038] Connecting 5000000 / 8835401
[62.551398] Connecting 6000000 / 8835401
[63.946740] Connecting 7000000 / 8835401
[65.325190] Connecting 8000000 / 8835401
[66.444718] Cleaning up memory
[66.446429] Done creating preGraph
[66.446438] Concatenation...
[67.476089] Renumbering preNodes
[67.476103] Initial preNode count 4634137
[67.583979] Destroyed 3481578 preNodes
[67.583979] Concatenation over!
[67.583979] Clipping short tips off preGraph
[67.942621] Concatenation...
[68.203200] Renumbering preNodes
[68.203268] Initial preNode count 1152559
[68.232034] Destroyed 924513 preNodes
[68.232043] Concatenation over!
[68.232045] 517511 tips cut off
[68.232046] 228046 nodes left
[68.232707] Writing into pregraph file G04878_55/PreGraph...
[69.377976] Reading read set file G04878_55/Sequences;
[82.857706] 8835401 sequences found
[110.763508] Done
[117.464820] Reading pre-graph file G04878_55/PreGraph
[117.515503] Graph has 228046 nodes and 8835401 sequences
[118.022730] Scanning pre-graph file G04878_55/PreGraph for k-mers
[118.357465] 8157340 kmers found
[118.998832] Sorting kmer occurence table ... 
[122.618727] Sorting done.
[122.618745] Computing acceleration table... 
[122.716591] Computing offsets... 
[122.784652] Ghost Threading through reads 0 / 8835401
[135.345720] Ghost Threading through reads 1000000 / 8835401
[147.948670] Ghost Threading through reads 2000000 / 8835401
[160.542351] Ghost Threading through reads 3000000 / 8835401
[173.395780] Ghost Threading through reads 4000000 / 8835401
[185.952059] Ghost Threading through reads 5000000 / 8835401
[198.767219] Ghost Threading through reads 6000000 / 8835401
[211.027492] Ghost Threading through reads 7000000 / 8835401
[223.416416] Ghost Threading through reads 8000000 / 8835401
[233.404099]  === Ghost-Threaded in 110.619448 s
[233.404278] Threading through reads 0 / 8835401
[247.149642] Threading through reads 1000000 / 8835401
[260.789729] Threading through reads 2000000 / 8835401
[274.409506] Threading through reads 3000000 / 8835401
[288.625753] Threading through reads 4000000 / 8835401
[301.474951] Threading through reads 5000000 / 8835401
[314.567850] Threading through reads 6000000 / 8835401
[327.684980] Threading through reads 7000000 / 8835401
[340.707573] Threading through reads 8000000 / 8835401
[351.153193]  === Threaded in 117.748914 s
[351.613530] Correcting graph with cutoff 0.200000
[351.620056] Determining eligible starting points
[351.883530] Done listing starting nodes
[351.883548] Initializing todo lists
[351.957636] Done with initilization
[351.957654] Activating arc lookup table
[352.093842] Done activating arc lookup table
[352.109954] 10000 / 228046 nodes visited
[352.157779] 20000 / 228046 nodes visited
[352.210544] 30000 / 228046 nodes visited
[352.232741] 40000 / 228046 nodes visited
[352.269662] 50000 / 228046 nodes visited
[352.321687] 60000 / 228046 nodes visited
[352.412743] 70000 / 228046 nodes visited
[352.510638] 80000 / 228046 nodes visited
[352.625995] 90000 / 228046 nodes visited
[352.714075] 100000 / 228046 nodes visited
[352.798537] 110000 / 228046 nodes visited
[352.889010] 120000 / 228046 nodes visited
[353.010317] 130000 / 228046 nodes visited
[353.096480] 140000 / 228046 nodes visited
[353.197392] 150000 / 228046 nodes visited
[353.294700] 160000 / 228046 nodes visited
[353.386514] 170000 / 228046 nodes visited
[353.481332] 180000 / 228046 nodes visited
[353.576470] 190000 / 228046 nodes visited
[353.675140] 200000 / 228046 nodes visited
[353.779508] 210000 / 228046 nodes visited
[353.890313] 220000 / 228046 nodes visited
[354.013335] 230000 / 228046 nodes visited
[354.142602] 240000 / 228046 nodes visited
[354.265972] 250000 / 228046 nodes visited
[354.398265] 260000 / 228046 nodes visited
[354.543226] 270000 / 228046 nodes visited
[354.702417] 280000 / 228046 nodes visited
[354.867007] 290000 / 228046 nodes visited
[355.053004] 300000 / 228046 nodes visited
[355.235496] 310000 / 228046 nodes visited
[355.424671] 320000 / 228046 nodes visited
[355.630184] 330000 / 228046 nodes visited
[355.820660] 340000 / 228046 nodes visited
[356.032094] 350000 / 228046 nodes visited
[356.044713] 360000 / 228046 nodes visited
[356.054347] 370000 / 228046 nodes visited
[356.106086] 380000 / 228046 nodes visited
[356.113456] Concatenation...
[356.171188] Renumbering nodes
[356.171239] Initial node count 228046
[356.176084] Removed 6179 null nodes
[356.176097] Concatenation over!
[356.176099] Clipping short tips off graph, drastic
[356.482602] Concatenation...
[357.197824] Renumbering nodes
[357.197836] Initial node count 221867
[357.199036] Removed 206687 null nodes
[357.199041] Concatenation over!
[357.199042] 15180 nodes left
[357.199817] Writing into graph file G04878_55/Graph2...
[359.803680] Measuring median coverage depth...
[359.808173] Median coverage depth = 80.592998
[359.808415] Removing contigs with coverage < 40.296499...
[359.817200] Concatenation...
[360.825053] Renumbering nodes
[360.825067] Initial node count 15180
[360.825127] Removed 14461 null nodes
[360.825129] Concatenation over!
[360.825143] Concatenation...
[360.825210] Renumbering nodes
[360.825212] Initial node count 719
[360.825213] Removed 0 null nodes
[360.825214] Concatenation over!
[360.825223] Clipping short tips off graph, drastic
[360.825282] Concatenation...
[360.825481] Renumbering nodes
[360.825483] Initial node count 719
[360.825490] Removed 37 null nodes
[360.825491] Concatenation over!
[360.825492] 682 nodes left
[360.825493] Read coherency...
[360.825498] Identifying unique nodes
[360.825509] Done, 429 unique nodes counted
[360.825510] Trimming read tips
[360.825533] Renumbering nodes
[360.825534] Initial node count 682
[360.825535] Removed 0 null nodes
[360.825536] Confronted to 0 multiple hits and 0 null over 0
[360.825538] Read coherency over!
[360.856377] Starting pebble resolution...
[360.909720] Computing read to node mapping array sizes
[361.129617] Computing read to node mappings
[362.123084] Estimating library insert lengths...
[362.209729] Paired-end library 1 has length: 165, sample standard deviation: 335
[362.209746] Done
[362.209760] Computing direct node to node mappings
[362.930579] Scaffolding node 0
[363.641153]  === Nodes Scaffolded in 1.427403 s
[363.796123] Preparing to correct graph with cutoff 0.200000
[363.797858] Cleaning memory
[363.797862] Deactivating local correction settings
[363.797875] Pebble done.
[363.797876] Starting pebble resolution...
[363.824672] Computing read to node mapping array sizes
[364.063795] Computing read to node mappings
[365.069519] Estimating library insert lengths...
[365.157661] Paired-end library 1 has length: 165, sample standard deviation: 335
[365.157680] Done
[365.157688] Computing direct node to node mappings
[365.875150] Scaffolding node 0
[366.587114]  === Nodes Scaffolded in 1.429416 s
[366.748760] Preparing to correct graph with cutoff 0.200000
[366.750596] Cleaning memory
[366.750601] Deactivating local correction settings
[366.750616] Pebble done.
[366.750618] Concatenation...
[366.750638] Renumbering nodes
[366.750640] Initial node count 682
[366.750642] Removed 0 null nodes
[366.750644] Concatenation over!
[366.750645] Removing reference contigs with coverage < 40.296499...
[366.750691] Concatenation...
[366.750709] Renumbering nodes
[366.750710] Initial node count 682
[366.750726] Removed 0 null nodes
[366.750729] Concatenation over!
[366.770413] Writing contigs into G04878_55/contigs.fa...
[368.797593] Writing into stats file G04878_55/stats.txt...
[368.917782] Writing into graph file G04878_55/LastGraph...
[371.399430] Estimated Coverage = 80.592998
[371.399443] Estimated Coverage cutoff = 40.296499
Final graph has 682 nodes and n50 of 22711, max 116791, total 4275613, using 8291131/8835401 reads


>>>>> assemblathon_stats <<<<<


assemblathon_stats.pl ./G04878_55/contigs.fa


assemblathon_stats.pl ./G04878_41/contigs.fa


assemblathon_stats.pl ./G04878_49/contigs.fa


assemblathon_stats.pl ./G04878_55/contigs.fa


 

 Highest quality k_mer : 55




>>>>> abacas_align_contigs <<<<<


cd G04878_55 &&  cp ../NC000962_3.fasta ./ && abacas.pl -r ../NC000962_3.fasta -q contigs.fa -p promer -b -d -a 




>>>>> prokka_annotation <<<<<


cd ./G04878_55 && prokka --outdir ./G04878_prokka --prefix G04878 contigs.fa_NC000962_3.fasta.fasta


[20:08:11] This is prokka 1.12
[20:08:11] Written by Torsten Seemann <torsten.seemann@gmail.com>
[20:08:11] Homepage is https://github.com/tseemann/prokka
[20:08:11] Local time is Tue Dec 12 20:08:11 2017
[20:08:11] You are centos
[20:08:11] Operating system is linux
[20:08:11] You have BioPerl 1.007001
[20:08:12] System has 1 cores.
[20:08:12] Option --cpu asked for 8 cores, but system only has 1
[20:08:12] Will use maximum of 1 cores.
[20:08:12] Annotating as >>> Bacteria <<<
[20:08:12] Generating locus_tag from 'contigs.fa_NC000962_3.fasta.fasta' contents.
[20:08:12] Setting --locustag DEEPGJFF from MD5 dee903ff8ed5d9635a1b6945fdfc915b
[20:08:12] Creating new output folder: ./G04878_prokka
[20:08:12] Running: mkdir -p \.\/G04878_prokka
[20:08:12] Using filename prefix: G04878.XXX
[20:08:12] Setting HMMER_NCPU=1
[20:08:12] Writing log to: ./G04878_prokka/G04878.log
[20:08:12] Command: /opt/prokka/bin/prokka --outdir ./G04878_prokka --prefix G04878 contigs.fa_NC000962_3.fasta.fasta
[20:08:12] Appending to PATH: /opt/prokka/bin/../binaries/linux
[20:08:12] Appending to PATH: /opt/prokka/bin/../binaries/linux/../common
[20:08:12] Appending to PATH: /opt/prokka/bin
[20:08:12] Looking for 'aragorn' - found /opt/prokka/binaries/linux/aragorn
[20:08:12] Determined aragorn version is 1.2
[20:08:12] Looking for 'blastp' - found /opt/prokka/binaries/linux/blastp
[20:08:12] Determined blastp version is 2.4
[20:08:12] Looking for 'cmpress' - found /opt/prokka/binaries/linux/cmpress
[20:08:12] Determined cmpress version is 1.1
[20:08:12] Looking for 'cmscan' - found /opt/prokka/binaries/linux/cmscan
[20:08:12] Determined cmscan version is 1.1
[20:08:12] Looking for 'egrep' - found /usr/bin/egrep
[20:08:12] Looking for 'find' - found /usr/bin/find
[20:08:12] Looking for 'grep' - found /usr/bin/grep
[20:08:12] Looking for 'hmmpress' - found /opt/prokka/binaries/linux/hmmpress
[20:08:12] Determined hmmpress version is 3.1
[20:08:12] Looking for 'hmmscan' - found /opt/prokka/binaries/linux/hmmscan
[20:08:12] Determined hmmscan version is 3.1
[20:08:12] Looking for 'java' - found /usr/bin/java
[20:08:12] Looking for 'less' - found /usr/bin/less
[20:08:12] Looking for 'makeblastdb' - found /opt/prokka/binaries/linux/makeblastdb
[20:08:12] Determined makeblastdb version is 2.4
[20:08:12] Looking for 'minced' - found /opt/prokka/binaries/common/minced
[20:08:12] Determined minced version is 2.0
[20:08:12] Looking for 'parallel' - found /opt/snippy/binaries/noarch/parallel
[20:08:12] Determined parallel version is 20150122
[20:08:12] Looking for 'prodigal' - found /opt/prokka/binaries/linux/prodigal
[20:08:12] Determined prodigal version is 2.6
[20:08:12] Looking for 'prokka-genbank_to_fasta_db' - found /opt/prokka/bin/prokka-genbank_to_fasta_db
[20:08:12] Looking for 'sed' - found /usr/bin/sed
[20:08:12] Looking for 'tbl2asn' - found /opt/prokka/binaries/linux/tbl2asn
[20:08:12] Determined tbl2asn version is 25.3
[20:08:12] Using genetic code table 11.
[20:08:12] Loading and checking input file: contigs.fa_NC000962_3.fasta.fasta
[20:08:13] Wrote 1 contigs totalling 4504917 bp.
[20:08:13] Predicting tRNAs and tmRNAs
[20:08:13] Running: aragorn -l -gc11  -w \.\/G04878_prokka\/G04878\.fna
[20:08:16] 1 tRNA-Ile [37709,37783] 35 (gat)
[20:08:16] 2 tRNA-Ala [37934,38007] 34 (tgc)
[20:08:16] 3 tRNA-Leu [52448,52532] 34 (cag)
[20:08:16] 4 tRNA-Gly c[415168,415240] 33 (ccc)
[20:08:16] 5 tRNA-Tyr [688853,688936] 35 (gta)
[20:08:16] 6 tRNA-Thr [762368,762441] 34 (ggt)
[20:08:16] 7 tRNA-Met [762477,762551] 35 (cat)
[20:08:16] 8 tRNA-Trp [764398,764471] 34 (cca)
[20:08:16] 9 tRNA-Thr c[882569,882641] 33 (tgt)
[20:08:16] 10 tRNA-Lys c[955873,955946] 34 (ttt)
[20:08:16] 11 tRNA-Glu [956070,956145] 36 (ttc)
[20:08:16] 12 tRNA-Asp [956181,956257] 35 (gtc)
[20:08:16] 13 tRNA-Phe [956284,956360] 35 (gaa)
[20:08:16] 14 tRNA-Arg c[1057500,1057573] 34 (cct)
[20:08:16] 15 tRNA-Ala [1146277,1146352] 34 (cgc)
[20:08:16] 16 tRNA-Gln c[1170970,1171042] 34 (ttg)
[20:08:16] 17 tRNA-Leu [1210682,1210756] 35 (taa)
[20:08:16] 18 tRNA-Arg c[1481487,1481560] 34 (ccg)
[20:08:16] 19 tRNA-Leu [1548385,1548469] 35 (tag)
[20:08:16] 20 tRNA-Leu c[1617542,1617628] 32 (aag)
[20:08:16] 21 tRNA-Arg c[1663236,1663332] 37 (ccg)
[20:08:16] 22 tRNA-Leu c[1868237,1868313] 35 (caa)
[20:08:16] 23 tRNA-Pro [1987081,1987155] 35 (ggg)
[20:08:16] 24 tRNA-Leu [2454054,2454140] 34 (gag)
[20:08:16] 25 tRNA-Val c[2563359,2563432] 33 (tac)
[20:08:16] 26 tRNA-Met c[2634940,2635014] 35 (cat)
[20:08:16] 27 tRNA-Asn c[2672919,2672992] 34 (gtt)
[20:08:16] 28 tRNA-Pro c[2819592,2819668] 35 (tgg)
[20:08:16] 29 tRNA-Gly [2819805,2819876] 33 (tcc)
[20:08:16] 30 tRNA-Arg [2847082,2847156] 35 (tct)
[20:08:16] 31 tRNA-His [2882254,2882329] 34 (gtg)
[20:08:16] 32 tRNA-Lys [2889894,2889967] 34 (ctt)
[20:08:16] 33 tRNA-Val c[3024346,3024418] 33 (cac)
[20:08:16] 34 tRNA-Gly [3024603,3024676] 34 (gcc)
[20:08:16] 35 tRNA-Cys [3024705,3024778] 33 (gca)
[20:08:16] 36 tRNA-Val [3024792,3024866] 33 (gac)
[20:08:16] 37 tRNA-Thr c[3200027,3200101] 33 (ggt)
[20:08:16] 38 tRNA-Glu c[3407172,3407245] 35 (ctc)
[20:08:16] 39 tRNA-Gln c[3407284,3407356] 34 (ctg)
[20:08:16] 40 tRNA-Ala c[3491097,3491170] 34 (ggc)
[20:08:16] 41 tmRNA c[3527347,3527713] 95,133 ADSHQRDYALAA*
[20:08:16] 42 tRNA-Met c[3619172,3619248] 35 (cat)
[20:08:16] 43 tRNA-His [3822379,3822478] 38 (gtg)
[20:08:16] 44 tRNA-Gly c[4048254,4048346] 33 (gcc)
[20:08:16] 45 tRNA-Thr c[4146050,4146125] 34 (cgt)
[20:08:16] 46 tRNA-Pro [4191229,4191304] 35 (cgg)
[20:08:16] 47 tRNA-Ser [4233181,4233269] 35 (gga)
[20:08:16] 48 tRNA-Ser c[4263964,4264053] 35 (cga)
[20:08:16] 49 tRNA-Arg c[4281844,4281917] 34 (acg)
[20:08:16] 50 tRNA-Ser c[4281947,4282036] 35 (gct)
[20:08:16] 51 tRNA-Ser c[4287560,4287647] 35 (tga)
[20:08:16] Found 51 tRNAs
[20:08:16] Predicting Ribosomal RNAs
[20:08:16] You need either Barrnap or RNAmmer installed to predict rRNAs!
[20:08:16] Skipping ncRNA search, enable with --rfam if desired.
[20:08:16] Total of 50 tRNA + rRNA features
[20:08:16] Searching for CRISPR repeats
[20:08:17] CRISPR1 ordered_NC000962_3 2584833 with 3 spacers
[20:08:17] CRISPR2 ordered_NC000962_3 3175273 with 28 spacers
[20:08:17] CRISPR3 ordered_NC000962_3 3178099 with 4 spacers
[20:08:17] CRISPR4 ordered_NC000962_3 3178862 with 15 spacers
[20:08:17] Found 4 CRISPRs
[20:08:17] Predicting coding sequences
[20:08:17] Contigs total 4504917 bp, so using single mode
[20:08:17] Running: prodigal -i \.\/G04878_prokka\/G04878\.fna -c -m -g 11 -p single -f sco -q
[20:08:24] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:37945..38163 on - strand
[20:08:24] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:52490..52819 on + strand
[20:08:24] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:956106..956273 on - strand
[20:08:25] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:1617370..1618194 on - strand
[20:08:25] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:1663190..1664311 on - strand
[20:08:26] Excluding CDS which overlaps existing RNA (repeat_region) at ordered_NC000962_3:3175393..3175629 on - strand
[20:08:26] Excluding CDS which overlaps existing RNA (repeat_region) at ordered_NC000962_3:3176140..3176376 on - strand
[20:08:26] Excluding CDS which overlaps existing RNA (repeat_region) at ordered_NC000962_3:3176520..3177161 on - strand
[20:08:26] Excluding CDS which overlaps existing RNA (repeat_region) at ordered_NC000962_3:3178975..3179871 on - strand
[20:08:26] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:3199094..3200404 on - strand
[20:08:27] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:3822366..3822506 on + strand
[20:08:27] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:4047405..4048424 on - strand
[20:08:27] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:4233202..4233363 on + strand
[20:08:27] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:4263867..4264034 on + strand
[20:08:27] Excluding CDS which overlaps existing RNA (tRNA) at ordered_NC000962_3:4286069..4287628 on + strand
[20:08:27] Found 4276 CDS
[20:08:27] Connecting features back to sequences
[20:08:27] Not using genus-specific database. Try --usegenus to enable it.
[20:08:27] Annotating CDS, please be patient.
[20:08:27] Will use 1 CPUs for similarity searching.
[20:08:30] There are still 4276 unannotated CDS left (started with 4276)
[20:08:30] Will use blast to search against /opt/prokka/bin/../db/kingdom/Bacteria/sprot with 1 CPUs
[20:08:30] Running: cat \.\/G04878_prokka\/sprot\.faa | parallel --gnu --plain -j 1 --block 646596 --recstart '>' --pipe blastp -query - -db /opt/prokka/bin/../db/kingdom/Bacteria/sprot -evalue 1e-06 -num_threads 1 -num_descriptions 1 -num_alignments 1 -seg no > \.\/G04878_prokka\/sprot\.blast 2> /dev/null
[20:12:25] Modify product: Probable peptidoglycan biosynthesis protein MviN => putative peptidoglycan biosynthesis protein MviN
[20:12:25] Modify product: Probable chromosome-partitioning protein ParB => putative chromosome-partitioning protein ParB
[20:12:25] Modify product: Uncharacterized FtsW-like protein => putative FtsW-like protein
[20:12:25] Modify product: Putative monooxygenase Rv1533 => Putative monooxygenase
[20:12:25] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0023 => putative HTH-type transcriptional regulator
[20:12:25] Modify product: Probable endopeptidase cgR_2070 => putative endopeptidase
[20:12:25] Modify product: Putative ligase Rv1013 => Putative ligase
[20:12:25] Modify product: Uncharacterized MFS-type transporter Rv0037c => putative MFS-type transporter
[20:12:25] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0880 => putative HTH-type transcriptional regulator
[20:12:25] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0043c => putative HTH-type transcriptional regulator
[20:12:25] Modify product: Putative coenzyme F420-dependent oxidoreductase Rv3520c => Putative coenzyme F420-dependent oxidoreductase
[20:12:25] Modify product: Uncharacterized FAD-linked oxidoreductase YvdP => putative FAD-linked oxidoreductase YvdP
[20:12:25] Modify product: Uncharacterized ABC transporter permease Rv0072 => putative ABC transporter permease
[20:12:25] Modify product: Uncharacterized ABC transporter ATP-binding protein Rv0073 => putative ABC transporter ATP-binding protein
[20:12:25] Modify product: Uncharacterized HTH-type transcriptional regulator MT0088 => putative HTH-type transcriptional regulator
[20:12:25] Modify product: Uncharacterized PPE family protein PPE2 => putative PPE family protein PPE2
[20:12:25] Modify product: Putative dioxygenase Rv0097 => Putative dioxygenase
[20:12:25] Modify product: Uncharacterized NTE family protein Rv2565 => putative NTE family protein
[20:12:25] Modify product: Probable cation-transporting ATPase I => putative cation-transporting ATPase I
[20:12:25] Modify product: Probable hydrogen peroxide-inducible genes activator => putative hydrogen peroxide-inducible genes activator
[20:12:25] Modify product: Probable sulfoacetate--CoA ligase => putative sulfoacetate--CoA ligase
[20:12:25] Modify product: Probable enoyl-CoA hydratase 1 => putative enoyl-CoA hydratase 1
[20:12:25] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv0145 => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:25] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv0146 => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:25] Modify product: Putative short-chain type dehydrogenase/reductase Rv0148 => Putative short-chain type dehydrogenase/reductase
[20:12:25] Modify product: Uncharacterized PE-PGRS family protein PE_PGRS24 => putative PE-PGRS family protein PE_PGRS24
[20:12:25] Modify product: Uncharacterized FAD-linked oxidoreductase Rv2280 => putative FAD-linked oxidoreductase
[20:12:25] Modify product: Probable phospholipid ABC transporter permease protein MlaE => putative phospholipid ABC transporter permease protein MlaE
[20:12:25] Modify product: Probable phospholipid ABC transporter permease protein MlaE => putative phospholipid ABC transporter permease protein MlaE
[20:12:25] Modify product: Proline-rich antigen homolog => Proline-rich antigen 
[20:12:25] Modify product: Putative quercetin 2,3-dioxygenase Rv0181c => Putative quercetin 2,3-dioxygenase
[20:12:25] Modify product: Putative O-methyltransferase Rv1220c => Putative O-methyltransferase
[20:12:25] Modify product: Multidrug efflux ATP-binding/permease protein Rv0194 => Multidrug efflux ATP-binding/permease protein
[20:12:25] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0196 => putative HTH-type transcriptional regulator
[20:12:25] Modify product: Heme-binding protein Rv0203 => Heme-binding protein
[20:12:25] Modify product: Putative diacyglycerol O-acyltransferase Rv0221 => Putative diacyglycerol O-acyltransferase
[20:12:25] Modify product: Putative diacyglycerol O-acyltransferase Rv0221 => Putative diacyglycerol O-acyltransferase
[20:12:25] Modify product: Uncharacterized methyltransferase Rv0224c => putative methyltransferase
[20:12:25] Modify product: Putative secreted protein Rv0236.1 => Putative secreted protein.1
[20:12:25] Modify product: Uncharacterized PPE family protein PPE2 => putative PPE family protein PPE2
[20:12:25] Modify product: Uncharacterized PPE family protein PPE3 => putative PPE family protein PPE3
[20:12:25] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv0281 => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:25] Modify product: Probable trans-aconitate 2-methyltransferase => putative trans-aconitate 2-methyltransferase
[20:12:25] Modify product: Antitoxin Rv0298 => Antitoxin
[20:12:25] Modify product: Toxin Rv0299 => Toxin
[20:12:26] Modify product: Probable aspartate aminotransferase => putative aspartate aminotransferase
[20:12:26] Modify product: Uncharacterized membrane protein Rv0364 => putative membrane protein
[20:12:26] Modify product: Probable xanthine dehydrogenase subunit A => putative xanthine dehydrogenase subunit A
[20:12:26] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0377 => putative HTH-type transcriptional regulator
[20:12:26] Modify product: Putative HTH-type transcriptional regulator Rv0890c => Putative HTH-type transcriptional regulator
[20:12:26] Modify product: Uncharacterized PPE family protein PPE32 => putative PPE family protein PPE32
[20:12:26] Modify product: NADH dehydrogenase-like protein SA0802 => NADH dehydrogenase-like protein
[20:12:26] Modify product: Probable transport protein MmpL1 => putative transport protein MmpL1
[20:12:26] Modify product: Probable metallo-hydrolase YflN => putative metallo-hydrolase YflN
[20:12:26] Modify product: Probable lipoprotein aminopeptidase LpqL => putative lipoprotein aminopeptidase LpqL
[20:12:26] Modify product: Probable cation-transporting ATPase I => putative cation-transporting ATPase I
[20:12:26] Modify product: Uncharacterized oxidoreductase Rv0769 => putative oxidoreductase
[20:12:26] Modify product: Uncharacterized PPE family protein PPE10 => putative PPE family protein PPE10
[20:12:26] Modify product: Uncharacterized PPE family protein PPE20 => putative PPE family protein PPE20
[20:12:26] Modify product: Probable aldehyde dehydrogenase => putative aldehyde dehydrogenase
[20:12:26] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0472c => putative HTH-type transcriptional regulator
[20:12:26] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0474 => putative HTH-type transcriptional regulator
[20:12:26] Modify product: Hydrolase Rv0480c => Hydrolase
[20:12:26] Modify product: Uncharacterized oxidoreductase Rv0484c => putative oxidoreductase
[20:12:26] Modify product: Transcriptional regulator Rv0485 => Transcriptional regulator
[20:12:26] Modify product: Uncharacterized GMC-type oxidoreductase Rv0492c => putative GMC-type oxidoreductase
[20:12:26] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0494 => putative HTH-type transcriptional regulator
[20:12:26] Modify product: Putative DNA-binding protein Rv0500A => Putative DNA-binding proteinA
[20:12:26] Modify product: Probable transport accessory protein MmpS2 => putative transport accessory protein MmpS2
[20:12:26] Modify product: Probable transport protein MmpL2 => putative transport protein MmpL2
[20:12:26] Modify product: Uncharacterized glycosyltransferase Rv0539 => putative glycosyltransferase
[20:12:26] Modify product: Probable 2-succinylbenzoate--CoA ligase => putative 2-succinylbenzoate--CoA ligase
[20:12:26] Modify product: Protease HtpX homolog => Protease HtpX 
[20:12:26] Modify product: Probable glycerol-3-phosphate dehydrogenase 2 [NAD(P)+] => putative glycerol-3-phosphate dehydrogenase 2 [NAD(P)+]
[20:12:26] Modify product: Putative phosphoribosyl transferase MT0597 => Putative phosphoribosyl transferase
[20:12:26] Modify product: Probable polyglutamine synthesis accessory protein MT0602 => putative polyglutamine synthesis accessory protein
[20:12:26] Modify product: HTH-type transcriptional regulator Rv2034 => HTH-type transcriptional regulator
[20:12:26] Modify product: Probable phospholipid ABC transporter permease protein MlaE => putative phospholipid ABC transporter permease protein MlaE
[20:12:26] Modify product: Probable phospholipid ABC transporter permease protein MlaE => putative phospholipid ABC transporter permease protein MlaE
[20:12:26] Modify product: TVP38/TMEM64 family membrane protein Rv0625c => TVP38/TMEM64 family membrane protein
[20:12:26] Modify product: Probable enoyl-CoA hydratase echA17 => putative enoyl-CoA hydratase echA17
[20:12:26] Modify product: Probable polyketide biosynthesis zinc-dependent hydrolase BaeB => putative polyketide biosynthesis zinc-dependent hydrolase BaeB
[20:12:26] Modify product: Probable protein kinase UbiB => putative protein kinase UbiB
[20:12:26] Modify product: Probable ribonucleotide transport ATP-binding protein mkl => putative ribonucleotide transport ATP-binding protein mkl
[20:12:26] Modify product: Probable endoribonuclease MazF2 => putative endoribonuclease MazF2
[20:12:26] Modify product: Probable endonuclease 4 => putative endonuclease 4
[20:12:26] Modify product: Uncharacterized NAD-dependent oxidoreductase Rv0687 => putative NAD-dependent oxidoreductase
[20:12:26] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv1729c => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:26] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv0726c => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:26] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv0731c => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:26] Modify product: Probable 3-hydroxyisobutyrate dehydrogenase => putative 3-hydroxyisobutyrate dehydrogenase
[20:12:26] Modify product: Probable 3-hydroxyisobutyrate dehydrogenase => putative 3-hydroxyisobutyrate dehydrogenase
[20:12:26] Modify product: Uncharacterized PPE family protein PPE40 => putative PPE family protein PPE40
[20:12:26] Modify product: Probable transcriptional regulatory protein TcrX => putative transcriptional regulatory protein TcrX
[20:12:26] Modify product: Probable sensor histidine kinase TcrY => putative sensor histidine kinase TcrY
[20:12:26] Modify product: Uncharacterized HIT-like protein Rv0759c => putative HIT-like protein
[20:12:26] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0767c => putative HTH-type transcriptional regulator
[20:12:26] Modify product: Putative aldehyde dehydrogenase SAV2122 => Putative aldehyde dehydrogenase
[20:12:26] Modify product: Uncharacterized oxidoreductase Rv0769 => putative oxidoreductase
[20:12:26] Modify product: Uncharacterized oxidoreductase Rv0770 => putative oxidoreductase
[20:12:26] Modify product: Probable transport protein HsrA => putative transport protein HsrA
[20:12:26] Modify product: KsdD-like steroid dehydrogenase Rv0785 => KsdD-like steroid dehydrogenase
[20:12:26] Modify product: Putative monooxygenase Rv0793 => Putative monooxygenase
[20:12:26] Modify product: Probable deferrochelatase/peroxidase YfeX => putative deferrochelatase/peroxidase YfeX
[20:12:26] Modify product: Probable M18 family aminopeptidase 2 => putative M18 family aminopeptidase 2
[20:12:26] Modify product: Putative succinyl-CoA transferase Rv0802c => Putative succinyl-CoA transferase
[20:12:26] Modify product: Phosphate-specific transport system accessory protein PhoU homolog 2 => Phosphate-specific transport system accessory protein PhoU 
[20:12:26] Modify product: Probable tRNA-dihydrouridine synthase => putative tRNA-dihydrouridine synthase
[20:12:26] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv0830 => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:26] Modify product: Probable transcriptional regulatory protein NarL => putative transcriptional regulatory protein NarL
[20:12:26] Modify product: Probable sensor histidine kinase NarS => putative sensor histidine kinase NarS
[20:12:26] Modify product: Probable siderophore biosynthesis protein SbnA => putative siderophore biosynthesis protein SbnA
[20:12:26] Modify product: Probable N-succinyldiaminopimelate aminotransferase DapC => putative N-succinyldiaminopimelate aminotransferase DapC
[20:12:26] Modify product: Putative acyltransferase Rv0859 => Putative acyltransferase
[20:12:26] Modify product: Probable acyl-CoA dehydrogenase FadE10 => putative acyl-CoA dehydrogenase FadE10
[20:12:26] Modify product: Uncharacterized PPE family protein PPE40 => putative PPE family protein PPE40
[20:12:26] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0880 => putative HTH-type transcriptional regulator
[20:12:26] Modify product: Probable ferredoxin/ferredoxin--NADP reductase => putative ferredoxin/ferredoxin--NADP reductase
[20:12:26] Modify product: Putative HTH-type transcriptional regulator Rv0890c => Putative HTH-type transcriptional regulator
[20:12:26] Modify product: Uncharacterized monooxygenase Rv0892 => putative monooxygenase
[20:12:26] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv0893c => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:26] Modify product: Putative HTH-type transcriptional regulator Rv0890c => Putative HTH-type transcriptional regulator
[20:12:26] Modify product: Putative diacyglycerol O-acyltransferase Rv0895 => Putative diacyglycerol O-acyltransferase
[20:12:26] Modify product: Uncharacterized membrane protein ArfB => putative membrane protein ArfB
[20:12:26] Modify product: Uncharacterized membrane protein ArfC => putative membrane protein ArfC
[20:12:26] Modify product: Probable enoyl-CoA hydratase echA6 => putative enoyl-CoA hydratase echA6
[20:12:26] Modify product: Probable cation-transporting ATPase E => putative cation-transporting ATPase E
[20:12:26] Modify product: Antitoxin Rv0909 => Antitoxin
[20:12:26] Modify product: Toxin Rv0910 => Toxin
[20:12:26] Modify product: Uncharacterized PPE family protein PPE29 => putative PPE family protein PPE29
[20:12:26] Modify product: Uncharacterized oxidoreductase Rv0927c => putative oxidoreductase
[20:12:26] Modify product: Uncharacterized formamidopyrimidine-DNA glycosylase-like protein => putative formamidopyrimidine-DNA glycosylase-like protein
[20:12:26] Modify product: Uncharacterized oxidoreductase Rv0945 => putative oxidoreductase
[20:12:26] Modify product: 34 kDa antigenic protein homolog => 34 kDa antigenic protein 
[20:12:26] Modify product: Probable copper-exporting P-type ATPase V => putative copper-exporting P-type ATPase V
[20:12:26] Modify product: Probable enoyl-CoA hydratase => putative enoyl-CoA hydratase
[20:12:26] Modify product: Probable propionyl-CoA carboxylase beta chain 5 => putative propionyl-CoA carboxylase beta chain 5
[20:12:26] Modify product: Uncharacterized ABC transporter ATP-binding protein Rv0986 => putative ABC transporter ATP-binding protein
[20:12:26] Modify product: Probable dolichyl-phosphate-mannose--protein mannosyltransferase => putative dolichyl-phosphate-mannose--protein mannosyltransferase
[20:12:26] Modify product: Uncharacterized metal-dependent hydrolase TatD => putative metal-dependent hydrolase TatD
[20:12:26] Modify product: Putative ligase Rv1013 => Putative ligase
[20:12:26] Modify product: Probable sensor histidine kinase TcrY => putative sensor histidine kinase TcrY
[20:12:26] Modify product: Probable transcriptional regulatory protein TcrX => putative transcriptional regulatory protein TcrX
[20:12:26] Modify product: Uncharacterized PE family protein PE23 => putative PE family protein PE23
[20:12:26] Modify product: Uncharacterized NTE family protein Rv1063c => putative NTE family protein
[20:12:26] Modify product: Probable enoyl-CoA hydratase echA8 => putative enoyl-CoA hydratase echA8
[20:12:26] Modify product: Putative cystathionine beta-synthase Rv1077 => Putative cystathionine beta-synthase
[20:12:26] Modify product: Proline-rich antigen homolog => Proline-rich antigen 
[20:12:26] Modify product: Uncharacterized PPE family protein PPE40 => putative PPE family protein PPE40
[20:12:26] Modify product: Uncharacterized oxidoreductase Rv1144 => putative oxidoreductase
[20:12:26] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv0830 => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:26] Modify product: Probable monoacyl phosphatidylinositol tetramannoside-binding protein LpqW => putative monoacyl phosphatidylinositol tetramannoside-binding protein LpqW
[20:12:26] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1556 => putative HTH-type transcriptional regulator
[20:12:26] Modify product: Uncharacterized PPE family protein PPE17 => putative PPE family protein PPE17
[20:12:26] Modify product: Probable ECF RNA polymerase sigma factor SigI => putative ECF RNA polymerase sigma factor SigI
[20:12:26] Modify product: Putative O-methyltransferase Rv1220c => Putative O-methyltransferase
[20:12:26] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1255c => putative HTH-type transcriptional regulator
[20:12:26] Modify product: Uncharacterized FAD-linked oxidoreductase Rv2280 => putative FAD-linked oxidoreductase
[20:12:26] Modify product: Probable multidrug-efflux transporter Rv1258c => putative multidrug-efflux transporter
[20:12:26] Modify product: F420H(2)-dependent quinone reductase Rv1261c => F420H(2)-dependent quinone reductase
[20:12:26] Modify product: Uncharacterized HIT-like protein HP_0404 => putative HIT-like protein
[20:12:26] Modify product: pH-sensitive adenylate cyclase Rv1264 => pH-sensitive adenylate cyclase
[20:12:26] Modify product: Uncharacterized ABC transporter ATP-binding protein Rv1272c => putative ABC transporter ATP-binding protein
[20:12:26] Modify product: Uncharacterized ABC transporter ATP-binding protein Rv1273c => putative ABC transporter ATP-binding protein
[20:12:26] Modify product: Uncharacterized GMC-type oxidoreductase Rv1279 => putative GMC-type oxidoreductase
[20:12:26] Modify product: Probable monoacyl phosphatidylinositol tetramannoside-binding protein LpqW => putative monoacyl phosphatidylinositol tetramannoside-binding protein LpqW
[20:12:26] Modify product: Uncharacterized ABC transporter ATP-binding protein Rv1281c => putative ABC transporter ATP-binding protein
[20:12:26] Modify product: Putative peptide transport permease protein Rv1282c => Putative peptide transport permease protein
[20:12:26] Modify product: Putative peptide transport permease protein Rv1283c => Putative peptide transport permease protein
[20:12:26] Modify product: Putative HTH-type transcriptional regulator Rv1287 => Putative HTH-type transcriptional regulator
[20:12:26] Modify product: Probable bifunctional transcriptional activator/DNA repair enzyme AlkA => putative bifunctional transcriptional activator/DNA repair enzyme AlkA
[20:12:26] Modify product: Probable acetyl-CoA acetyltransferase => putative acetyl-CoA acetyltransferase
[20:12:26] Modify product: Probable ATP-dependent helicase DinG homolog => putative ATP-dependent helicase DinG 
[20:12:26] Modify product: Uncharacterized aminopeptidase Rv1333 => putative aminopeptidase
[20:12:26] Modify product: Uncharacterized oxidoreductase Rv1350 => putative oxidoreductase
[20:12:26] Modify product: Uncharacterized signaling protein CC_0091 => putative signaling protein
[20:12:26] Modify product: Probable adenylyltransferase/sulfurtransferase MoeZ => putative adenylyltransferase/sulfurtransferase MoeZ
[20:12:26] Modify product: Uncharacterized signaling protein CC_0091 => putative signaling protein
[20:12:26] Modify product: Putative HTH-type transcriptional regulator Rv0890c => Putative HTH-type transcriptional regulator
[20:12:26] Modify product: Putative coenzyme F420-dependent oxidoreductase Rv3520c => Putative coenzyme F420-dependent oxidoreductase
[20:12:26] Modify product: Glycolipid sulfotransferase Rv1373 => Glycolipid sulfotransferase
[20:12:26] Modify product: Uncharacterized PPE family protein PPE20 => putative PPE family protein PPE20
[20:12:26] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1395 => putative HTH-type transcriptional regulator
[20:12:26] Modify product: Uncharacterized membrane protein Rv1401 => putative membrane protein
[20:12:26] Modify product: Probable primosomal protein N' => putative primosomal protein N'
[20:12:26] Modify product: Putative methyltransferase Rv1407 => Putative methyltransferase
[20:12:26] Modify product: Probable triacylglyceride transporter BCG_1471c => putative triacylglyceride transporter
[20:12:26] Modify product: Nucleotide-binding protein Rv1421 => Nucleotide-binding protein
[20:12:26] Modify product: Putative diacyglycerol O-acyltransferase Rv1425 => Putative diacyglycerol O-acyltransferase
[20:12:26] Modify product: Probable L,D-transpeptidase 3 => putative L,D-transpeptidase 3
[20:12:26] Modify product: Probable protein-export membrane protein SecG => putative protein-export membrane protein SecG
[20:12:26] Modify product: Alpha-(1->6)-mannopyranosyltransferase Rv1459c => Alpha-(1->6)-mannopyranosyltransferase
[20:12:26] Modify product: Probable cysteine desulfurase => putative cysteine desulfurase
[20:12:26] Modify product: Uncharacterized PE-PGRS family protein PE_PGRS24 => putative PE-PGRS family protein PE_PGRS24
[20:12:26] Modify product: Probable cobalt/nickel-exporting P-type ATPase => putative cobalt/nickel-exporting P-type ATPase
[20:12:26] Modify product: Probable enoyl-CoA hydratase echA12 => putative enoyl-CoA hydratase echA12
[20:12:26] Modify product: Uncharacterized ABC transporter ATP-binding protein YbiT => putative ABC transporter ATP-binding protein YbiT
[20:12:26] Modify product: Probable methylmalonyl-CoA mutase large subunit => putative methylmalonyl-CoA mutase large subunit
[20:12:27] Modify product: Probable methylmalonyl-CoA mutase small subunit => putative methylmalonyl-CoA mutase small subunit
[20:12:27] Modify product: Probable methylmalonyl-CoA mutase large subunit => putative methylmalonyl-CoA mutase large subunit
[20:12:27] Modify product: Probable antitoxin MazE4 => putative antitoxin MazE4
[20:12:27] Modify product: Probable GTPase Rv1496 => putative GTPase
[20:12:27] Modify product: Uncharacterized glycosyltransferase Rv1514c => putative glycosyltransferase
[20:12:27] Modify product: Uncharacterized glycosyltransferase Rv1524 => putative glycosyltransferase
[20:12:27] Modify product: Uncharacterized glycosyltransferase Rv1524 => putative glycosyltransferase
[20:12:27] Modify product: Probable alcohol dehydrogenase adh => putative alcohol dehydrogenase adh
[20:12:27] Modify product: Putative monooxygenase Rv1533 => Putative monooxygenase
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator YfiR => putative HTH-type transcriptional regulator YfiR
[20:12:27] Modify product: Probable L-asparaginase => putative L-asparaginase
[20:12:27] Modify product: Uncharacterized RNA pseudouridine synthase Rv1540 => putative RNA pseudouridine synthase
[20:12:27] Modify product: Uncharacterized oxidoreductase Rv1543 => putative oxidoreductase
[20:12:27] Modify product: Uncharacterized oxidoreductase SA2266 => putative oxidoreductase
[20:12:27] Modify product: Toxin Rv0910 => Toxin
[20:12:27] Modify product: Uncharacterized PPE family protein PPE40 => putative PPE family protein PPE40
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1556 => putative HTH-type transcriptional regulator
[20:12:27] Modify product: F420H(2)-dependent quinone reductase Rv1558 => F420H(2)-dependent quinone reductase
[20:12:27] Modify product: Glycogen operon protein GlgX homolog => Glycogen operon protein GlgX 
[20:12:27] Modify product: Probable inactive lipase Rv1592c => putative inactive lipase
[20:12:27] Modify product: Probable inositol 1-monophosphatase ImpA => putative inositol 1-monophosphatase ImpA
[20:12:27] Modify product: Putative peroxiredoxin Rv1608c => Putative peroxiredoxin
[20:12:27] Modify product: Probable ABC transporter ATP-binding protein HI_0664 => putative ABC transporter ATP-binding protein
[20:12:27] Modify product: Probable transcriptional regulatory protein pdtaR => putative transcriptional regulatory protein pdtaR
[20:12:27] Modify product: Probable multidrug-efflux transporter Rv1634 => putative multidrug-efflux transporter
[20:12:27] Modify product: Universal stress protein Rv1636 => Universal stress protein
[20:12:27] Modify product: pH-sensitive adenylate cyclase Rv1264 => pH-sensitive adenylate cyclase
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator MT0088 => putative HTH-type transcriptional regulator
[20:12:27] Modify product: HTH-type transcriptional repressor Rv3405c => HTH-type transcriptional repressor
[20:12:27] Modify product: Uncharacterized ABC transporter ATP-binding protein YbhF => putative ABC transporter ATP-binding protein YbhF
[20:12:27] Modify product: Uncharacterized PPE family protein PPE32 => putative PPE family protein PPE32
[20:12:27] Modify product: Uncharacterized RNA pseudouridine synthase Rv1711 => putative RNA pseudouridine synthase
[20:12:27] Modify product: Probable 3-hydroxybutyryl-CoA dehydrogenase => putative 3-hydroxybutyryl-CoA dehydrogenase
[20:12:27] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv1729c => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:27] Modify product: Probable membrane protein Rv1733c => putative membrane protein
[20:12:27] Modify product: Uncharacterized membrane protein MT1776 => putative membrane protein
[20:12:27] Modify product: Probable nitrate/nitrite transporter NarK2 => putative nitrate/nitrite transporter NarK2
[20:12:27] Modify product: Probable sulfate transporter Rv1739c => putative sulfate transporter
[20:12:27] Modify product: ABC transporter ATP-binding/permease protein Rv1747 => ABC transporter ATP-binding/permease protein
[20:12:27] Modify product: Probable cutinase Rv1984c => putative cutinase
[20:12:27] Modify product: Putative diacyglycerol O-acyltransferase Rv1760 => Putative diacyglycerol O-acyltransferase
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1255c => putative HTH-type transcriptional regulator
[20:12:27] Modify product: Uncharacterized PPE family protein PPE42 => putative PPE family protein PPE42
[20:12:27] Modify product: Uncharacterized PPE family protein PPE33 => putative PPE family protein PPE33
[20:12:27] Modify product: Uncharacterized PPE family protein PPE29 => putative PPE family protein PPE29
[20:12:27] Modify product: Uncharacterized PPE family protein PPE32 => putative PPE family protein PPE32
[20:12:27] Modify product: Uncharacterized PPE family protein PPE33 => putative PPE family protein PPE33
[20:12:27] Modify product: NADH dehydrogenase-like protein MT1860 => NADH dehydrogenase-like protein
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1816 => putative HTH-type transcriptional regulator
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1828 => putative HTH-type transcriptional regulator
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1830 => putative HTH-type transcriptional regulator
[20:12:27] Modify product: Probable glycine dehydrogenase (decarboxylating) => putative glycine dehydrogenase (decarboxylating)
[20:12:27] Modify product: Putative serine esterase Rv1835c => Putative serine esterase
[20:12:27] Modify product: Uncharacterized oxidoreductase Rv1843c => putative oxidoreductase
[20:12:27] Modify product: Putative esterase Rv1847 => Putative esterase
[20:12:27] Modify product: NADH dehydrogenase-like protein SA0802 => NADH dehydrogenase-like protein
[20:12:27] Modify product: Putative oxidoreductase Rv1856c => Putative oxidoreductase
[20:12:27] Modify product: Probable alcohol dehydrogenase AdhA => putative alcohol dehydrogenase AdhA
[20:12:27] Modify product: Probable oxidoreductase EphD => putative oxidoreductase EphD
[20:12:27] Modify product: Probable CoA-transferase Rv1866 => putative CoA-transferase
[20:12:27] Modify product: Probable lipoprotein LppE => putative lipoprotein LppE
[20:12:27] Modify product: Uncharacterized oxidoreductase SA2266 => putative oxidoreductase
[20:12:27] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv3787c => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:27] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv3399 => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:27] Modify product: Putative monooxygenase Rv1533 => Putative monooxygenase
[20:12:27] Modify product: Putative monooxygenase Rv1533 => Putative monooxygenase
[20:12:27] Modify product: Probable zinc-binding alcohol dehydrogenase Rv1895 => putative zinc-binding alcohol dehydrogenase
[20:12:27] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv1896c => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:27] Modify product: Probable D-amino-acid oxidase => putative D-amino-acid oxidase
[20:12:27] Modify product: HTH-type transcriptional regulator Rv1931c => HTH-type transcriptional regulator
[20:12:27] Modify product: Probable thiol peroxidase => putative thiol peroxidase
[20:12:27] Modify product: Probable endoribonuclease MazF5 => putative endoribonuclease MazF5
[20:12:27] Modify product: Probable endoribonuclease HigB1 => putative endoribonuclease HigB1
[20:12:27] Modify product: SecB-like chaperone Rv1957 => SecB-like chaperone
[20:12:27] Modify product: Probable phospholipid ABC transporter permease protein MlaE => putative phospholipid ABC transporter permease protein MlaE
[20:12:27] Modify product: Probable phospholipid ABC transporter permease protein MlaE => putative phospholipid ABC transporter permease protein MlaE
[20:12:27] Modify product: Probable phospholipid ABC transporter-binding protein MlaD => putative phospholipid ABC transporter-binding protein MlaD
[20:12:27] Modify product: Uncharacterized transporter Rv1979c => putative transporter
[20:12:27] Modify product: Probable cutinase Rv1984c => putative cutinase
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1985c => putative HTH-type transcriptional regulator
[20:12:27] Modify product: Putative short-chain type dehydrogenase/reductase MSMEG_6031/MSMEI_5872 => Putative short-chain type dehydrogenase/reductase/MSMEI_5872
[20:12:27] Modify product: Probable cation-transporting ATPase G => putative cation-transporting ATPase G
[20:12:27] Modify product: Universal stress protein Rv1996 => Universal stress protein
[20:12:27] Modify product: Probable cation-transporting ATPase F => putative cation-transporting ATPase F
[20:12:27] Modify product: Uncharacterized transporter Rv1999c => putative transporter
[20:12:27] Modify product: Universal stress protein MT2061 => Universal stress protein
[20:12:27] Modify product: Uncharacterized glycosyl hydrolase MT2062 => putative glycosyl hydrolase
[20:12:27] Modify product: Probable cation efflux system protein Rv2025c => putative cation efflux system protein
[20:12:27] Modify product: Universal stress protein Rv2026c => Universal stress protein
[20:12:27] Modify product: Universal stress protein MT2087 => Universal stress protein
[20:12:27] Modify product: Putative phosphoribosyl transferase MT0597 => Putative phosphoribosyl transferase
[20:12:27] Modify product: HTH-type transcriptional regulator Rv2034 => HTH-type transcriptional regulator
[20:12:27] Modify product: Probable endoribonuclease MazF7 => putative endoribonuclease MazF7
[20:12:27] Modify product: Uncharacterized oxidoreductase Rv2073c => putative oxidoreductase
[20:12:27] Modify product: Probable dipeptidase PepE => putative dipeptidase PepE
[20:12:27] Modify product: Probable helicase HelY => putative helicase HelY
[20:12:27] Modify product: Uncharacterized PPE family protein PPE36 => putative PPE family protein PPE36
[20:12:27] Modify product: Uncharacterized PPE family protein PPE37 => putative PPE family protein PPE37
[20:12:27] Modify product: Uncharacterized NAD-dependent oxidoreductase MAP_4146 => putative NAD-dependent oxidoreductase
[20:12:27] Modify product: Putative phosphoribosyl transferase MT0597 => Putative phosphoribosyl transferase
[20:12:27] Modify product: Laccase domain protein Rv2149c => Laccase domain protein
[20:12:27] Modify product: Probable peptidoglycan glycosyltransferase FtsW => putative peptidoglycan glycosyltransferase FtsW
[20:12:27] Modify product: DNA-binding protein Rv2175c => DNA-binding protein
[20:12:27] Modify product: 3'-5' exoribonuclease MT2234.1 => 3'-5' exoribonuclease.1
[20:12:27] Modify product: Probable endopeptidase cgR_2070 => putative endopeptidase
[20:12:27] Modify product: Probable cytochrome c oxidase subunit 3 => putative cytochrome c oxidase subunit 3
[20:12:27] Modify product: Probable transport accessory protein MmpS3 => putative transport accessory protein MmpS3
[20:12:27] Modify product: pH-sensitive adenylate cyclase Rv1264 => pH-sensitive adenylate cyclase
[20:12:27] Modify product: Probable cytosol aminopeptidase => putative cytosol aminopeptidase
[20:12:27] Modify product: Probable oxidoreductase EphD => putative oxidoreductase EphD
[20:12:27] Modify product: Epimerase family protein Rv2216 => Epimerase family protein
[20:12:27] Modify product: GTP cyclohydrolase 1 type 2 homolog => GTP cyclohydrolase 1 type 2 
[20:12:27] Modify product: Uncharacterized aminotransferase Rv2231c => putative aminotransferase
[20:12:27] Modify product: Probable low molecular weight protein-tyrosine-phosphatase => putative low molecular weight protein-tyrosine-phosphatase
[20:12:27] Modify product: Uncharacterized SURF1-like protein Rv2235 => putative SURF1-like protein
[20:12:27] Modify product: Putative peroxiredoxin Rv2238c => Putative peroxiredoxin
[20:12:27] Modify product: Probable propionyl-CoA carboxylase beta chain 6 => putative propionyl-CoA carboxylase beta chain 6
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator Rv2250c => putative HTH-type transcriptional regulator
[20:12:27] Modify product: Uncharacterized FAD-linked oxidoreductase Rv2280 => putative FAD-linked oxidoreductase
[20:12:27] Modify product: Uncharacterized FAD-linked oxidoreductase Rv2280 => putative FAD-linked oxidoreductase
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator Rv2282c => putative HTH-type transcriptional regulator
[20:12:27] Modify product: Putative diacyglycerol O-acyltransferase Rv2285 => Putative diacyglycerol O-acyltransferase
[20:12:27] Modify product: Thioredoxin-like reductase Rv2466c => Thioredoxin-like reductase
[20:12:27] Modify product: Uncharacterized Na(+)/H(+) exchanger Rv2287 => putative Na(+)/H(+) exchanger
[20:12:27] Modify product: Probable CDP-diacylglycerol pyrophosphatase => putative CDP-diacylglycerol pyrophosphatase
[20:12:27] Modify product: Uncharacterized oxidoreductase Rv2298 => putative oxidoreductase
[20:12:27] Modify product: Probable metallo-hydrolase Rv2300c => putative metallo-hydrolase
[20:12:27] Modify product: Probable cutinase cut2 => putative cutinase cut2
[20:12:27] Modify product: TVP38/TMEM64 family membrane protein Rv0625c => TVP38/TMEM64 family membrane protein
[20:12:27] Modify product: Probable ABC transporter-binding protein DR_1438 => putative ABC transporter-binding protein
[20:12:27] Modify product: Universal stress protein Rv2319c => Universal stress protein
[20:12:27] Modify product: Universal stress protein Rv2319c => Universal stress protein
[20:12:27] Modify product: Uncharacterized amino acid permease YhdG => putative amino acid permease YhdG
[20:12:27] Modify product: Uncharacterized ABC transporter ATP-binding protein Rv2326c => putative ABC transporter ATP-binding protein
[20:12:27] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0880 => putative HTH-type transcriptional regulator
[20:12:27] Modify product: Uncharacterized PE family protein PE23 => putative PE family protein PE23
[20:12:27] Modify product: Probable transport protein MmpL9 => putative transport protein MmpL9
[20:12:27] Modify product: Uncharacterized PPE family protein PPE38 => putative PPE family protein PPE38
[20:12:27] Modify product: Uncharacterized PPE family protein PPE40 => putative PPE family protein PPE40
[20:12:27] Modify product: Uncharacterized PPE family protein PPE40 => putative PPE family protein PPE40
[20:12:27] Modify product: Uncharacterized PE-PGRS family protein PE_PGRS24 => putative PE-PGRS family protein PE_PGRS24
[20:12:27] Modify product: Oxygen-independent coproporphyrinogen-III oxidase-like protein Rv2388c => Oxygen-independent coproporphyrinogen-III oxidase-like protein
[20:12:27] Modify product: Probable phosphoadenosine phosphosulfate reductase => putative phosphoadenosine phosphosulfate reductase
[20:12:27] Modify product: DegV domain-containing protein Rv2417c => DegV domain-containing protein
[20:12:27] Modify product: Probable nicotinate-nucleotide adenylyltransferase => putative nicotinate-nucleotide adenylyltransferase
[20:12:27] Modify product: Probable hydrogen peroxide-inducible genes activator => putative hydrogen peroxide-inducible genes activator
[20:12:27] Modify product: Putative trans-acting enoyl reductase Rv2449c => Putative trans-acting enoyl reductase
[20:12:27] Modify product: Probable molybdenum cofactor guanylyltransferase => putative molybdenum cofactor guanylyltransferase
[20:12:27] Modify product: Uncharacterized MFS-type transporter Rv2456c => putative MFS-type transporter
[20:12:27] Modify product: Thioredoxin-like reductase Rv2466c => Thioredoxin-like reductase
[20:12:27] Modify product: Putative diacyglycerol O-acyltransferase Rv2484c => Putative diacyglycerol O-acyltransferase
[20:12:27] Modify product: Probable enoyl-CoA hydratase echA14 => putative enoyl-CoA hydratase echA14
[20:12:27] Modify product: Putative HTH-type transcriptional regulator Rv0890c => Putative HTH-type transcriptional regulator
[20:12:27] Modify product: Putative HTH-type transcriptional regulator Rv0890c => Putative HTH-type transcriptional regulator
[20:12:27] Modify product: Probable succinyl-CoA:3-ketoacid coenzyme A transferase subunit B => putative succinyl-CoA:3-ketoacid coenzyme A transferase subunit B
[20:12:27] Modify product: Probable succinyl-CoA:3-ketoacid coenzyme A transferase subunit A => putative succinyl-CoA:3-ketoacid coenzyme A transferase subunit A
[20:12:27] Modify product: Putative peroxiredoxin Rv2521 => Putative peroxiredoxin
[20:12:27] Modify product: Putative peptidoglycan hydrolase Rv2525c => Putative peptidoglycan hydrolase
[20:12:27] Modify product: N utilization substance protein B homolog => N utilization substance protein B 
[20:12:27] Modify product: Uncharacterized AAA domain-containing protein Rv2559c => putative AAA domain-containing protein
[20:12:27] Modify product: Uncharacterized ABC transporter permease Rv2563 => putative ABC transporter permease
[20:12:27] Modify product: Uncharacterized ABC transporter ATP-binding protein Rv2564 => putative ABC transporter ATP-binding protein
[20:12:27] Modify product: Uncharacterized NTE family protein Rv2565 => putative NTE family protein
[20:12:27] Modify product: Probable peptidyl-prolyl cis-trans isomerase B => putative peptidyl-prolyl cis-trans isomerase B
[20:12:27] Modify product: Uncharacterized lipoprotein Rv2585c => putative lipoprotein
[20:12:27] Modify product: Probable transcriptional regulatory protein Rv2603c => putative transcriptional regulatory protein
[20:12:27] Modify product: Uncharacterized PPE family protein PPE42 => putative PPE family protein PPE42
[20:12:27] Modify product: Universal stress protein MT2698 => Universal stress protein
[20:12:27] Modify product: Universal stress protein MT2699 => Universal stress protein
[20:12:27] Modify product: Putative uncharacterized protein MT2703 => Putative uncharacterized protein
[20:12:27] Modify product: Probable protein archease => putative protein archease
[20:12:28] Modify product: Putative O-phosphotransferase Rv2636 => Putative O-phosphotransferase
[20:12:28] Modify product: Uncharacterized membrane protein Rv2637 => putative membrane protein
[20:12:28] Modify product: Uncharacterized N-acetyltransferase Rv2669 => putative N-acetyltransferase
[20:12:28] Modify product: Putative heme-dependent peroxidase SA0544 => Putative heme-dependent peroxidase
[20:12:28] Modify product: Probable enoyl-CoA hydratase echA12 => putative enoyl-CoA hydratase echA12
[20:12:28] Modify product: Uncharacterized transporter Rv2685 => putative transporter
[20:12:28] Modify product: Uncharacterized transporter Rv2685 => putative transporter
[20:12:28] Modify product: Uncharacterized transporter Rv2685 => putative transporter
[20:12:28] Modify product: Fluoroquinolones export permease protein Rv2686c => Fluoroquinolones export permease protein
[20:12:28] Modify product: Fluoroquinolones export permease protein Rv2687c => Fluoroquinolones export permease protein
[20:12:28] Modify product: Fluoroquinolones export ATP-binding protein Rv2688c => Fluoroquinolones export ATP-binding protein
[20:12:28] Modify product: Probable soluble pyridine nucleotide transhydrogenase => putative soluble pyridine nucleotide transhydrogenase
[20:12:28] Modify product: Uncharacterized membrane protein Rv2723 => putative membrane protein
[20:12:28] Modify product: Putative monooxygenase Rv0793 => Putative monooxygenase
[20:12:28] Modify product: Uncharacterized oxidoreductase Rv2750 => putative oxidoreductase
[20:12:28] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv0830 => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:28] Modify product: Putative monooxygenase Rv1533 => Putative monooxygenase
[20:12:28] Modify product: Uncharacterized zinc protease Rv2782c => putative zinc protease
[20:12:28] Modify product: Probable enoyl-CoA hydratase echA8 => putative enoyl-CoA hydratase echA8
[20:12:28] Modify product: Uncharacterized MFS-type transporter EfpA => putative MFS-type transporter EfpA
[20:12:28] Modify product: Acetyltransferase SACOL1063 => Acetyltransferase
[20:12:28] Modify product: Probable malate:quinone oxidoreductase => putative malate:quinone oxidoreductase
[20:12:28] Modify product: Putative glutamine amidotransferase Rv2859c => Putative glutamine amidotransferase
[20:12:28] Modify product: Probable dual-specificity RNA methyltransferase RlmN => putative dual-specificity RNA methyltransferase RlmN
[20:12:28] Modify product: Uncharacterized HTH-type transcriptional regulator Rv2887 => putative HTH-type transcriptional regulator
[20:12:28] Modify product: Uncharacterized PPE family protein PPE32 => putative PPE family protein PPE32
[20:12:28] Modify product: Uncharacterized oxidoreductase Rv2900c => putative oxidoreductase
[20:12:28] Modify product: Uncharacterized HTH-type transcriptional regulator Rv2912c => putative HTH-type transcriptional regulator
[20:12:28] Modify product: Probable thioesterase TesA => putative thioesterase TesA
[20:12:28] Modify product: Probable doxorubicin resistance ABC transporter permease protein DrrC => putative doxorubicin resistance ABC transporter permease protein DrrC
[20:12:28] Modify product: PGL/p-HBAD biosynthesis glycosyltransferase Rv2957 => PGL/p-HBAD biosynthesis glycosyltransferase
[20:12:28] Modify product: PGL/p-HBAD biosynthesis glycosyltransferase Rv2958c => PGL/p-HBAD biosynthesis glycosyltransferase
[20:12:28] Modify product: Putative permease Rv2963 => Putative permease
[20:12:28] Modify product: Uncharacterized oxidoreductase Mb2996 => putative oxidoreductase
[20:12:28] Modify product: Probable 8-oxo-dGTP diphosphatase 1 => putative 8-oxo-dGTP diphosphatase 1
[20:12:28] Modify product: DNA-binding protein HU homolog => DNA-binding protein HU 
[20:12:28] Modify product: Uncharacterized MFS-type transporter Rv2994 => putative MFS-type transporter
[20:12:28] Modify product: Uncharacterized MFS-type transporter Rv2994 => putative MFS-type transporter
[20:12:28] Modify product: Probable iron export permease protein FetB => putative iron export permease protein FetB
[20:12:28] Modify product: Probable S-adenosylmethionine-dependent methyltransferase Rv3030 => putative S-adenosylmethionine-dependent methyltransferase
[20:12:28] Modify product: Probable 1,4-alpha-glucan branching enzyme Rv3031 => putative 1,4-alpha-glucan branching enzyme
[20:12:28] Modify product: Putative acetyltransferase Rv3034c => Putative acetyltransferase
[20:12:28] Modify product: Probable enoyl-CoA hydratase echA17 => putative enoyl-CoA hydratase echA17
[20:12:28] Modify product: Uncharacterized ABC transporter ATP-binding protein YlmA => putative ABC transporter ATP-binding protein YlmA
[20:12:28] Modify product: Probable cytochrome c oxidase subunit 1 => putative cytochrome c oxidase subunit 1
[20:12:28] Modify product: Probable siderophore-binding lipoprotein YfiY => putative siderophore-binding lipoprotein YfiY
[20:12:28] Modify product: Uncharacterized HTH-type transcriptional regulator Rv0472c => putative HTH-type transcriptional regulator
[20:12:28] Modify product: Putative diacyglycerol O-acyltransferase Rv3087 => Putative diacyglycerol O-acyltransferase
[20:12:28] Modify product: Probable diacyglycerol O-acyltransferase Tgs4 => putative diacyglycerol O-acyltransferase Tgs4
[20:12:28] Modify product: Putative coenzyme F420-dependent oxidoreductase Rv3520c => Putative coenzyme F420-dependent oxidoreductase
[20:12:28] Modify product: Uncharacterized HTH-type transcriptional regulator Rv3095 => putative HTH-type transcriptional regulator
[20:12:28] Modify product: Putative toxin Rv3098A/RVBD_3098A => Putative toxinA/RVBD_3098A
[20:12:28] Modify product: Uncharacterized MscS family protein YkuT => putative MscS family protein YkuT
[20:12:28] Modify product: Uncharacterized FAD-linked oxidoreductase Rv2280 => putative FAD-linked oxidoreductase
[20:12:28] Modify product: Probable adenylyltransferase/sulfurtransferase MoeZ => putative adenylyltransferase/sulfurtransferase MoeZ
[20:12:28] Modify product: Uncharacterized PPE family protein PPE38 => putative PPE family protein PPE38
[20:12:28] Modify product: Probable diacyglycerol O-acyltransferase tgs1 => putative diacyglycerol O-acyltransferase tgs1
[20:12:28] Modify product: Putative NAD(P)H nitroreductase MT3217 => Putative NAD(P)H nitroreductase
[20:12:28] Modify product: Universal stress protein MT3220 => Universal stress protein
[20:12:28] Modify product: Uncharacterized PPE family protein PPE51 => putative PPE family protein PPE51
[20:12:28] Modify product: Uncharacterized response regulatory protein Rv3143 => putative response regulatory protein
[20:12:28] Modify product: Uncharacterized PPE family protein PPE42 => putative PPE family protein PPE42
[20:12:28] Modify product: Uncharacterized PPE family protein PPE40 => putative PPE family protein PPE40
[20:12:28] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1556 => putative HTH-type transcriptional regulator
[20:12:28] Modify product: Uncharacterized oxidoreductase SA2266 => putative oxidoreductase
[20:12:28] Modify product: Probable protein kinase UbiB => putative protein kinase UbiB
[20:12:28] Modify product: Probable transcriptional regulator WhiB7 => putative transcriptional regulator WhiB7
[20:12:28] Modify product: Putative glutaredoxin Rv3198.1 => Putative glutaredoxin.1
[20:12:28] Modify product: Probable adenylyltransferase/sulfurtransferase MoeZ => putative adenylyltransferase/sulfurtransferase MoeZ
[20:12:28] Modify product: Probable transport accessory protein MmpS3 => putative transport accessory protein MmpS3
[20:12:28] Modify product: Probable sensor histidine kinase pdtaS => putative sensor histidine kinase pdtaS
[20:12:28] Modify product: Uncharacterized NAD-dependent oxidoreductase MAP_4146 => putative NAD-dependent oxidoreductase
[20:12:28] Modify product: Putative diacyglycerol O-acyltransferase Rv1760 => Putative diacyglycerol O-acyltransferase
[20:12:28] Modify product: Probable diacyglycerol O-acyltransferase tgs3 => putative diacyglycerol O-acyltransferase tgs3
[20:12:28] Modify product: Uncharacterized NTE family protein Rv2565 => putative NTE family protein
[20:12:28] Modify product: Uncharacterized HTH-type transcriptional regulator Rv1255c => putative HTH-type transcriptional regulator
[20:12:28] Modify product: Uncharacterized amino acid permease YhdG => putative amino acid permease YhdG
[20:12:28] Modify product: Probable manganese/zinc-exporting P-type ATPase => putative manganese/zinc-exporting P-type ATPase
[20:12:28] Modify product: Probable acyl-CoA dehydrogenase fadE25 => putative acyl-CoA dehydrogenase fadE25
[20:12:28] Modify product: Probable propionyl-CoA carboxylase beta chain 5 => putative propionyl-CoA carboxylase beta chain 5
[20:12:28] Modify product: Maf-like protein Rv3282 => Maf-like protein
[20:12:28] Modify product: Uncharacterized SufE-like protein Rv3284 => putative SufE-like protein
[20:12:28] Modify product: Probable L-lysine-epsilon aminotransferase => putative L-lysine-epsilon aminotransferase
[20:12:28] Modify product: Probable ATP-dependent RNA helicase YfmL => putative ATP-dependent RNA helicase YfmL
[20:12:28] Modify product: Phosphate-specific transport system accessory protein PhoU homolog 1 => Phosphate-specific transport system accessory protein PhoU 
[20:12:28] Modify product: Uncharacterized hydrolase YxeP => putative hydrolase YxeP
[20:12:28] Modify product: Probable phosphomannomutase => putative phosphomannomutase
[20:12:28] Modify product: Probable aminotransferase Rv3329 => putative aminotransferase
[20:12:28] Modify product: Uncharacterized methyltransferase Rv3342 => putative methyltransferase
[20:12:28] Modify product: Bifunctional protein FolD => Bifunctional protein FolD protein
[20:12:28] Modify product: ABC transporter ATP-binding/permease protein Rv1747 => ABC transporter ATP-binding/permease protein
[20:12:28] Modify product: Serine protease inhibitor Rv3364c => Serine protease inhibitor
[20:12:28] Modify product: Putative diacyglycerol O-acyltransferase Rv3371 => Putative diacyglycerol O-acyltransferase
[20:12:28] Modify product: Phosphatase Rv3376 => Phosphatase
[20:12:28] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv3399 => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:28] Modify product: Uncharacterized glycosyl hydrolase MT2062 => putative glycosyl hydrolase
[20:12:28] Modify product: Uncharacterized glycosyl hydrolase Rv3401 => putative glycosyl hydrolase
[20:12:28] Modify product: HTH-type transcriptional repressor Rv3405c => HTH-type transcriptional repressor
[20:12:28] Modify product: Uncharacterized oxidoreductase Rv3410c => putative oxidoreductase
[20:12:28] Modify product: Probable cutinase cut3 => putative cutinase cut3
[20:12:28] Modify product: Probable cutinase cut3 => putative cutinase cut3
[20:12:28] Modify product: Uncharacterized membrane protein Rv3479 => putative membrane protein
[20:12:28] Modify product: Putative diacyglycerol O-acyltransferase Rv3480c => Putative diacyglycerol O-acyltransferase
[20:12:28] Modify product: Uncharacterized oxidoreductase Rv0769 => putative oxidoreductase
[20:12:28] Modify product: Uncharacterized membrane protein Rv3486 => putative membrane protein
[20:12:28] Modify product: Probable phospholipid ABC transporter permease protein MlaE => putative phospholipid ABC transporter permease protein MlaE
[20:12:28] Modify product: Probable phospholipid ABC transporter permease protein MlaE => putative phospholipid ABC transporter permease protein MlaE
[20:12:28] Modify product: Putative short-chain type dehydrogenase/reductase Rv0148 => Putative short-chain type dehydrogenase/reductase
[20:12:28] Modify product: Putative coenzyme F420-dependent oxidoreductase Rv3520c => Putative coenzyme F420-dependent oxidoreductase
[20:12:28] Modify product: Uncharacterized PPE family protein PPE51 => putative PPE family protein PPE51
[20:12:28] Modify product: Uncharacterized PPE family protein PPE40 => putative PPE family protein PPE40
[20:12:28] Modify product: Putative short-chain type dehydrogenase/reductase Rv0148 => Putative short-chain type dehydrogenase/reductase
[20:12:28] Modify product: Probable 2,4-dienoyl-CoA reductase => putative 2,4-dienoyl-CoA reductase
[20:12:28] Modify product: Putative CoA-transferase subunit alpha Rv3551 => Putative CoA-transferase subunit alpha
[20:12:28] Modify product: Putative CoA-transferase subunit beta Rv3552 => Putative CoA-transferase subunit beta
[20:12:28] Modify product: Uncharacterized PPE family protein PPE40 => putative PPE family protein PPE40
[20:12:28] Modify product: Uncharacterized oxidoreductase Rv1350 => putative oxidoreductase
[20:12:28] Modify product: Uncharacterized tRNA/rRNA methyltransferase Rv3579c => putative tRNA/rRNA methyltransferase
[20:12:28] Modify product: Uncharacterized PPE family protein PPE32 => putative PPE family protein PPE32
[20:12:28] Modify product: Probable adenosine monophosphate-protein transferase fic => putative adenosine monophosphate-protein transferase fic
[20:12:28] Modify product: Probable cold shock protein A => putative cold shock protein A
[20:12:28] Modify product: Uncharacterized PE-PGRS family protein PE_PGRS24 => putative PE-PGRS family protein PE_PGRS24
[20:12:28] Modify product: Apoptosis inhibitor Rv3654c => Apoptosis inhibitor
[20:12:28] Modify product: Apoptosis inhibitor Rv3655c => Apoptosis inhibitor
[20:12:28] Modify product: Putative conjugal transfer protein Rv3659c => Putative conjugal transfer protein
[20:12:28] Modify product: Probable phosphatase Rv3661 => putative phosphatase
[20:12:28] Modify product: Serine protease Rv3671c => Serine protease
[20:12:28] Modify product: Probable polyketide biosynthesis zinc-dependent hydrolase PksB => putative polyketide biosynthesis zinc-dependent hydrolase PksB
[20:12:28] Modify product: Putative ATPase Rv3679 => Putative ATPase
[20:12:28] Modify product: Probable siderophore biosynthesis protein SbnA => putative siderophore biosynthesis protein SbnA
[20:12:28] Modify product: Uncharacterized membrane protein Rv3691 => putative membrane protein
[20:12:28] Modify product: Probable hercynylcysteine sulfoxide lyase => putative hercynylcysteine sulfoxide lyase
[20:12:28] Modify product: Nucleoid-associated protein Rv3716c => Nucleoid-associated protein
[20:12:28] Modify product: N-acetylmuramoyl-L-alanine amidase Rv3717 => N-acetylmuramoyl-L-alanine amidase
[20:12:28] Modify product: Probable fatty acid methyltransferase Rv3720 => putative fatty acid methyltransferase
[20:12:28] Modify product: Putative aminotransferase MSMEG_6286/MSMEI_6121 => Putative aminotransferase/MSMEI_6121
[20:12:28] Modify product: Probable cutinase Rv1984c => putative cutinase
[20:12:28] Modify product: Uncharacterized NTE family protein Rv2565 => putative NTE family protein
[20:12:28] Modify product: Probable diacyglycerol O-acyltransferase tgs2 => putative diacyglycerol O-acyltransferase tgs2
[20:12:28] Modify product: Uncharacterized PPE family protein PPE47/PPE48 => putative PPE family protein PPE47/PPE48
[20:12:28] Modify product: Putative diacyglycerol O-acyltransferase Rv3740c => Putative diacyglycerol O-acyltransferase
[20:12:28] Modify product: Probable cation-transporting P-type ATPase J => putative cation-transporting P-type ATPase J
[20:12:28] Modify product: Uncharacterized membrane protein Rv3760 => putative membrane protein
[20:12:28] Modify product: Probable sensor histidine kinase TcrY => putative sensor histidine kinase TcrY
[20:12:28] Modify product: Probable transcriptional regulatory protein TcrX => putative transcriptional regulatory protein TcrX
[20:12:28] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv3767c => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:28] Modify product: Probable enoyl-CoA hydratase echA8 => putative enoyl-CoA hydratase echA8
[20:12:28] Modify product: Putative S-adenosyl-L-methionine-dependent methyltransferase Rv3787c => Putative S-adenosyl-L-methionine-dependent methyltransferase
[20:12:28] Modify product: Arabinogalactan biosynthesis recruiting protein Rv3789 => Arabinogalactan biosynthesis recruiting protein
[20:12:28] Modify product: Probable arabinosyltransferase C => putative arabinosyltransferase C
[20:12:28] Modify product: Probable arabinosyltransferase A => putative arabinosyltransferase A
[20:12:28] Modify product: Probable arabinosyltransferase B => putative arabinosyltransferase B
[20:12:28] Modify product: Probable propionyl-CoA carboxylase beta chain 5 => putative propionyl-CoA carboxylase beta chain 5
[20:12:29] Modify product: Putative decaprenylphosphoryl-5-phosphoribose phosphatase Rv3807c => Putative decaprenylphosphoryl-5-phosphoribose phosphatase
[20:12:29] Modify product: Putative Rieske 2Fe-2S iron-sulfur protein Rv3818 => Putative Rieske 2Fe-2S iron-sulfur protein
[20:12:29] Modify product: Probable long-chain-fatty-acid--CoA ligase FadD23 => putative long-chain-fatty-acid--CoA ligase FadD23
[20:12:29] Modify product: Probable S-adenosylmethionine-dependent methyltransferase MSMEG_2350/MSMEI_2290 => putative S-adenosylmethionine-dependent methyltransferase/MSMEI_2290
[20:12:29] Modify product: Uncharacterized membrane protein Rv3835 => putative membrane protein
[20:12:29] Modify product: Probable glycerophosphodiester phosphodiesterase 1 => putative glycerophosphodiester phosphodiesterase 1
[20:12:29] Modify product: Probable transcriptional regulator WhiB6 => putative transcriptional regulator WhiB6
[20:12:29] Modify product: Uncharacterized PPE family protein PPE40 => putative PPE family protein PPE40
[20:12:29] Modify product: ABC transporter ATP-binding/permease protein Rv1747 => ABC transporter ATP-binding/permease protein
[20:12:29] Modify product: Uncharacterized PPE family protein PPE32 => putative PPE family protein PPE32
[20:12:29] Modify product: Uncharacterized PPE family protein PPE47/PPE48 => putative PPE family protein PPE47/PPE48
[20:12:29] Cleaned 533 /product names
[20:12:29] Deleting unwanted file: ./G04878_prokka/sprot.faa
[20:12:29] Deleting unwanted file: ./G04878_prokka/sprot.blast
[20:12:30] There are still 1697 unannotated CDS left (started with 4276)
[20:12:30] Will use hmmer3 to search against /opt/prokka/bin/../db/hmm/HAMAP.hmm with 1 CPUs
[20:12:30] Running: cat \.\/G04878_prokka\/HAMAP\.hmm\.faa | parallel --gnu --plain -j 1 --block 174879 --recstart '>' --pipe hmmscan --noali --notextw --acc -E 1e-06 --cpu 1 /opt/prokka/bin/../db/hmm/HAMAP.hmm /dev/stdin > \.\/G04878_prokka\/HAMAP\.hmm\.hmmer3 2> /dev/null
[20:13:33] Modify product: Uncharacterized Nudix hydrolase NudL => putative Nudix hydrolase NudL
[20:13:33] Cleaned 1 /product names
[20:13:33] Deleting unwanted file: ./G04878_prokka/HAMAP.hmm.faa
[20:13:33] Deleting unwanted file: ./G04878_prokka/HAMAP.hmm.hmmer3
[20:13:33] Labelling remaining 1640 proteins as 'hypothetical protein'
[20:13:33] Possible /pseudo 'putative HTH-type transcriptional regulator' at ordered_NC000962_3 position 74174
[20:13:33] Possible /pseudo 'Putative S-adenosyl-L-methionine-dependent methyltransferase' at ordered_NC000962_3 position 199740
[20:13:33] Possible /pseudo 'PE family protein PE3' at ordered_NC000962_3 position 206848
[20:13:33] Possible /pseudo 'PE family protein PE3' at ordered_NC000962_3 position 216460
[20:13:33] Possible /pseudo 'putative phospholipid ABC transporter permease protein MlaE' at ordered_NC000962_3 position 225191
[20:13:33] Possible /pseudo 'Putative diacyglycerol O-acyltransferase' at ordered_NC000962_3 position 292755
[20:13:33] Possible /pseudo 'PPE family protein PPE34' at ordered_NC000962_3 position 401771
[20:13:33] Possible /pseudo 'L-asparagine permease 2' at ordered_NC000962_3 position 445760
[20:13:33] Possible /pseudo 'Carbon monoxide dehydrogenase medium chain' at ordered_NC000962_3 position 482215
[20:13:33] Possible /pseudo 'Sarcosine/dimethylglycine N-methyltransferase' at ordered_NC000962_3 position 643094
[20:13:33] Possible /pseudo 'putative phospholipid ABC transporter permease protein MlaE' at ordered_NC000962_3 position 716671
[20:13:33] Possible /pseudo 'Putative S-adenosyl-L-methionine-dependent methyltransferase' at ordered_NC000962_3 position 850335
[20:13:33] Possible /pseudo 'Para-nitrobenzyl esterase' at ordered_NC000962_3 position 1265773
[20:13:33] Possible /pseudo '1,2-epoxyphenylacetyl-CoA isomerase' at ordered_NC000962_3 position 1302738
[20:13:33] Possible /pseudo 'Nitrate reductase-like protein NarX' at ordered_NC000962_3 position 1327748
[20:13:33] Possible /pseudo 'putative ABC transporter ATP-binding protein' at ordered_NC000962_3 position 1457312
[20:13:33] Possible /pseudo 'Putative peptide transport permease protein' at ordered_NC000962_3 position 1471150
[20:13:33] Possible /pseudo 'Adenylate cyclase 2' at ordered_NC000962_3 position 1516710
[20:13:33] Possible /pseudo 'Adenylate cyclase 2' at ordered_NC000962_3 position 1518327
[20:13:33] Possible /pseudo 'Carboxylesterase NlhH' at ordered_NC000962_3 position 1612059
[20:13:33] Possible /pseudo 'Putative thioredoxin 2' at ordered_NC000962_3 position 1695882
[20:13:33] Possible /pseudo 'putative glycosyltransferase' at ordered_NC000962_3 position 1755969
[20:13:33] Possible /pseudo 'putative oxidoreductase' at ordered_NC000962_3 position 1783546
[20:13:33] Possible /pseudo 'Phenolphthiocerol synthesis polyketide synthase type I Pks15/1' at ordered_NC000962_3 position 1922068
[20:13:33] Possible /pseudo 'PE-PGRS family protein PE_PGRS33' at ordered_NC000962_3 position 2103476
[20:13:33] Possible /pseudo 'Haloalkane dehalogenase 2' at ordered_NC000962_3 position 2121850
[20:13:33] Possible /pseudo 'Hemolysin C' at ordered_NC000962_3 position 2132648
[20:13:33] Possible /pseudo 'Putative S-adenosyl-L-methionine-dependent methyltransferase' at ordered_NC000962_3 position 2180612
[20:13:33] Possible /pseudo 'Putative monooxygenase' at ordered_NC000962_3 position 2183349
[20:13:33] Possible /pseudo 'putative phospholipid ABC transporter permease protein MlaE' at ordered_NC000962_3 position 2248489
[20:13:33] Possible /pseudo 'Lipoprotein LprN' at ordered_NC000962_3 position 2255239
[20:13:33] Possible /pseudo 'Nicotinamidase/pyrazinamidase' at ordered_NC000962_3 position 2338768
[20:13:33] Possible /pseudo 'Precorrin-6A reductase' at ordered_NC000962_3 position 2378821
[20:13:33] Possible /pseudo 'Long-chain-fatty-acid--CoA ligase FadD15' at ordered_NC000962_3 position 2500861
[20:13:33] Possible /pseudo 'Universal stress protein' at ordered_NC000962_3 position 2645237
[20:13:33] Possible /pseudo 'putative PPE family protein PPE40' at ordered_NC000962_3 position 2691531
[20:13:33] Possible /pseudo 'Putative HTH-type transcriptional regulator' at ordered_NC000962_3 position 2852129
[20:13:33] Possible /pseudo 'Putative lipoprotein LppA' at ordered_NC000962_3 position 2922142
[20:13:33] Possible /pseudo 'Universal stress protein' at ordered_NC000962_3 position 3005339
[20:13:33] Possible /pseudo 'putative transporter' at ordered_NC000962_3 position 3056956
[20:13:33] Possible /pseudo 'putative transporter' at ordered_NC000962_3 position 3057580
[20:13:33] Possible /pseudo 'Fluoroquinolones export permease protein' at ordered_NC000962_3 position 3059008
[20:13:33] Possible /pseudo 'Trk system potassium uptake protein TrkA' at ordered_NC000962_3 position 3064998
[20:13:33] Possible /pseudo 'PGL/p-HBAD biosynthesis rhamnosyltransferase' at ordered_NC000962_3 position 3372425
[20:13:33] Possible /pseudo 'putative MFS-type transporter' at ordered_NC000962_3 position 3410135
[20:13:33] Possible /pseudo 'Acyl-CoA dehydrogenase' at ordered_NC000962_3 position 3566372
[20:13:33] Possible /pseudo 'Putative flavin-containing monoamine oxidase AofH' at ordered_NC000962_3 position 3601761
[20:13:33] Possible /pseudo 'putative glycosyl hydrolase' at ordered_NC000962_3 position 3881760
[20:13:33] Possible /pseudo 'putative cutinase cut3' at ordered_NC000962_3 position 3937569
[20:13:33] Possible /pseudo 'putative phospholipid ABC transporter permease protein MlaE' at ordered_NC000962_3 position 3984377
[20:13:33] Possible /pseudo 'ATP-dependent Clp protease ATP-binding subunit ClpC1' at ordered_NC000962_3 position 4103435
[20:13:33] Possible /pseudo 'Apoptosis inhibitor' at ordered_NC000962_3 position 4159689
[20:13:33] Possible /pseudo '1-acyl-sn-glycerol-3-phosphate acyltransferase' at ordered_NC000962_3 position 4345013
[20:13:33] Possible /pseudo '1-acyl-sn-glycerol-3-phosphate acyltransferase' at ordered_NC000962_3 position 4345772
[20:13:33] Possible /pseudo 'ESX-1 secretion-associated protein EspJ' at ordered_NC000962_3 position 4422297
[20:13:33] Possible /pseudo 'ESX-1 secretion-associated protein EspK' at ordered_NC000962_3 position 4424498
[20:13:33] Possible /pseudo 'ESX-1 secretion-associated protein EspB' at ordered_NC000962_3 position 4426469
[20:13:34] Found 1651 unique /gene codes.
[20:13:34] Fixed 2 duplicate /gene - dapE_1 dapE_2
[20:13:34] Fixed 4 duplicate /gene - fgd2_1 fgd2_2 fgd2_3 fgd2_4
[20:13:34] Fixed 2 duplicate /gene - fbiB_1 fbiB_2
[20:13:34] Fixed 2 duplicate /gene - bacC_1 bacC_2
[20:13:34] Fixed 3 duplicate /gene - tcrX_1 tcrX_2 tcrX_3
[20:13:34] Fixed 2 duplicate /gene - pyrE_1 pyrE_2
[20:13:34] Fixed 2 duplicate /gene - pbpB_1 pbpB_2
[20:13:34] Fixed 2 duplicate /gene - tesB_1 tesB_2
[20:13:34] Fixed 3 duplicate /gene - tcrY_1 tcrY_2 tcrY_3
[20:13:34] Fixed 2 duplicate /gene - ansP2_1 ansP2_2
[20:13:34] Fixed 2 duplicate /gene - hsaA_1 hsaA_2
[20:13:34] Fixed 2 duplicate /gene - esxJ_1 esxJ_2
[20:13:34] Fixed 2 duplicate /gene - papA2_1 papA2_2
[20:13:34] Fixed 2 duplicate /gene - pra_1 pra_2
[20:13:34] Fixed 2 duplicate /gene - eccB3_1 eccB3_2
[20:13:34] Fixed 2 duplicate /gene - rpsR1_1 rpsR1_2
[20:13:34] Fixed 2 duplicate /gene - htpX_1 htpX_2
[20:13:34] Fixed 3 duplicate /gene - fabG_1 fabG_2 fabG_3
[20:13:34] Fixed 2 duplicate /gene - dnaB_1 dnaB_2
[20:13:34] Fixed 2 duplicate /gene - tetC_1 tetC_2
[20:13:34] Fixed 3 duplicate /gene - clpC1_1 clpC1_2 clpC1_3
[20:13:34] Fixed 2 duplicate /gene - higB2_1 higB2_2
[20:13:34] Fixed 3 duplicate /gene - dhmA2_1 dhmA2_2 dhmA2_3
[20:13:34] Fixed 2 duplicate /gene - dinB1_1 dinB1_2
[20:13:34] Fixed 2 duplicate /gene - arsR1_1 arsR1_2
[20:13:34] Fixed 2 duplicate /gene - csoR_1 csoR_2
[20:13:34] Fixed 2 duplicate /gene - PE5_1 PE5_2
[20:13:34] Fixed 2 duplicate /gene - vapB5_1 vapB5_2
[20:13:34] Fixed 3 duplicate /gene - inhA_1 inhA_2 inhA_3
[20:13:34] Fixed 2 duplicate /gene - trxC_1 trxC_2
[20:13:34] Fixed 2 duplicate /gene - galE_1 galE_2
[20:13:34] Fixed 3 duplicate /gene - atsA_1 atsA_2 atsA_3
[20:13:34] Fixed 2 duplicate /gene - gmd_1 gmd_2
[20:13:34] Fixed 2 duplicate /gene - lprE_1 lprE_2
[20:13:34] Fixed 2 duplicate /gene - nicC_1 nicC_2
[20:13:34] Fixed 2 duplicate /gene - fbpA_1 fbpA_2
[20:13:34] Fixed 2 duplicate /gene - hyfB_1 hyfB_2
[20:13:34] Fixed 2 duplicate /gene - secA_1 secA_2
[20:13:34] Fixed 2 duplicate /gene - lpdA_1 lpdA_2
[20:13:34] Fixed 2 duplicate /gene - pucR_1 pucR_2
[20:13:34] Fixed 3 duplicate /gene - narK_1 narK_2 narK_3
[20:13:34] Fixed 3 duplicate /gene - sugC_1 sugC_2 sugC_3
[20:13:34] Fixed 2 duplicate /gene - lnt_1 lnt_2
[20:13:34] Fixed 2 duplicate /gene - bmr3_1 bmr3_2
[20:13:34] Fixed 2 duplicate /gene - dmdC_1 dmdC_2
[20:13:34] Fixed 2 duplicate /gene - gcvT_1 gcvT_2
[20:13:34] Fixed 2 duplicate /gene - msrR_1 msrR_2
[20:13:34] Fixed 2 duplicate /gene - cya_1 cya_2
[20:13:34] Fixed 2 duplicate /gene - walR_1 walR_2
[20:13:34] Fixed 2 duplicate /gene - frdB_1 frdB_2
[20:13:34] Fixed 2 duplicate /gene - nfdA_1 nfdA_2
[20:13:34] Fixed 2 duplicate /gene - ahpD_1 ahpD_2
[20:13:34] Fixed 3 duplicate /gene - rsfA_1 rsfA_2 rsfA_3
[20:13:34] Fixed 2 duplicate /gene - ideR_1 ideR_2
[20:13:34] Fixed 2 duplicate /gene - espK_1 espK_2
[20:13:34] Fixed 3 duplicate /gene - dacB_1 dacB_2 dacB_3
[20:13:34] Fixed 3 duplicate /gene - lprJ_1 lprJ_2 lprJ_3
[20:13:34] Fixed 2 duplicate /gene - ctpI_1 ctpI_2
[20:13:34] Fixed 2 duplicate /gene - caeB_1 caeB_2
[20:13:34] Fixed 2 duplicate /gene - pgsA1_1 pgsA1_2
[20:13:34] Fixed 2 duplicate /gene - moaA1_1 moaA1_2
[20:13:34] Fixed 2 duplicate /gene - pknH_1 pknH_2
[20:13:34] Fixed 2 duplicate /gene - espB_1 espB_2
[20:13:34] Fixed 3 duplicate /gene - narX_1 narX_2 narX_3
[20:13:34] Fixed 4 duplicate /gene - espI_1 espI_2 espI_3 espI_4
[20:13:34] Fixed 3 duplicate /gene - mmpS4_1 mmpS4_2 mmpS4_3
[20:13:34] Fixed 8 duplicate /gene - mlaE_1 mlaE_2 mlaE_3 mlaE_4 mlaE_5 mlaE_6 mlaE_7 mlaE_8
[20:13:34] Fixed 2 duplicate /gene - rpmG2_1 rpmG2_2
[20:13:34] Fixed 2 duplicate /gene - paaG_1 paaG_2
[20:13:34] Fixed 2 duplicate /gene - bicA_1 bicA_2
[20:13:34] Fixed 2 duplicate /gene - ethR_1 ethR_2
[20:13:34] Fixed 3 duplicate /gene - ligD_1 ligD_2 ligD_3
[20:13:34] Fixed 2 duplicate /gene - ppsA_1 ppsA_2
[20:13:34] Fixed 2 duplicate /gene - ftsH_1 ftsH_2
[20:13:34] Fixed 2 duplicate /gene - menG_1 menG_2
[20:13:34] Fixed 4 duplicate /gene - plsC_1 plsC_2 plsC_3 plsC_4
[20:13:34] Fixed 2 duplicate /gene - ubiE_1 ubiE_2
[20:13:34] Fixed 2 duplicate /gene - moaE2_1 moaE2_2
[20:13:34] Fixed 2 duplicate /gene - resA_1 resA_2
[20:13:34] Fixed 3 duplicate /gene - sadH_1 sadH_2 sadH_3
[20:13:34] Fixed 2 duplicate /gene - qorA_1 qorA_2
[20:13:34] Fixed 3 duplicate /gene - trxA_1 trxA_2 trxA_3
[20:13:34] Fixed 2 duplicate /gene - dagK_1 dagK_2
[20:13:34] Fixed 2 duplicate /gene - pimB_1 pimB_2
[20:13:34] Fixed 2 duplicate /gene - aspC_1 aspC_2
[20:13:34] Fixed 2 duplicate /gene - virS_1 virS_2
[20:13:34] Fixed 2 duplicate /gene - gpmA_1 gpmA_2
[20:13:34] Fixed 2 duplicate /gene - adhB_1 adhB_2
[20:13:34] Fixed 2 duplicate /gene - ripA_1 ripA_2
[20:13:34] Fixed 4 duplicate /gene - cyaB_1 cyaB_2 cyaB_3 cyaB_4
[20:13:34] Fixed 2 duplicate /gene - eccD1_1 eccD1_2
[20:13:34] Fixed 2 duplicate /gene - srpR_1 srpR_2
[20:13:34] Fixed 2 duplicate /gene - tadA_1 tadA_2
[20:13:34] Fixed 2 duplicate /gene - uvrD1_1 uvrD1_2
[20:13:34] Fixed 3 duplicate /gene - pnbA_1 pnbA_2 pnbA_3
[20:13:34] Fixed 2 duplicate /gene - cobQ_1 cobQ_2
[20:13:34] Fixed 2 duplicate /gene - aofH_1 aofH_2
[20:13:34] Fixed 2 duplicate /gene - estB_1 estB_2
[20:13:34] Fixed 2 duplicate /gene - tuf_1 tuf_2
[20:13:34] Fixed 2 duplicate /gene - eccC5_1 eccC5_2
[20:13:34] Fixed 2 duplicate /gene - dxs_1 dxs_2
[20:13:34] Fixed 3 duplicate /gene - accD5_1 accD5_2 accD5_3
[20:13:34] Fixed 2 duplicate /gene - embR_1 embR_2
[20:13:34] Fixed 3 duplicate /gene - uctC_1 uctC_2 uctC_3
[20:13:34] Fixed 2 duplicate /gene - pknD_1 pknD_2
[20:13:34] Fixed 4 duplicate /gene - pcaF_1 pcaF_2 pcaF_3 pcaF_4
[20:13:34] Fixed 2 duplicate /gene - pip_1 pip_2
[20:13:34] Fixed 2 duplicate /gene - aidB_1 aidB_2
[20:13:34] Fixed 2 duplicate /gene - yhdG_1 yhdG_2
[20:13:34] Fixed 5 duplicate /gene - crt_1 crt_2 crt_3 crt_4 crt_5
[20:13:34] Fixed 2 duplicate /gene - ettA_1 ettA_2
[20:13:34] Fixed 2 duplicate /gene - ddn_1 ddn_2
[20:13:34] Fixed 2 duplicate /gene - mutB_1 mutB_2
[20:13:34] Fixed 2 duplicate /gene - noeI_1 noeI_2
[20:13:34] Fixed 2 duplicate /gene - sdhA_1 sdhA_2
[20:13:34] Fixed 5 duplicate /gene - rebM_1 rebM_2 rebM_3 rebM_4 rebM_5
[20:13:34] Fixed 2 duplicate /gene - vapC5_1 vapC5_2
[20:13:34] Fixed 2 duplicate /gene - ord_1 ord_2
[20:13:34] Fixed 2 duplicate /gene - serB1_1 serB1_2
[20:13:34] Fixed 2 duplicate /gene - rcdA_1 rcdA_2
[20:13:34] Fixed 3 duplicate /gene - lipF_1 lipF_2 lipF_3
[20:13:34] Fixed 2 duplicate /gene - recD_1 recD_2
[20:13:34] Fixed 2 duplicate /gene - vapC9_1 vapC9_2
[20:13:34] Fixed 2 duplicate /gene - mmpL8_1 mmpL8_2
[20:13:34] Fixed 3 duplicate /gene - hsaB_1 hsaB_2 hsaB_3
[20:13:34] Fixed 2 duplicate /gene - gppA_1 gppA_2
[20:13:34] Fixed 2 duplicate /gene - sbnA_1 sbnA_2
[20:13:34] Fixed 3 duplicate /gene - ethA_1 ethA_2 ethA_3
[20:13:34] Fixed 3 duplicate /gene - fgd1_1 fgd1_2 fgd1_3
[20:13:34] Fixed 2 duplicate /gene - mmsB_1 mmsB_2
[20:13:34] Fixed 2 duplicate /gene - betI_1 betI_2
[20:13:34] Fixed 2 duplicate /gene - fabG2_1 fabG2_2
[20:13:34] Fixed 4 duplicate /gene - lipY_1 lipY_2 lipY_3 lipY_4
[20:13:34] Fixed 2 duplicate /gene - ubiB_1 ubiB_2
[20:13:34] Fixed 2 duplicate /gene - tetR_1 tetR_2
[20:13:34] Fixed 2 duplicate /gene - trkA_1 trkA_2
[20:13:34] Fixed 3 duplicate /gene - rpmB_1 rpmB_2 rpmB_3
[20:13:34] Fixed 2 duplicate /gene - mca_1 mca_2
[20:13:34] Fixed 2 duplicate /gene - rbn_1 rbn_2
[20:13:34] Fixed 7 duplicate /gene - nlhH_1 nlhH_2 nlhH_3 nlhH_4 nlhH_5 nlhH_6 nlhH_7
[20:13:34] Fixed 3 duplicate /gene - drrA_1 drrA_2 drrA_3
[20:13:34] Fixed 2 duplicate /gene - acg_1 acg_2
[20:13:34] Fixed 2 duplicate /gene - xerC_1 xerC_2
[20:13:34] Fixed 2 duplicate /gene - epsE_1 epsE_2
[20:13:34] Fixed 3 duplicate /gene - echA8_1 echA8_2 echA8_3
[20:13:34] Fixed 2 duplicate /gene - cobB_1 cobB_2
[20:13:34] Fixed 2 duplicate /gene - ribBA_1 ribBA_2
[20:13:34] Fixed 3 duplicate /gene - ephA_1 ephA_2 ephA_3
[20:13:34] Fixed 2 duplicate /gene - esxR_1 esxR_2
[20:13:34] Fixed 2 duplicate /gene - dprE1_1 dprE1_2
[20:13:34] Fixed 2 duplicate /gene - cobK_1 cobK_2
[20:13:34] Fixed 2 duplicate /gene - hepT_1 hepT_2
[20:13:34] Fixed 3 duplicate /gene - bpoC_1 bpoC_2 bpoC_3
[20:13:34] Fixed 2 duplicate /gene - vapC_1 vapC_2
[20:13:34] Fixed 2 duplicate /gene - accA1_1 accA1_2
[20:13:34] Fixed 3 duplicate /gene - soj_1 soj_2 soj_3
[20:13:34] Fixed 2 duplicate /gene - acrC_1 acrC_2
[20:13:34] Fixed 2 duplicate /gene - dhbF_1 dhbF_2
[20:13:34] Fixed 4 duplicate /gene - pspA_1 pspA_2 pspA_3 pspA_4
[20:13:34] Fixed 2 duplicate /gene - caiD_1 caiD_2
[20:13:34] Fixed 2 duplicate /gene - sap_1 sap_2
[20:13:34] Fixed 2 duplicate /gene - chp1_1 chp1_2
[20:13:34] Fixed 5 duplicate /gene - acr1_1 acr1_2 acr1_3 acr1_4 acr1_5
[20:13:34] Fixed 2 duplicate /gene - PPE2_1 PPE2_2
[20:13:34] Fixed 2 duplicate /gene - tlyC_1 tlyC_2
[20:13:34] Fixed 2 duplicate /gene - tam_1 tam_2
[20:13:34] Fixed 3 duplicate /gene - oatA_1 oatA_2 oatA_3
[20:13:34] Fixed 3 duplicate /gene - araQ_1 araQ_2 araQ_3
[20:13:34] Fixed 2 duplicate /gene - ugpA_1 ugpA_2
[20:13:34] Fixed 5 duplicate /gene - PE3_1 PE3_2 PE3_3 PE3_4 PE3_5
[20:13:34] Fixed 2 duplicate /gene - oxyR_1 oxyR_2
[20:13:34] Fixed 2 duplicate /gene - mshD_1 mshD_2
[20:13:34] Fixed 2 duplicate /gene - cmoB_1 cmoB_2
[20:13:34] Fixed 7 duplicate /gene - kstR2_1 kstR2_2 kstR2_3 kstR2_4 kstR2_5 kstR2_6 kstR2_7
[20:13:34] Fixed 2 duplicate /gene - fadD3_1 fadD3_2
[20:13:34] Fixed 2 duplicate /gene - greA_1 greA_2
[20:13:34] Fixed 2 duplicate /gene - ephD_1 ephD_2
[20:13:34] Fixed 2 duplicate /gene - pknE_1 pknE_2
[20:13:34] Fixed 5 duplicate /gene - lprN_1 lprN_2 lprN_3 lprN_4 lprN_5
[20:13:34] Fixed 7 duplicate /gene - mmgC_1 mmgC_2 mmgC_3 mmgC_4 mmgC_5 mmgC_6 mmgC_7
[20:13:34] Fixed 2 duplicate /gene - mmpS3_1 mmpS3_2
[20:13:34] Fixed 3 duplicate /gene - lppA_1 lppA_2 lppA_3
[20:13:34] Fixed 2 duplicate /gene - lrp_1 lrp_2
[20:13:34] Fixed 2 duplicate /gene - yedY_1 yedY_2
[20:13:34] Fixed 2 duplicate /gene - argO_1 argO_2
[20:13:34] Fixed 2 duplicate /gene - vapC3_1 vapC3_2
[20:13:34] Fixed 2 duplicate /gene - priA_1 priA_2
[20:13:34] Fixed 2 duplicate /gene - ubiG_1 ubiG_2
[20:13:34] Fixed 2 duplicate /gene - higA2_1 higA2_2
[20:13:34] Fixed 2 duplicate /gene - mmpL5_1 mmpL5_2
[20:13:34] Fixed 2 duplicate /gene - pbpE_1 pbpE_2
[20:13:34] Fixed 2 duplicate /gene - moaE1_1 moaE1_2
[20:13:34] Fixed 2 duplicate /gene - fdxA_1 fdxA_2
[20:13:34] Fixed 2 duplicate /gene - stp_1 stp_2
[20:13:34] Fixed 2 duplicate /gene - murF_1 murF_2
[20:13:34] Fixed 2 duplicate /gene - hspX_1 hspX_2
[20:13:34] Fixed 2 duplicate /gene - fadA_1 fadA_2
[20:13:34] Fixed 2 duplicate /gene - icaR_1 icaR_2
[20:13:34] Fixed 2 duplicate /gene - pncA_1 pncA_2
[20:13:34] Fixed 2 duplicate /gene - lysX_1 lysX_2
[20:13:34] Fixed 2 duplicate /gene - mmpL3_1 mmpL3_2
[20:13:34] Fixed 2 duplicate /gene - cocE_1 cocE_2
[20:13:34] Fixed 4 duplicate /gene - acdA_1 acdA_2 acdA_3 acdA_4
[20:13:34] Fixed 2 duplicate /gene - moaC1_1 moaC1_2
[20:13:34] Fixed 2 duplicate /gene - espJ_1 espJ_2
[20:13:34] Fixed 2 duplicate /gene - cut3_1 cut3_2
[20:13:34] Fixed 2 duplicate /gene - fadH_1 fadH_2
[20:13:34] Fixed 3 duplicate /gene - gloC_1 gloC_2 gloC_3
[20:13:34] Fixed 2 duplicate /gene - dnaX_1 dnaX_2
[20:13:34] Fixed 2 duplicate /gene - nei1_1 nei1_2
[20:13:34] Fixed 2 duplicate /gene - hypB_1 hypB_2
[20:13:34] Fixed 2 duplicate /gene - menA_1 menA_2
[20:13:34] Fixed 3 duplicate /gene - moeZ_1 moeZ_2 moeZ_3
[20:13:34] Fixed 4 duplicate /gene - lcfB_1 lcfB_2 lcfB_3 lcfB_4
[20:13:34] Fixed 4 duplicate /gene - cpo_1 cpo_2 cpo_3 cpo_4
[20:13:34] Fixed 2 duplicate /gene - mtnN_1 mtnN_2
[20:13:34] Fixed 216 colliding /gene names.
[20:13:34] Adding /locus_tag identifiers
[20:13:34] Assigned 4327 locus_tags to CDS and RNA features.
[20:13:34] Writing outputs to ./G04878_prokka/
[20:13:38] Generating annotation statistics file
[20:13:38] Generating Genbank and Sequin files
[20:13:38] Running: tbl2asn -V b -a r10k -l paired-ends -M n -N 1 -y 'Annotated using prokka 1.12 from https://github.com/tseemann/prokka' -Z \.\/G04878_prokka\/G04878\.err -i \.\/G04878_prokka\/G04878\.fsa 2> /dev/null
[20:13:45] Deleting unwanted file: ./G04878_prokka/errorsummary.val
[20:13:45] Deleting unwanted file: ./G04878_prokka/G04878.dr
[20:13:45] Deleting unwanted file: ./G04878_prokka/G04878.fixedproducts
[20:13:45] Deleting unwanted file: ./G04878_prokka/G04878.ecn
[20:13:45] Deleting unwanted file: ./G04878_prokka/G04878.val
[20:13:45] Repairing broken .GBK output that tbl2asn produces...
[20:13:45] Running: sed 's/COORDINATES: profile/COORDINATES:profile/' < \.\/G04878_prokka\/G04878\.gbf > \.\/G04878_prokka\/G04878\.gbk
[20:13:45] Deleting unwanted file: ./G04878_prokka/G04878.gbf
[20:13:45] Output files:
[20:13:45] ./G04878_prokka/G04878.ffn
[20:13:45] ./G04878_prokka/G04878.sqn
[20:13:45] ./G04878_prokka/G04878.err
[20:13:45] ./G04878_prokka/G04878.gff
[20:13:45] ./G04878_prokka/G04878.gbk
[20:13:45] ./G04878_prokka/G04878.txt
[20:13:45] ./G04878_prokka/G04878.fsa
[20:13:45] ./G04878_prokka/G04878.log
[20:13:45] ./G04878_prokka/G04878.tbl
[20:13:45] ./G04878_prokka/G04878.fna
[20:13:45] ./G04878_prokka/G04878.faa
[20:13:45] ./G04878_prokka/G04878.tsv
[20:13:45] Annotation finished successfully.
[20:13:45] Walltime used: 5.57 minutes
[20:13:45] If you use this result please cite the Prokka paper:
[20:13:45] Seemann T (2014) Prokka: rapid prokaryotic genome annotation. Bioinformatics. 30(14):2068-9.
[20:13:45] Type 'prokka --citation' for more details.
[20:13:45] Thank you, come again.


>>>>> gzip_compression <<<<<


gzip -c G04878_1.fastq > G04878_1.fastq.gz







>>>>> gzip_compression <<<<<


gzip -c G04878_2.fastq > G04878_2.fastq.gz







>>>>> snippy_command <<<<<


snippy --cpus 1 --outdir G04878 --ref ./NC000962_3.gbk --R1 ./G04878_1.fastq.gz --R2 ./G04878_2.fastq.gz


[20:18:04] This is snippy 3.2-dev
[20:18:04] Written by Torsten Seemann <torsten.seemann@gmail.com>
[20:18:04] Obtained from https://github.com/tseemann/snippy
[20:18:04] Detected operating system: linux
[20:18:04] Enabling bundled linux tools.
[20:18:04] Found bwa - /opt/snippy/binaries/linux/bwa
[20:18:04] Found samtools - /opt/snippy/binaries/linux/samtools
[20:18:04] Found tabix - /opt/snippy/binaries/linux/tabix
[20:18:04] Found bgzip - /opt/snippy/binaries/linux/bgzip
[20:18:04] Found snpEff - /opt/snippy/binaries/noarch/snpEff
[20:18:04] Found java - /usr/bin/java
[20:18:04] Found gzip - /usr/bin/gzip
[20:18:04] Found parallel - /opt/snippy/binaries/noarch/parallel
[20:18:04] Found freebayes - /opt/snippy/binaries/linux/freebayes
[20:18:04] Found freebayes-parallel - /opt/snippy/binaries/noarch/freebayes-parallel
[20:18:04] Found fasta_generate_regions.py - /opt/snippy/binaries/noarch/fasta_generate_regions.py
[20:18:04] Found vcfstreamsort - /opt/snippy/binaries/linux/vcfstreamsort
[20:18:04] Found vcfuniq - /opt/snippy/binaries/linux/vcfuniq
[20:18:04] Found vcffirstheader - /opt/snippy/binaries/noarch/vcffirstheader
[20:18:04] Found vcf-consensus - /opt/snippy/binaries/noarch/vcf-consensus
[20:18:04] Found snippy-vcf_to_tab - /opt/snippy/bin/snippy-vcf_to_tab
[20:18:04] Found snippy-vcf_report - /opt/snippy/bin/snippy-vcf_report
[20:18:04] Found snippy-vcf_filter - /opt/snippy/bin/snippy-vcf_filter
[20:18:04] Checking version: samtools --version is >= 1.3 - ok, have 1.3
[20:18:04] Using reference: /media/sf_BioDragon/Genesis/test/G04878_analysis/NC000962_3.gbk
[20:18:04] Treating reference as 'genbank' format.
[20:18:04] Will use 1 CPU cores.
[20:18:04] Using read file: /media/sf_BioDragon/Genesis/test/G04878_analysis/G04878_1.fastq.gz
[20:18:04] Using read file: /media/sf_BioDragon/Genesis/test/G04878_analysis/G04878_2.fastq.gz
[20:18:04] Creating folder: G04878
[20:18:04] Changing working directory: G04878
[20:18:04] Creating reference folder: reference
[20:18:04] Extracting FASTA and GFF from reference.
[20:18:07] Wrote 1 sequences to ref.fa
[20:18:07] Wrote 4416 features to ref.gff
[20:18:07] Creating reference/snpeff.config
[20:18:07] Freebayes will process 4 chunks of 1121279 bp, 1 chunks at a time.
[20:18:07] Using BAM RG (Read Group) ID: snps
[20:18:07] Running: samtools faidx reference/ref.fa 2>> snps.log
[20:18:07] Running: bwa index reference/ref.fa 2>> snps.log
[20:18:08] Running: mkdir reference/genomes && cp -f reference/ref.fa reference/genomes/ref.fa 2>> snps.log
[20:18:08] Running: mkdir reference/ref && bgzip -c reference/ref.gff > reference/ref/genes.gff.gz 2>> snps.log
[20:18:08] Running: snpEff build -c reference/snpeff.config -dataDir . -gff3 ref 2>> snps.log
[20:18:14] Running: (bwa mem  -v 2 -M -R '@RG\tID:snps\tSM:snps' -t 1 reference/ref.fa /media/sf_BioDragon/Genesis/test/G04878_analysis/G04878_1.fastq.gz /media/sf_BioDragon/Genesis/test/G04878_analysis/G04878_2.fastq.gz | samtools sort --reference reference/ref.fa > snps.bam) 2>> snps.log
[20:26:01] Running: samtools index snps.bam 2>> snps.log
[20:26:11] Running: samtools depth -q 20 snps.bam | bgzip > snps.depth.gz 2>> snps.log
[20:26:38] Running: tabix -s 1 -b 2 -e 2 snps.depth.gz 2>> snps.log
[20:26:39] Running: fasta_generate_regions.py reference/ref.fa.fai 1121279 > reference/ref.txt 2>> snps.log
[20:26:39] Running: freebayes-parallel reference/ref.txt 1 -p 1 -q 20 -m 60 --min-coverage 10 -V -f reference/ref.fa snps.bam > snps.raw.vcf 2>> snps.log
[20:33:14] Running: /opt/snippy/bin/snippy-vcf_filter --minqual 10 --mincov 10 --minfrac 0.9  snps.raw.vcf > snps.filt.vcf 2>> snps.log
[20:33:14] Running: snpEff ann -no-downstream -no-upstream -no-intergenic -no-utr -c reference/snpeff.config -dataDir . -noStats ref snps.filt.vcf > snps.vcf 2>> snps.log
[20:33:22] Running: bgzip -c snps.vcf > snps.vcf.gz 2>> snps.log
[20:33:22] Running: tabix -p vcf snps.vcf.gz 2>> snps.log
[20:33:22] Running: /opt/snippy/bin/snippy-vcf_to_tab --gff reference/ref.gff --ref reference/ref.fa --vcf snps.vcf > snps.tab 2>> snps.log
[20:33:25] Running: vcf-consensus snps.vcf.gz < reference/ref.fa > snps.consensus.fa 2>> snps.log
[20:33:25] Running: /opt/snippy/bin/snippy-vcf_filter --subs snps.filt.vcf > snps.filt.subs.vcf 2>> snps.log
[20:33:26] Running: bgzip -c snps.filt.subs.vcf > snps.filt.subs.vcf.gz 2>> snps.log
[20:33:26] Running: tabix -p vcf snps.filt.subs.vcf.gz 2>> snps.log
[20:33:26] Running: vcf-consensus snps.filt.subs.vcf.gz < reference/ref.fa > snps.consensus.subs.fa 2>> snps.log
[20:33:26] Generating aligned/masked FASTA relative to reference: snps.aligned.fa
[20:33:30] Creating extra output files: BED GFF CSV TXT HTML
[20:33:30] Identified 2293 variants.
[20:33:30] Result folder: G04878
[20:33:30] Result files:
[20:33:30] * G04878/snps.aligned.fa
[20:33:30] * G04878/snps.bam
[20:33:30] * G04878/snps.bam.bai
[20:33:30] * G04878/snps.bed
[20:33:30] * G04878/snps.consensus.fa
[20:33:30] * G04878/snps.consensus.subs.fa
[20:33:30] * G04878/snps.csv
[20:33:30] * G04878/snps.depth.gz
[20:33:30] * G04878/snps.depth.gz.tbi
[20:33:30] * G04878/snps.filt.subs.vcf
[20:33:30] * G04878/snps.filt.subs.vcf.gz
[20:33:30] * G04878/snps.filt.subs.vcf.gz.tbi
[20:33:30] * G04878/snps.filt.vcf
[20:33:30] * G04878/snps.gff
[20:33:30] * G04878/snps.html
[20:33:30] * G04878/snps.log
[20:33:30] * G04878/snps.raw.vcf
[20:33:30] * G04878/snps.tab
[20:33:30] * G04878/snps.txt
[20:33:30] * G04878/snps.vcf
[20:33:30] * G04878/snps.vcf.gz
[20:33:30] * G04878/snps.vcf.gz.tbi
[20:33:30] Walltime used: 15 minutes, 26 seconds
[20:33:30] Questionable SNP? Try the --report option to see the alignments.
[20:33:30] Done.


>>>>> snippy_core <<<<<


snippy-core  G04878 


[20:33:30] Loading Snippy folders
[20:33:30] Using ID 'G04878' for G04878
[20:33:30] G04878: found 2155 variants totalling 2311 bp in G04878/snps.tab
[20:33:30] Loaded 1 SNP tables.
[20:33:30] Including 'Reference' in results.
[20:33:30] Will write alignment statistics to core.txt
[20:33:30] Loading pre-masked/aligned sequences...
[20:33:31] 1/1	G04878 coverage 4380493/4411532 = 99.30%
[20:33:31] Patching variant sites into whole genome alignment...
[20:33:31] Constructing alignment object for core.full.aln
[20:33:31] Writing 'fasta' alignment to core.full.aln
[20:33:31] Writing core SNP table
[20:33:31] Found 2221 core SNPs from 2311 variant sites.
[20:33:31] Saved SNP table: core.tab
[20:33:31] Constructing alignment object for core.aln
[20:33:31] Writing 'fasta' alignment to core.aln
[20:33:31] Done.


>>>>> SNPtable <<<<<


SNPtable_filter_Mtb.R core.tab


SNPtable_filter_Mtb v1.0

Author: João Perdigão

Usage: SNPtable_filter_Mtb.R [snippy-core SNP tab file]

Loading required package: ape
Loading required package: seqinr

Attaching package: ‘seqinr’

The following objects are masked from ‘package:ape’:

    as.alignment, consensus

Reading M. tuberculosis library files...

Reading M. tuberculosis library files...OK!

PE/PPE Positions removed...

Positions with mapping quality below k-mer 49/50 removed...

Converting to multifasta format...

Conversion to multifasta format... OK!

Output FASTA file: coreSNP_alignment_filtered.fas


>>>>> HammingFasta <<<<<


HammingFasta.R coreSNP_alignment_filtered.fas


HammingFasta v1.0

Author: João Perdigão

Usage: HammingFasta.R [FASTA alignment file]


Loading required package: ape
Loading required package: seqinr

Attaching package: ‘seqinr’

The following objects are masked from ‘package:ape’:

    as.alignment, consensus

       H37Rv
G04878  2128

Output Matrix File: dists_hamming.csv


>>>>> move_out_of_genome_folder <<<<<




>>>>> Current Directory => /media/sf_BioDragon/Genesis/test/G04878_analysis


