
I will ask you to follow that manual in Pdf.

For a reference, use this one "Reference renamed". I just renamed my reference to the same name as the manual. They must be the same, but just to be sure.

We need a script for:

1 - Trimmomatric (Page 26). Can you see that paramers which make part into the command? We can adjust them. I am sharing with you the one which I used. A worked in that script (See file called "2_Trimommatic_Script_Fiocruz"), but you can follow this manual command.

2 - Mapping (page 27 to 30):

    this script will use the 'paired' files generated by Trimomatric. In our meeting we were using the files called "R1_p.fastq" instead of "_1_trimmed_paired.fastq".
    In our lab we think tha is better giving short names as "p" meaning "paired" and "s" meaning "unpaired" (from "singleton").
    You can follow the manual too.
    For visualizaton of Mapped Data I think we don't need a script.


3 - Variant calling (page 35 to 40).

4 - De novo assembly (page 44 to 46):

    here on page 44 you will have to perfome this script for 49 and 55 (see the figure Observation1 which I've attached).
    See Observation2: among those 3 k-mer, we will use for each genome the one with higher total size information. In this exercise it was k-mer 49, but for my genome yesterday it was 55.
    There's a mistake, which is corrected in figure Observation3.

5 - Contig ordering and scaffolding (page 48 - 51): there's a mistake in this command. See figure Observation4. We stopped here. Our analysis didn't work.

6 - Annotation (page 55 - ): I am also sharing with you our command here.

7 - Snippy calling (page 64): I am also sharing with you our command here.

8 - Snippy core: (pages 65 - 66).
